---
title: "2A. Assessment of the Pacific cod stock in the Aleutian Islands"
output:
  pdf_document:
    fig_caption: true
  word_document: default
---

---
header-includes:
 \usepackage{booktabs}
 \usepackage{float}
 \usepackage{graphicx}
 \usepackage{floatrow}
 \usepackage{subcaption}
 \usepackage{subfigure}
 \usepackage{subfloat}


geometry:
  left=1in,right=1in,top=1in,bottom=1in

---


\begin{centering}
\fontfamily{cmr}
\fontsize{10}{12}
  Ingrid Spies, Maia Kapur, Steve Barbeaux, Melissa Haltuch, Pete Hulson, Ivonne Ortiz, Laura Spencer, and Sandra Lowe \\
\selectfont
  Alaska Fisheries Science Center, National Marine Fisheries Service \\
  National Oceanic and Atmospheric Administration \\ 
  7600 Sand Point Way NE., Seattle, WA 98115-6349 \\
  {`r format(Sys.time(), '%d %B, %Y')`}\\
  
  \includegraphics[width=1.5in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/data/qrcode_github.com.png}

\end{centering}

This report may be cited as:
Spies, I., Kapur, M., Barbeaux, S., Haltuch, M., Hulson, P., Ortiz, I., Spencer, L., and Lowe, S. 2024. Assessment of the Pacific cod stock\ in the Aleutian Islands. North Pacific Fishery Management Council, Anchorage, AK. Available from https://www.npfmc.org/library/safe-reports/.



```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
library(ggplot2)
library(kableExtra)
library(stringr)
library(sjlabelled)
library(sjmisc)
library(r4ss)
library(FSA)
library(nlstools)
library(data.table)
library(sizeMat)
library(ggthemes)
library(plotrix)
library(reshape2)
library(dplyr)
library(png)
library(grid)
library(gridExtra)
library(devtools)
library(captioner)
library(readr)




knitr::opts_chunk$set(fig.align = 'left')

knitr::opts_chunk$set(fig.pos = 'H')
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE, cache=FALSE,results="asis")

f.ref <- function(x) {
  stringr::str_extract(table_nums(x), "[^:]*")
}

table_nums <- captioner::captioner(prefix = "Table 2A.",levels=1,auto_space = FALSE)
figure_nums <- captioner::captioner(prefix="Figure 2A.",levels = 1, auto_space = FALSE)

thisyr    = 2024
lastyr    = thisyr-1
nextyr    = thisyr+1

Dir_RMD="/Users/ingrid.spies/Documents/AI_PCOD/2024/AIPcod2024"

#AKFIN under Observer reports 4/25/24
AI_len_fish=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2025/norpac_length_report_AI_Pcod_9_22_24.csv",header=TRUE)

#Needed a special query for data from 1991 and 1992
cat9192=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/AIcod_catch_1991_1992.csv",header=TRUE)

#AKFIN Special analyst report - Spies. 	Thompson - PCod Total Catch by Jurisdiction  
catchLOC=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/Spies - AI Pcod NORPAC_9_22_24.csv",header=TRUE)
 
#by full months
Num_months=sum(catchLOC$EXTRAPOLATED_WEIGHT[which(catchLOC$MONTH<9&catchLOC$YEAR>2018&catchLOC$YEAR<2024)])

Num_Sept=sum(catchLOC$EXTRAPOLATED_WEIGHT[which(catchLOC$MONTH==9&catchLOC$DAY<23&catchLOC$YEAR>2018&catchLOC$YEAR<2024)])

#Just in September
Denom_months=sum(catchLOC$EXTRAPOLATED_WEIGHT[which(catchLOC$YEAR>2018&catchLOC$YEAR<2024)])
#Denom_September=sum(catchLOC$EXTRAPOLATED_WEIGHT[which(catchLOC$YEAR>2018&catchLOC$MONTH==9&catchLOC$YEAR<2024)])

prop=(Num_months+Num_Sept)/(Denom_months)


cat24=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/GroundfishTotalCatch_AICod_9_22_24.csv",header=TRUE)

yrs=as.numeric(names(table(cat24$Year)))
cat24AI=vector();
cat24trw=vector();
cat24llpot=vector();
cat24ll=vector();
cat24pot=vector();
cat24other=vector();
for(i in 1:length(yrs)){
  cat24AI[i]=sum(cat24$Catch..mt.[which(cat24$Year==yrs[i])])
  cat24trw[i]=sum(cat24$Catch..mt.[which(cat24$Year==yrs[i]&cat24$Gear%in%c("BTR","NPT","PTR","TRW"))])
  cat24llpot[i]=sum(cat24$Catch..mt.[which(cat24$Year==yrs[i]&cat24$Gear%in%c("HAL","POT"))])
  cat24ll[i]=sum(cat24$Catch..mt.[which(cat24$Year==yrs[i]&cat24$Gear%in%c("HAL"))])
  cat24pot[i]=sum(cat24$Catch..mt.[which(cat24$Year==yrs[i]&cat24$Gear%in%c("POT"))])
  cat24other[i]=sum(cat24$Catch..mt.[which(cat24$Year==yrs[i]&cat24$Gear%in%c("GIL","JIG","OTH"))])
}
cat24AI[34]/prop#5132.103t is the total estimate for 2024. Prop=0.7633 as ofSept 22
cat24AI[34]#Total for 2024 as of end of Sept
cat24trw[34]/prop#3419.545
cat24ll[34]/prop#46.20484
cat24pot[34]/prop#4432.612
cat24other[34]
GTC=cbind(cat24trw,cat24ll,cat24pot,cat24other,cat24AI)
colnames(GTC)=c("Trawl","Longline","Pot","Other","Total")
write.csv(GTC,"/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/GTC.csv")
#October ~18


fish_catch=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/GTC.csv",header=TRUE)


#SS3 models
#Dir_M24_1=("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/M24_1/run")
#Dir_M24_0=("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/M24_0/run")
Dir_M24_1=("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run")
Dir_M24_1a=("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1a_data/run")
Dir_M24_0=("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_0_data/run")
Dir_RMD="/Users/ingrid.spies/Documents/AI_PCOD/2024/AIPcod2024"
M24_0 <- SS_output(Dir_M24_0)
M24_1 <- SS_output(Dir_M24_1)
M24_1a <- SS_output(Dir_M24_1a)

mod.sum <- SSsummarize(list(M24_1,M24_1a,M24_0))

retroModelsM1 <- SSgetoutput(
  dirvec = file.path(Dir_M24_1, "retrospectives", paste("retro", 0:-10, sep = ""))
)

retroSummary1 <- SSsummarize(retroModelsM1)

#exploitable biomass (by the survey)
expbio=retroSummary1$indices$Vuln_bio[which(retroSummary1$indices$imodel==1&retroSummary1$indices$Yr==2024)]

#Trying to get to how the mocel calculates $Bio_smry
#Age selectivity

asel=M24_1$ageselex[M24_1$ageselex$Factor == "Asel2", ]
#asel[which(asel$Yr==2025&asel$Fleet==1),][9:21] #Fishery selectivity
#Asel=asel[which(asel$Yr==2025&asel$Fleet==2),][9:21] #Survey selectivity

#Best guess for total biomass
#natage=M24_1$natage[which(M24_1$natage$Yr==2024&M24_1$natage$`Beg/Mid`=="M"),][14:26] #natatage
#wtage=M24_1$wtatage[which(M24_1$wtatage$Yr==2024&M24_1$wtatage$Fleet==2),][8:20] 
#sum(natage*wtage)

#Should catchablity be added? exp(-0.110266) No, it doesnot work
#natage=M24_1$natage[which(M24_1$natage$Yr==2024&M24_1$natage$`Beg/Mid`=="M"),][14:26] #natatage
#wtage=M24_1$wtatage[which(M24_1$wtatage$Yr==2024&M24_1$wtatage$Fleet==2),][8:20] 
#sum(natage*wtage*0.8955959)

#Best guess for total exploitable biomass - MATCHES!!!??
natage=M24_1$natage[which(M24_1$natage$Yr==2024&M24_1$natage$`Beg/Mid`=="M"),][14:26] #natatage
wtage=M24_1$wtatage[which(M24_1$wtatage$Yr==2024&M24_1$wtatage$Fleet==2),][8:20] 
Aselsrv=asel[which(asel$Yr==2025&asel$Fleet==2),][9:21]
expbio25=sum(natage*wtage*Aselsrv)

fsep = "/"


Srv_this=M24_0$cpue$Obs[which(M24_0$cpue$Yr==2024&M24_0$cpue$Fleet_name=="Srv")]
Srv_thisF=formatC(Srv_this,format="d",big.mark=",")

Srv_last=M24_0$cpue$Obs[which(M24_0$cpue$Yr==2022&M24_0$cpue$Fleet_name=="Srv")]
Srv_lastF=formatC(Srv_last,format="d",big.mark=",")

#run AK scenarios 2024 with M24_1
#write.csv(PM24$Two_year,paste(Dir_M24_1a,"AK_Scenarios/Two_year.csv",sep="/"))
#write.csv(PM24$Tables$Catch,paste(Dir_M24_1a,"AK_Scenarios/Catch_proj.csv",sep="/"))
#write.csv(PM24$Tables$F,paste(Dir_M24_1a,"AK_Scenarios/F_proj.csv",sep="/"))
#write.csv(PM24$Tables$SSB,paste(Dir_M24_1a,"AK_Scenarios/SSB_proj.csv",sep="/"))
#PM24$FIGS[[1]]$ALL save as AK_Scenarios_ProjALL
#PM24$FIGS[[2]]$ALL save as AK_Scenarios_CatchALL

Two_year=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run/AK_Scenarios/Two_year.csv")

Two_year24_0=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_0_data/run/AK_Scenarios/Two_year.csv")

Two_year24_1a=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1a_data/run/AK_Scenarios/Two_year.csv")

Catch_proj=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run/AK_Scenarios/Catch_proj.csv")
F_proj=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run/AK_Scenarios/F_proj.csv")
SSB_proj=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run/AK_Scenarios/SSB_proj.csv")

TotBio_thisF=formatC(M24_0$timeseries$Bio_smry[which(M24_0$timeseries$Yr==thisyr+1)],big.mark=",",format="d")

TotBio_nextF=formatC(M24_0$timeseries$Bio_smry[which(M24_0$timeseries$Yr==thisyr+2)],big.mark=",",format="d")

SSB_thisF=formatC(Two_year$SSB[1],big.mark=",",format="d")
SSB_nextF=formatC(Two_year$SSB[2],big.mark=",",format="d")
FOFL_this=round(Two_year$F35[1],3)
FOFL_next=round(Two_year$F35[2],3)
FABC_this=round(Two_year$F40[1],3)
FABC_next=round(Two_year$F40[2],3)
ABC_this=round(Two_year$C_ABC[1])
ABC_next=round(Two_year$C_ABC[2])
OFL_this=round(Two_year$C_OFL[1])
OFL_next=round(Two_year$C_OFL[2])
ABC_thisF=formatC(ABC_this,big.mark=",",format="d")
ABC_nextF=formatC(ABC_next,big.mark=",",format="d")
OFL_thisF=formatC(OFL_this,big.mark=",",format="d")
OFL_nextF=formatC(OFL_next,big.mark=",",format="d")

B35_this=Two_year$SB35[1]
B35_thisF =formatC(Two_year$SB35[1],big.mark=",",format="d")


#M24_0_Mohns=read.csv(paste(Dir_M24_0a,"/retrospectives1/Mohn.csv",sep=""),header=TRUE)
#Rho_M24_0=round(M24_0_Mohns[11,4],3)

#Tier 5 below M13.2
REMA=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/REMA/AI_REMA.csv")

T5bio=REMA$Biomass[nrow(REMA)]#biomass
T5_low90=REMA$LCI[nrow(REMA)]#low90th
T5_upp90=REMA$UCI[nrow(REMA)]#upp90th

#Tier 5 reference points
T5TotBio_this=REMA$Biomass[length(REMA$Biomass)]
T5TotBio_thisF=formatC(T5TotBio_this,digits=0,big.mark=",",format="d")

T5TotBio_next=T5TotBio_this
T5TotBio_nextF=T5TotBio_thisF
T5TotBio_last=54165
T5TotBio_lastF="54,165"

T5m=0.34

T5OFL_this=T5m*T5TotBio_this
T5OFL_thisF=formatC(T5OFL_this,digits=0,format="d",big.mark=",")

T5OFL_this24_2=0.417*T5TotBio_this
T5OFL_thisF24_2=formatC(T5OFL_this24_2,digits=0,format="d",big.mark=",")

T5OFL_next=T5OFL_this
T5OFL_nextF=T5OFL_thisF
T5OFL_last=18416
T5OFL_lastF="18,416"

T5ABC_this=0.75*T5OFL_this
T5ABC_thisF=formatC(T5ABC_this,digits=0,big.mark=",",format="d")

T5ABC_this24_2=0.75*T5OFL_this24_2
T5ABC_thisF24_2=formatC(T5ABC_this24_2,digits=0,big.mark=",",format="d")

T5ABC_next=T5ABC_this
T5ABC_nextF=T5ABC_thisF
T5ABC_last=13812
T5ABC_lastF="13,812"

T5TAC_lastF="8,425"

T5ABC_2024="12,431" #This is the actual ABC
T5OFL_2024="18,416"  #Actual OFL

#`r opts_chunk$set(cache=TRUE)`


#Evaluate FABC and FOFL 
#In that code:
#F=data.table(mods1[[8]]$sprseries)[Yr%in%c(SYR:EYR)]$F_report
#F=data.table(mods1[[i]]$sprseries)[Yr%in%c(SYR:EYR)]$F_report


#Calculate the probabiliyt of being below B20%
M24_1DT=data.table(M24_1$derived_quants)
M24_1DT[Label=="SSB_2024"]$Value
SSB0_M24_1<-data.table(M24_1$derived_quants)[Label=='SSB_unfished']$Value
x=rnorm(100000, M24_1DT[Label=="SSB_2025"]$Value, M24_1DT[Label=="SSB_2025"]$StdDev)
BX2023=data.table(M24_1$derived_quants)[Label=='SSB_2025']$Value/SSB0_M24_1

x2=x/SSB0_M24_1
P_belowB20=round((sum(x2<=0.2)/100000),2)

#Sara file for total. bimoass estimate I used 
# M24_1DT[Label=="SSB_2024"]$StdDev for total biomass stddev.
#totbio round(M24_1$timeseries$Bio_all)[3:36]
#round(M24_1$timeseries$Recruit_0)[3:36]
#round((M24_1$timeseries$SpawnBio)[3:36]/2)
# Fishing mortality M24_1$derived_quants$Value[152:185]
#multiplying a random variable by a constant b causes the variance to be multiplied by b^2
#est=M24_1DT[Label=="SSB_2026"]$Value/SSB0_M24_1
#var=(M24_1DT[Label=="SSB_2026"]$StdDev)^2*(1/SSB0_M24_1)^2
#sd=sqrt(var)
#P_belowB20_2=pnorm(.2,est,sd);P_belowB20_2

#data.table(M24_0$derived_quants)[Label=='SSB_2023']$Value/SSB0_M24_0

#This is the scenarios file


#write.csv(PM24$Two_year,"/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/M24_1/run/Two_year.csv")

Rho_24_0=round(read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_0_data/run/retrospectives/Mohn.csv",header=TRUE)[11,4],3)
Rho_24_1=round(read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run/retrospectives/Mohn.csv",header=TRUE)[11,4],3)

M24_1_Two_Year=read.csv(file.path(Dir_M24_1,'AK_Scenarios','Two_Year.csv'))

#M24_0_Two_Year=read.csv(file.path(Dir_M24_0,'AK_Scenarios','Two_Year.csv'))

#Get data gap products
#library(devtools)
#devtools::install_github("afsc-gap-products/gapindex")
#library(gapindex)

#Get survey input sample size
#library(devtools)
#devtools::install_github("BenWilliams-NOAA/surveyISS")

#surveyISS::query_data(survey = 52, 
#                              region = 'ai', 
#                              species = 21720, 
#                              yrs = 2024)

```


```{r echo=FALSE, out.width = "30%", fig.align = "center"}
#knitr::include_graphics("/Users/ingrid.spies/Documents/AI_PCOD/Assessment/qrcode_github.com.png") 


```


```{r calcs,echo=FALSE,results=FALSE}
report=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/Thompson - PCod Total Catch by Jurisdiction_9_22_24.csv",header=TRUE)
#Retained and discarded Pacific Cod catch in all target fisheries, reported by subarea, jurisdiction, month, and gear. Source: NMFS AKRO BLEND/Catch Accounting System; available 1991-present.Time run: 10/26/2023 5:54:08 AM
yrs=as.numeric(names(table(catchLOC$YEAR)))
pot=vector()
for(i in 1:length(yrs)){
 pot[i]=sum(catchLOC$EXTRAPOLATED_WEIGHT[which(catchLOC$YEAR==yrs[i]&catchLOC$GEAR==6)])
}
cbind(yrs,pot/1000)

yrs2=as.numeric(names(table(report$Year)))
potF=vector();potS=vector();llF=vector();llS=vector();trawlF=vector();trawlS=vector()
for(i in 1:length(yrs)){
 trawlF[i]=sum(report$Catch..mt.[which(report$Gear%in%c("BTR","PTR","TRW","NPT")&report$Year==yrs[i]&report$State.Federal.Fishery=="F")])
  trawlS[i]=sum(report$Catch..mt.[which(report$Gear%in%c("BTR","PTR","TRW","NPT")&report$Year==yrs[i]&report$State.Federal.Fishery=="S")]) 
   llF[i]=sum(report$Catch..mt.[which(report$Gear=="HAL"&report$Year==yrs[i]&report$State.Federal.Fishery=="F")])
  llS[i]=sum(report$Catch..mt.[which(report$Gear=="HAL"&report$Year==yrs[i]&report$State.Federal.Fishery=="S")])  
 potF[i]=sum(report$Catch..mt.[which(report$Gear=="POT"&report$Year==yrs[i]&report$State.Federal.Fishery=="F")])
  potS[i]=sum(report$Catch..mt.[which(report$Gear=="POT"&report$Year==yrs[i]&report$State.Federal.Fishery=="S")])
}

cgear=cbind(yrs,round(pot/1000),round(potS),round(potF),round(llS),round(llF),round(trawlS),round(trawlF))
colnames(cgear)=c("Year","IS report pot catch","Thompson pot catch State","Thompson pot catch Federal","Longline state","Longline federal","Trawl state","Trawl federal")
write.csv(cgear,"/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/cgear.csv")

#####HERE INGRID
POT_prop=as.numeric(gsub(",","",fish_catch$Pot))/sum(as.numeric(gsub(",","",fish_catch$Total)))
POT_prop20_24=mean(POT_prop[30:34])#Proportion of pot out of all fisheries (state and federal)
Jig_avg=round(mean(report$Catch..mt.[which(report$Gear=="JIG"&report$State.Federal.Fishery=="F")]),0)#in tons...

#Proportion by gear in state catches
cgearSTATE=cgear[,3]+cgear[,5]+cgear[,7]
statepot=cgear[,3]/cgearSTATE
stateLL=cgear[,5]/cgearSTATE
stateTRAWL=cgear[,7]/cgearSTATE

#Proportion by gear in federal catches
cgearFED=cgear[,4]+cgear[,6]+cgear[,8]
FEDpot=cgear[,4]/cgearFED
FEDLL=cgear[,6]/cgearFED
FEDTRAWL=cgear[,8]/cgearFED

State_prop=round(sum(report$Catch..mt.[which(report$State.Federal.Fishery=="S"&report$Year==2024)])/sum(report$Catch..mt.[which(report$Year==2024)]),2)

catAI=cat24
W_cat=sum(catAI$Catch..mt.[which(catAI$NMFS.Area==543&catAI$Year>2019)])
C_cat=sum(catAI$Catch..mt.[which(catAI$NMFS.Area==542&catAI$Year>2019)])
E_cat=sum(catAI$Catch..mt.[which(catAI$NMFS.Area==541&catAI$Year>2019)])
tot=sum(catAI$Catch..mt.[which(catAI$Year>2019)])

#All selectivity curves were implemented as double normal, which contains six parameters, but only two were estimated (selectivity parameters 1 and 3). This allows for monotonically increasing asymptotic selectivity, and was configured so that the second double normal defining the descending slope was at the upper bound and only the first upward sloping normal was used to model selectivity. A third parameter was estimated in the case of dome-shaped selectivity. 


TableCompare=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/data/TableComparisons1.csv",header=FALSE)




```

# Executive summary

The Aleutian Islands (AI) and eastern Bering Sea (EBS) Pacific cod stocks were first managed separately in 2014. Since 2014, age-structured models have been explored in assessments but harvest specifications for Aleutian Islands (AI) Pacific cod have been managed under Tier 5.  This document presents three age structured models for the Aleutian Islands Pacific cod stock, as well as two Tier 5 harvest specification models.

Age structured models:

* Model 24.1: This model includes a timeblock on natural mortality from 2016 - 2024. The breakpoint between 2015 and 2016 corresponds to a shift to warmer temperatures in the Aleutian Islands during the past decade (Xiao and Ren 2022). This is the preferred model.

* Model 24.0: This is a sensitivity model, which is similar to Model 24.1 but does not contain the natural mortality timeblock from 2016-2024.

* Model 24.1a: This model shows sensitivity to the Richards vs. von Bertalanffy growth curves. It differs from Model 24.1 in its growth curve, using the von Bertalanffy rather than the Richards growth curve. 

Tier 5 random effects model:

* Model 13.4: There has been no change to the input data for the Tier 5 models; they use existing biomass estimates from 1991 - 2024  implemented using the REMA package. The natural mortality estimate (M = 0.34) used in past models is retained for `r thisyr`.

* Model 24.2: This is the same Tier 5 model as 13.4, except it assumes natural mortality, M = 0.417. This is the preferred Tier 5 model because it assumes a new externally estimated natural mortality requested by the SSC and Plan Team in 2023.

### Summary of changes in assessment inputs

The following substantive changes have been made to the Aleutian Islands Pacific cod age structured assessment relative to the November `r thisyr-1` assessment. 

#### Changes in the input data (Tier 3 models)

*	Realized catches for `r thisyr-1`, as well as a preliminary catch estimate through September 22, `r thisyr`. The current year's catch was projected to the end of the year based on the proportion caught over the past 5 years after this the period prior to September 22.
*	Commercial fishery size compositions for `r thisyr-1`, as well as preliminary size composition from the `r thisyr` commercial fisheries through September 22.
*	The 2024 survey biomass index and error estimates are incorporated, as well as the 2024 length compositions and estimate of survey input sample size.
* The maturity curve was updated through September 2024 with observer records of maturity at length.
* The fishery length composition sample sizes were updated with the number of hauls per year, weighted such that the mean is equivalent to the mean survey input sample size.

#### Changes in the input data (Tier 5 model)
 
* The Tier 5 model uses survey biomass estimates from 1991 - 2024.
* Model 24.2 incorporates a new fixed M, 0.417.

#### Changes in the assessment methodology

* Initial _F_ was estimated based on the average catch from 1981-1990. 
* Natural mortality was estimated externally using (http://barefootecologist.com.au/shiny_m.html) and fixed at 0.417, except in Model 24.1 after 2015 (which incorporated a time block on natural mortality), in response to SSC comments. 
* The timeblock on natural mortality from 2016-2024 (Model 24.1) was estimated.
* A Richards growth curve was estimated within the models, except Model 24.1a which used the von Bertalanffy growth curve.
* Maximum age was changed from 10$^+$ to 13$^+$. 
* Fishery length composition did not incorporate a plus group (max = 143 cm) whereas previous models used a plus group of 117$^+$ cm. 
* The time of settlement was changed to indicate that larvae settle as juveniles in the same year as spawning, rather than in the following year, for biological accuracy and consistency with other cod models.

## Summary of Results  

The 2024 catch of Pacific cod in the Aleutian Islands as of September 22, `r thisyr` was `r formatC(fish_catch[34,6],format="d",big.mark=",")` t.  Over the past 5 years (`r thisyr-5` - `r thisyr-1`), `r 100*round(prop,3)`% of the catch has taken place by this date. Therefore, the full year's estimate of catch in `r thisyr` was extrapolated to be `r formatC(fish_catch[34,6]/prop,big.mark=",",format="d")` t. This is lower than the average catch over the past five years of `r formatC(mean(fish_catch[29:33,6]),big.mark=",",format="d")` t. 

Model 24.1 improved the fit to the data over Model 24.0. Model 24.1 estimated a total biomass of `r formatC(M24_1$timeseries$Bio_all[37],format="d",big.mark=",")` t, a spawning biomass of `r formatC(Two_year$SSB[1],big.mark=",",format="d")` t, and an exploitable biomass of `r formatC(expbio25,big.mark=",",format="d")` t for `r thisyr+1`. Model 24.1 ABCs were `r formatC(Two_year$C_ABC[1],big.mark=",",format="d")` t and `r formatC(Two_year$C_ABC[2],big.mark=",",format="d")` t for `r thisyr+1` and `r thisyr+2`. Model 24.1 OFLs were `r formatC(Two_year$C_OFL[1],big.mark=",",format="d")` t and `r formatC(Two_year$C_OFL[2],big.mark=",",format="d")` t for `r thisyr+1` and `r thisyr+2` (`r  table_nums("M24_0_1_comparison", display="cite")`). 

The random effects model which estimated biomass used in Tier 5 models 13.4 and 24.2 estimated an exploitable biomass of `r formatC(round(REMA$Biomass[34]),format="d",big.mark=",")` t. This is comparable with the exploitable survey biomass estimated by Model 24.1 (`r formatC(expbio25,big.mark=",",format="d")` t). The Model 13.4 Tier 5 ABCs and OFLs for `r thisyr+1` and `r thisyr+2` are `r T5ABC_thisF` t and `r T5OFL_thisF` t (`r table_nums("Tier5_comparison",display="cite")`). Note the Tier 5 ABCs and OFLs are the same for `r thisyr+1` and `r thisyr+2`. The Tier 5 Model 24.2 estimated higher OFLs (`r T5OFL_thisF24_2` t) and ABCs (`r T5ABC_thisF24_2` t) for `r thisyr+1` and `r thisyr+2` due to the higher natural mortality parameter (`r  table_nums("Tier5_comparison", display="cite")`).

We prefer adopting the Tier 3 model 24.1 for ABC/OFL and status determimation advice under the FMP. This model makes use of more information (fishery and survey lengths, survey ages, maturity curve) and allows the Council to have clear advice on the status of this important stock. We recommend a reduction in ABC for the Tier 5 model to the Tier 3 recommended ABC.


Summary table for Model 24.2. The Tier 5 Model 13.4 was the accepted model in 2023.

\begin{table}[ht]
\centering
\begin{tabular}{lrr|rr}
  \hline
       & \multicolumn{2}{c|}{As estimated or $\mathit{specified}$ } & \multicolumn{2}{c}{As estimated or $\mathit{recommended}$ }  \\
       & \multicolumn{2}{c|}{$\mathit{last}$ year for:}  & \multicolumn{2}{c}{$\mathit{this}$ year for: }               \\
        Quantity & `r thisyr`      &`r thisyr+1`   & `r thisyr+1`      &`r thisyr+2` \\ 
  \hline
	$M$ (natural mortality rate)	&	0.34	&	0.34	&	0.34	&	0.34 \\
Tier	&	5	&	5	&	5	&	5 \\
Biomass (t)	& `r T5TotBio_lastF` & `r T5TotBio_lastF` & `r T5TotBio_thisF` & `r T5TotBio_nextF`  \\
$F_{OFL}$    	&	0.34	&	0.34	&	0.417	&	0.417	\\
$maxF_{ABC}$   	&	0.255	&	0.255	&	0.313	&	0.313  	\\
$F_{ABC}$       &	0.255	&	0.255	&	0.313	&	0.313        \\
$OFL$    	      & `r T5OFL_2024`  & `r T5OFL_2024`  & `r T5OFL_thisF24_2`  & `r T5OFL_thisF24_2`      \\
$maxABC$    	  & `r T5ABC_2024` & `r T5ABC_2024` & `r T5ABC_thisF24_2` & `r T5ABC_thisF24_2`  \\
$ABC$ 	       & `r T5ABC_2024` & `r T5ABC_2024` & `r formatC(Two_year$C_ABC[1],big.mark=",",format="d")` & `r formatC(Two_year$C_ABC[2],big.mark=",",format="d")`      \\
\hline
Status	                              &	`r thisyr-2`	      &	 `r thisyr-1`	      &		`r thisyr-1`             &	`r thisyr`          \\
\hline
Overfishing	                          &	No	        &	n/a	      &	No	                       &	n/a                 \\
\hline
\end{tabular}
\end{table}


Summary table for Model 24.1. The Tier 5 Model 13.4 was the accepted model in 2023. *Natural mortality estimated within 2016-2024 timeblock.

\begin{table}[ht]
\centering
\begin{tabular}{lrr|rr}
  \hline
       & \multicolumn{2}{c|}{As estimated or $\mathit{specified}$ } & \multicolumn{2}{c}{As estimated or $\mathit{recommended}$ }  \\
       & \multicolumn{2}{c|}{$\mathit{last}$ year for:}  & \multicolumn{2}{c}{$\mathit{this}$ year for: }               \\
        Quantity & `r thisyr`      &`r thisyr+1`   & `r thisyr+1`      &`r thisyr+2` \\ 
  \hline
$M$ (natural mortality rate)          &  0.34   &  0.34   &  `r round(as.numeric(M24_1$Natural_Mortality[25,23]),2)`, `r round(as.numeric(M24_1$Natural_Mortality[36,23]),2)`*   & `r round(as.numeric(M24_1$Natural_Mortality[25,23]),2)`, `r round(as.numeric(M24_1$Natural_Mortality[36,23]),2)`*   \\
Tier                                  &  5   &  5   &  3b   & 3b   \\
Projected total (age 1+) biomass (t)  &  `r T5TotBio_lastF`  &  `r T5TotBio_lastF`  &  `r formatC(M24_1$timeseries$Bio_smry[which(M24_1$timeseries$Yr==thisyr+1)],big.mark=",",format="d")` & `r formatC(M24_1$timeseries$Bio_smry[which(M24_1$timeseries$Yr==thisyr+2)],big.mark=",",format="d")` \\
Projected female spawning biomass (t) &  -  &  -  &  `r formatC(Two_year$SSB[1],big.mark=",",format="d")` & `r formatC(Two_year$SSB[2],big.mark=",",format="d")` \\
$\:\:\:\:\:\:B_{100\%}$                           &  -  &  -  &  `r formatC(Two_year$SB100[1],big.mark=",",format="d")` & `r formatC(Two_year$SB100[2],big.mark=",",format="d")` \\
$\:\:\:\:\:\:B_{40\%}$                            &  -  &  -  &  `r formatC(Two_year$SB40[1],big.mark=",",format="d")` & `r formatC(Two_year$SB40[2],big.mark=",",format="d")`  \\
$\:\:\:\:\:\:B_{35\%}$                            &  -  &  -  &  `r formatC(Two_year$SB35[1],big.mark=",",format="d")` & `r formatC(Two_year$SB35[2],big.mark=",",format="d")` \\
$F_{OFL}$                             &  0.34   &  0.34   &  `r round(Two_year$F35[1],3)`   & `r round(Two_year$F35[2],3)`    \\
$maxF_{ABC}$                          &  0.255  &  0.255  &  `r round(Two_year$F40[1],3)`    & `r round(Two_year$F40[2],3)`    \\
$F_{ABC}$                             &  0.255   &  0.255   &  `r round(Two_year$F40[1],3)`   & `r round(Two_year$F40[2],3)`  \\
$OFL$                                 &  `r T5OFL_2024`  &  `r T5OFL_2024`  &  `r formatC(Two_year$C_OFL[1],big.mark=",",format="d")` & `r formatC(Two_year$C_OFL[2],big.mark=",",format="d")` \\
$maxABC$                              &  `r T5ABC_2024` &  `r T5ABC_2024`  &  `r formatC(Two_year$C_ABC[1],big.mark=",",format="d")` & `r formatC(Two_year$C_ABC[2],big.mark=",",format="d")` \\
$ABC$                                 &  `r T5ABC_2024`  &  `r T5ABC_2024`  & `r formatC(Two_year$C_ABC[1],big.mark=",",format="d")` & `r formatC(Two_year$C_ABC[2],big.mark=",",format="d")` \\
\hline
Status                               & `r thisyr-2`       & `r thisyr-1`      &  `r thisyr-1`             & `r thisyr`          \\
\hline
Overfishing                           & No         & n/a       & No                        & n/a                 \\
Overfished                           & n/a         & No       & n/a                        & No                  \\
Approaching overfished               & n/a         & No       & n/a                        & No                  \\
\hline
\end{tabular}
\begin{tablenotes}
\item 
  \end{tablenotes}
\end{table}


\pagebreak

## Responses to SSC and Plan Team Comments on Assessments in General

_SSC October 2023_

The SSC is encouraged by the development of One-Step-Ahead (OSA) residuals as an improvement to Pearson residuals for assessing fits to compositional data. The SSC welcomes a presentation on their use and interpretation, as well as a discussion of how to select one age to remove from the calculation. The SSC recognizes that the first and last age in many assessments prove challenging to fit, and therefore are the target of specific evaluation of residuals, making it problematic to remove them. The SSC encourages exploration of alternative approaches that may include calculating the OSA residuals with multiple ages removed one at a time and/or adding a compositional bin (e.g., age-1 if the first age with appreciable data is age-2). Another approach to consider would be a two-step process, producing the OSA residuals with the youngest age removed, then using those residuals to identify the best fitting age, then removing that age in the next step.

_Response_

We have incorporated OSA residuals in this assessment.

_SSC December 2023_

When risk scores are reported, the SSC requests that a brief justification of the score be provided, even when that score indicates no elevated risk.

_Response_

Noted.


## Responses to SSC and Plan Team Comments Specific to this Assessment

_SSC December 2023_

When estimating catch for projections, a more realistic value than maxABC should be considered, given that maxABC has not been achieved in recent years.

_Response_

We used 0.4*max$F_{ABC}$ for all future years, which yields F = 0.201 for all future years for Model 24.1, F = 0.288 for Model 24.1a, and F = 0.333 for Model 24.0. This is much lower than the 2031 - 2039 $F_{ABC}$ = 0.804 in Scenario 1 for Model 24.1, $F_{ABC}$ = 1.156 for Model 24.1a, and $F_{ABC}$ = 0.813 for Model 24.0.

_SSC December 2023_

The SSC recommends a sensitivity analysis and a possible prior on M. It is surprising that estimating M was difficult in the data-rich EBS Pacific cod assessment, but  estimating M in the AI cod assessment was successful with fewer data points. The SSC encourages further collaboration among authors of the three cod assessments with regard to the treatment of M.

_Plan Team September 2023_

The Plan Team favors constraining M. They recommend M with a prior based on a reasonable approach. 

_Response to the two comments above in regard to M_

Based on further collaboration among cod assessment authors with regard to the treatment of M, we have estimated M outside the model using the _Then_lm_ methodology (http://barefootecologist.com.au/shiny_m.html), which resulted in a value of 0.417. A sensitivity was performed (`r table_nums("TableCompare",display="cite")`), and results are discussed below.

_SSC December 2023_

Public comment further highlighted challenges with the risk table. Elevated risk was identified but a recommended reduction from maximum ABC or rationale for no reduction was not presented for the base model. Their view was that AI cod have declined within the AI region, though they were uncertain as to the magnitude of the decline.

_Response_

In the 2024 assessment we have the same Risk Table scores as in 2023 (see Risk Table section). We recommend a reduction in ABC for the Tier 5 model to the Tier 3 recommended ABC. 

_SSC December 2023_

Fishery CPUE would be more appropriate to consider under fishery performance in the risk table.

_Response_ 

Fishery CPUE is not indicative of abundance or stock status. Declines in CPUE may be attributed to the timing of the fishery relative to spawning season or other factors such as hyperaggregation during spawning in the trawl fishery (Rose and Kulka 1999). Standardized surveys are needed to understand whether declines in fishery CPUE are indicative of declines or increases in Aleutian Islands Pacific cod stock size.

_SSC December 2023_

There were three parameters for natural mortality (M) where only two were required. This is confusing and might have influenced the model results in ambiguous ways that were not fully described in the document. Standard practice would be to estimate two lognormal parameters for the two M blocks.

_Response_ 

The M timeblock in Model 24.1 is now estimated using only a single parameter as suggested.

_SSC December 2023_

Similarly, there were three parameters estimated for time varying growth where only two were required. As a result, the SSC has the same concerns as noted in the previous bullet. In addition, the author presented a slide showing almost identical growth coefficients for time varying kappa in two periods, suggesting that time-varying growth may not be needed.

_Response_

Time-varying growth was removed from the model based on this suggestion.

_SSC December 2023_

The 'q sensitivity' model, where q is calculated analytically but was almost identical to the estimated value, resulted in significant impacts on model results, indicating potential convergence issues or other inconsistencies in the model.

_Response_

Noted. This statement was in reference to the 2023 assessment. There have been numerous changes and improvements to the model since 2023. `r table_nums("TableCompare",display="cite")` lists eight changes and details bridging model analysis towards the final recommended model.

_SSC December 2023_

The SSC recommends that the authors present a simplified version of the original September 23.0 model with minimal time varying parameters alongside a preferred model or set of models in September 2024.

_Response_

Based on this comment, Model 24.0 is provided which represents a simplified version of the September 23.0 model. 

_SSC December 2023_

In the ESR, the need for an indicator of winter bottom temperature during spawning was noted, possibly derived from winter fisheries data, to assess potential detrimental effects of high temperatures in the AI on spawning and egg survival.

_Response_

The ESR group indicated that reliable bottom temperatures in the Aleutians in winter is difficult. The ROMS model does not capture the Aleutians, which would require observational data, which is infrequent and may not be properly calibrated.

For the future, the marine mammal laboratory at AFSC has placed a mooring at Unimak Pass for 3 years which could provide bottom temperature throughout the year. This data is not available currently. In addition, bottom temperatures in some regions remain relatively constant year round so that summer data could serve as a potential indicator.

_SSC December 2023_

Although the SSC previously recommended that AI Pacific cod be on an annual cycle, the SSC welcomes author and GPT feedback on whether moving to a biennial assessment would be beneficial to allow for more model development time, while coinciding with new AI survey data. Because 2024 will have a new survey, this consideration for a biennial cycle could begin after 2024, especially if a Tier 3 model is accepted in 2024.

_Response_

Noted. 

_SSC October 2024_

One change that was unexpectedly impactful was the move to the Richard’s growth curve. As the BSAI GPT noted, the change in likelihood between the model with the LVB growth and the model with the Richard’s growth curves was quite substantial, given that the change represented the addition of one parameter only and the difference in shape of the two growth curves was fairly similar. The SSC concurs with the BSAI GPTs recommendation to explore how such a large improvement in likelihood occurred despite similar growth curves.

_Response_

The substantial change was a result of the bridging method, and is addressed in Results of Tier 3 models. In a new set of bridging models, the Richards growth curve still provides a better fit to the data, but the improvement is closer to what would be expected. 

_SSC October 2024_

The presentation to the BSAI GPT highlighted disparities in the conditional age at length fits at larger ages, but this was not discussed in the document. The SSC would appreciate some written evaluation of this in the final 2024 SAFE document.

_Response_

Conditional age at length plots and discussion have been added to this document.

_SSC October 2024_

The BSAI GPT endorsed bringing forward models 24.1, 24.0, and Tier 5 models for November, recommending an additional bridging model 24.1a, which uses von Bertalanffy growth instead of the Richard's curve. The SSC concurs with this recommendation and suggests an additional model run below.

_Response_

We have incorporated this request into the 2024 assessment.

_SSC October 2024_

While the use of the Richard’s curve resulted in the most dramatic change in the likelihood, an important new feature of Model 24.1 was the inclusion of a natural mortality block. The SSC appreciates the author’s effort at incorporating climate related parameters, but the current parameterization assumes that there is a distinct mechanism identified for the natural mortality block and that it is consistent over time without an identified threshold for returning to baseline natural mortality. The temperature thresholds identified in Laurel and Rogers (2020) point to hatch success having a narrow optimum temperature range. If hatch success was the main driver, then the SSC suggests it might be more appropriate to consider a recruitment covariate that utilizes bottom temperature predictions at the time of spawning based upon ROMS or future CEFI models. Additionally, if the higher mortality affects young ages or all ages, then the rationale for the two year lag becomes unclear. The higher temperatures in the AI are also lower than what were identified as high temperatures in the GOA. Regardless, future efforts could be enhanced by tying parameters to a specific covariate which would adjust to baseline when it returns to “normal”. Thus the SSC recommends one additional model “24.0b” which removes the block on M and uses the LVB growth model as a simpler but improved model from last year as an alternative. The SSC appreciates the climate related explorations and welcomes future efforts to include environmental covariates.

_Response_

This new model ("24.0b") was not presented due to the short turnaround between the October 2024 SSC meeting and the November assessment deadlines. We note that Model 24.0b would likely not be considered for management because it would removes two features that have been shown to deteriorate the fit to the data. 

We acknowledge that we do not fully understand the physiological effect that a prolonged heatwave would have on natural mortality in Aleutian Islands Pacific cod. Recent experimental work indicates that Pacific cod early life stages are sensitive to warming, negatively impacting survival rates from the embryo through juvenile stages (Laurel et al. 2023). Larval hatch rate decreases with temperature above the optimal range for incubation (Laurel and Rogers 2020), and survival from hatch to the post-flexion stage is considerably lower in warming (Slesinger et al. 2024). Juveniles, while more tolerant of warming, have reduced condition and liver lipid content, which is a critical energy source that is likely to affect overwintering success (Copeman et al. 2022). We acknowledge that if hatch success is a driver of natural mortality, then an environmentally-linked recruitment covariate may be appropriate for future research. 

We also acknowledge that heat stress has been shown to increase bioenergetic consumption rates, and that prey may become a limiting factor after prolonged periods of high temperature, which can result in higher mortality rates. Below average length-weight residuals have been observed in Aleutian Islands Pacific cod in all regions since 2012 (`r figure_nums("FishCondition",display="cite")`). The observations of lower fish condition, heatwave conditions, and decline in survey estimates of biomass are consistent with the hypothesis that high temperatures have imposed a cumulative stress effect on cod in the Aleutian Islands, resulting in climate-induced mortality. The natural mortality timeblock in Model 24.1 provides a better fit to the data. While this does not represent a comprehensive understanding of the effect of heat stress on Aleutian Islands Pacific cod, Model 24.1 is an improvement over Model 24.0.

We agree that future efforts should tie parameters to a specific covariate which would adjust to baseline when it returns to “normal”. This is an aspect of the Model 24.1 that will improve with more time and data. If covariates indicate normal conditions moving forward, then the timeblock on M will be implemented over a limited number of years, and the model will return to the base M for the following years. 

_SSC October 2024_

Consider a prior for M that accounts for maximum ages beyond what has been observed in recent survey data, which likely reflect a truncated age structure.

_Response_

The base value of natural mortality was calculated using all available age data for Aleutian Islands Pacific cod, from NMFS surveys 1991 - 2022. Properly incorporating natural mortality is important because it affects the magnitude of reference points, and populations with higher natural mortality will produce higher estimates of ABC and OFL. Populations whose age structure have been truncated due to climate effects or fishing mortality are subject to natural mortality _plus_ the added mortality due to either fishing or climate or both. This increase in natural mortality should not be attributed to natural mortality; rather it would be more appropriate to utilize climate or fishing mortality parameters to account for the truncated age structure. We support estimation of the base value of M using all available Pacific cod data.

_SSC October 2024_

Check total catches as they were inconsistent between tables 2A.5 and 2A.6.

_Response_

Noted.

_SSC October 2024_

Ensure that the “F ballpark” penalty is turned off in the final estimation phase. It appeared from likelihood profiles that this penalty was still turned on at the end of model runs. This is a convergence aid and should not be affecting model results at the end of convergence.

_Response_

This penalty has been turned off.

# Introduction

Pacific cod (_Gadus macrocephalus_) ranges across the northern Pacific Ocean from Santa Monica Bay, California, northward along the North American coast, Gulf of Alaska, Aleutian Islands, and Bering Sea north to Norton Sound; and southward along the Asian coast from the Gulf of Anadyr to the northern Yellow Sea. Cod occurs at depths from shoreline to 500 m (Ketchen 1961, Bakkala et al. 1984).  The southern limit of the species’ distribution is about 34$^{\circ}$N latitude, and until recently its northern limit was approximately 65$^{\circ}$N latitude (Stevenson and Lauth 2019). However, in recent years Pacific cod has been observed in the Chukchi sea (Cooper et al. 2023). Pacific cod is distributed widely over the eastern Bering Sea (EBS) as well as in the Aleutian Islands (AI) area. 

Separate harvest specifications for Pacific cod have been set for the Bering Sea and Aleutian Islands regions since the 2014 season. Pacific cod were managed in the combined EBS and AI (BSAI) region from 1977 through 2013. 

_Life history_

Pacific cod in the EBS form large spawning aggregations, and are believed to spawn once per year (Sakurai and Hattori 1996, Stark 2007), between February and April (Neidetcher et al. 2014).  Shimada and Kimura (1994) identified major spawning areas between Unalaska and Unimak Islands, and seaward of the Pribilof Islands along the shelf edge.  Neidetcher et al. (2014) identified spawning concentrations north of Unimak Island, in the vicinity of the Pribilof Islands, at the shelf break near Zhemchug Canyon, and adjacent to islands in the central and western Aleutian Islands along the continental shelf.  Pacific cod are known to undertake seasonal migrations as part of an annual migration between summer feeding grounds and winter spawning grounds, the timing and duration of which may be variable (Savin 2008). Travel distances have been observed in excess of 500 nautical miles (nmi), with a large number of travel distances in excess of 100 nmi (Shimada and Kimura 1994). Eggs hatched between 16-28 days after spawning in a laboratory study, with peak hatching on day 21 (Abookire et al. 2007, Hurst et al. 2009).  Settlement in the Gulf of Alaska is reported to occur from July onward (Blackburn and Jackson 1982, Abookire et al. 2007, Laurel et al. 2007), which, given a mean spawning date of mid-March (Neidetcher et al. 2014), and assuming that settlement occurs immediately after transformation, and subtracting about 20 days for the egg stage, implies that the larval life stage might last about 90 days.  In the laboratory study by Hurst et al. (2010), postflexion larvae were all younger than 106 days post-hatching, and juveniles were all older than 131 days post-hatching, so it might be inferred that transformation typically takes place between 106 and 131 days after hatching.  

Several studies have demonstrated an impact of temperature on survival and hatching of eggs and development of embryos and larvae (e.g., Laurel et al. 2008, Hurst et al. 2010, Laurel et al. 2011, Laurel et al. 2012, Bian et al. 2014, Bian et al. 2016). Recruitment of Pacific cod has been shown to be influenced by temperature (e.g., Doyle et al. 2009, Hurst et al. 2012). Pacific cod eggs are demersal (Thomson 1963). After hatching, Pacific cod larvae move quickly to surface waters (Rugen and Matarese 1988, Hurst et al. 2009), and may be transported considerable distances.  Rugen and Materese concluded that larval Pacific cod were transported from waters near the Kenai peninsula and Kodiak Island to locations as far as Unimak Island.  In the Gulf of Alaska, it is thought that movement of larvae has a significant shoreward component (Rugen and Materese, Abookire et al. 2001 and 2007, Laurel et al. 2007), but it is not obvious that this is always the case elsewhere in the species’ range (Hurst et al. 2012). For example, Hurst et al. (2015) found that age-0 Pacific cod in the EBS were most abundant in waters along the Alaska peninsula to depths of 50 m.  

Cold environments allow Pacific cod larvae to bridge gaps in prey availability (i.e., timing and magnitude), but negatively impact survival over longer periods (Laurel et al. 2011). Under warmer conditions, mismatches in prey significantly impacted growth and survival; however, both yolk reserves and compensatory growth mechanisms reduced the severity of mismatches occurring in the first 3 weeks of development (Laurel et al. 2011). Larval retention of Pacific cod during the month of April appears to be important to late spring abundance in the Gulf of Alaska, but it is unknown whether this result holds elsewhere in the species’ range (Doyle et al. 2009).

Juvenile Pacific cod typically settle near the seafloor (Abookire et al. 2007, Laurel et al. 2007). Some studies of Pacific cod in the Gulf of Alaska, and also some studies of Atlantic cod, suggest that young-of-the-year cod are dependent on eelgrass, but this may not be the case elsewhere in the species’ range. Key nursery habitat for age-0 Pacific cod across most of its range typically consists of sheltered embayments. Age-0 Pacific cod have also been observed in the shelf-pelagic zone (Hurst et al. 2012, Parker-Stetter et al. 2013). Habitat use of age-0 Pacific cod in the EBS occurs along a gradient from coastal-demersal (bottom depths < 50 m) to shelf-pelagic (bottom depths 60-80 m), with densities near the coastal waters of the Alaska peninsula much higher than elsewhere (Hurst et al. 2015). Hurst et al. (2012) found evidence of density-dependent habitat selection at the local scale, but no consistent shift in distribution of juvenile Pacific cod in response to interannual climate variability. Habitat use by age-0 Pacific cod in the EBS may be related to temperature and the distribution of large-bodied demersal predators (Hurst et al. 2015). Similarly, the habitat distribution of age-0 Atlantic cod is influenced by predators (Gotceitas et al. 1997).  

Adult Pacific cod in the EBS are strongly associated with the seafloor (Nichol et al. 2007), suggesting that fishing activity has the potential to disturb habitat.  Diel vertical migration has also been observed (Nichol et al. 2013). Patterns varied significantly by location, bottom depth, and time of year, with daily depth changes averaging 8 m. Although little is known about the likelihood of age-dependent natural mortality in adult Pacific cod, it has been suggested that Atlantic cod may exhibit increasing natural mortality with age (Greer-Walker 1970). At least one study (Ueda et al. 2006) indicates that age 2 Pacific cod may congregate more, relative to age 1 Pacific cod, in areas where trawling efficiency is reduced (e.g., areas of rough substrate), causing their selectivity to decrease. Also, Atlantic cod have been shown to dive in response to a passing vessel (Ona and Godø 1990, Handegard and Tjøstheim 2005), which may complicate attempts to estimate catchability (_q_) or selectivity. It is not known whether Pacific cod exhibit a similar response.

_Genetic population structure_

Low-coverage whole-genome sequencing analysis of 429 samples of Pacific cod from known spawning regions during spawning season indicated population structure similar to what was previously known, but with finer resolution and greater power owing to the larger number of markers. Using 1,922,927 polymorphic SNPs, the pattern of population structure mostly resembles isolation-by-distance, in which samples from proximate spawning areas are more genetically similar than samples from more distant areas. Isolation-by-distance was observed from western Gulf of Alaska (Kodiak and the Shumagin Islands) through Unimak Pass and the eastern Aleutian Islands. Previous studies have reported an isolation-by-distance pattern in Pacific cod using microsatellite markers (Cunningham et al. 2009 and Spies 2012) and reduced-representation sequencing (Drinan et al. 2018). Within the isolation-by-distance pattern, there were some distinct breaks in the population structure. The most significant genetic break occurs between western and eastern Gulf of Alaska (GOA) spawning samples, indicated by previous research that highlighted the zona pellucida gene region (Spies et al. 2021). Aleutian Island populations are highly diverged at a several genomic regions that are likely adaptively significant (Spies et al. 2022). These adaptive differences provide further support for the Aleutian Island management unit that was established as distinct from the Bering Sea in 2013.

_Movement_

In 2017, large scale movement was noted into the northern Bering Sea (NBS) by Eastern Bering Sea stocks (Spies et al. 2020). Tagging studies (e.g., Shimada and Kimura 1994) have demonstrated significant migration both within and between the EBS, AI, and Gulf of Alaska (GOA). Pacific cod likely return to their natal origin to spawn during winter months (January - April) but perform feeding migrations during other months. 

Recent research has been performed to understand whether cod move into or out of the Aleutian Islands during summer, outside of spawning season (Schaal et al. 2024, submitted). All of 145 summer-caught cod taken from two locations in the Aleutian Islands assigned back to the Aleutian Islands spawning stock. This result indicates no/minimal movement into our out of the Aleutians during summer when cod are known to perform feeding migrations. Previous tagging studies in groundfish species including Pacific cod and Pacific halibut (_Hippoglossus stenolepis_) also indicate limited movement in the Aleutian Islands (Bryan et al. 2021; Loher 2022). While further work is needed to gain a mechanistic understanding of limited movement in this region, our understanding is that this Pacific cod population is self-sustaining and locally adapted, with little movement into, or out of, the Aleutian Islands.

_Response to climate stress_

Warming in the Aleutian Islands may decrease Pacific cod recruitment. Experimental work indicates that Pacific cod early life stages are sensitive to warming, negatively impacting survival rates from the embryo through juvenile stages (Laurel et al. 2023). Larval hatch rate decreases with temperature above the optimal range for incubation, ~4-6$^{\circ}$C (Laurel and Rogers 2020), and survival from hatch to the post-flexion stage is considerably lower in warming (10$^{\circ}$C) (Slesinger et al. 2024). Juveniles, while more tolerant of warming (16$^{\circ}$C) based on survival and growth rates, have reduced condition and liver lipid content (unpublished), which is a critical energy source that is likely to affect overwintering success (Copeman et al. 2022). Gene expression analyses point to inflammation, altered lipid metabolism, and energetic limitations as mechanisms of mortality and lower condition in larvae and juveniles in response to warming (Spencer et al. in revision, unpublished). Warmer conditions may favor individuals that are more capable of allocating energy and regulating inflammation in warmer conditions, resulting in genetic/demographic shifts. This experimental work was conducted using individuals collected from the Gulf of Alaska spawning aggregation, which is genetically distinct from those that spawn near the Aleutian Islands, and therefore may differ in terms of thermotolerances. 

# Fishery and Management History

## History of the fishery

During the early 1960s, Japanese vessels began harvesting Pacific cod in the Aleutian Islands.  However, these catches were not large, and by the time the Magnuson Fishery Conservation and Management Act went into effect in 1977, foreign catches of Pacific cod in the AI had not exceeded 4,200 t (`r table_nums("histcatch",display="cite")`). Joint venture fisheries began operations in the AI in 1981, and peaked in 1987, with catches totaling over 10,000 t. Foreign fishing for AI Pacific cod ended in 1986, followed by an end to joint venture fishing in 1990 (`r table_nums("catch81_90",display="cite")`). Domestic fishing for AI Pacific cod began in 1981, with a peak catch of over 43,000 t in 1992 (`r table_nums("Fish_catch",display="cite")`).

Historically, Pacific cod were caught throughout the Aleutian Islands.  For the last five years prior to enactment of additional Steller sea lion (_Eumetopias jubatus_) protective regulations in 2011, the proportions of Pacific cod catch in statistical areas 541 (Eastern AI), 542 (Central AI), and 543 (Western AI) averaged 58%, 19%, and 23%, respectively.  For the period 2011-2014, the average distribution has was 84%, 16%, and 0%, respectively.  In 2015, area 543 was reopened to limited fishing for Pacific cod (see “Management History” below).  The average catch distribution for 2019-`r thisyr` (through September 22, `r thisyr`) was `r 100*round(E_cat/tot,2)`% from the eastern Aleutian Islands (NMFS area 541), `r 100*round(C_cat/tot,2)`% from the central Aleutian Islands (NMFS area 542), and `r 100*round(W_cat/tot,2)`% from the western Aleutian Islands (NMFS area 543).

Catches of Pacific cod taken in the AI for the periods 1964-1980, 1981-1990, and 1991-`r thisyr` are shown in `r table_nums("histcatch",display="cite")`, `r table_nums("catch81_90",display="cite")`, and `r table_nums("Fish_catch",display="cite")`, respectively. The catches in `r table_nums("histcatch",display="cite")` and `r table_nums("catch81_90",display="cite")` are broken down by fleet sector (foreign, joint venture, domestic annual processing). The catches in `r table_nums("catch81_90",display="cite")` are also broken down by gear to the extent possible. The catches in `r table_nums("Fish_catch",display="cite")` are broken down by gear.  `r table_nums("catch_AI",display="cite")` breaks down catches from 1994-`r thisyr` by statistical area (area breakdowns not available prior to 1994), both in absolute terms and as proportions of the yearly totals.

## Management History

Appendix 1 from the 2021 Aleutian Islands stock assessment and fishery evaluation lists implemented amendments to the BSAI Groundfish FMP that reference Pacific cod explicitly.  

_History with Respect to the EBS Stock_

Prior to 2014, the AI and EBS Pacific cod stocks were managed jointly, with a single TAC, ABC, and OFL.  Beginning with the 2014 fishery, the two stocks have since been managed separately.

The history of acceptable biological catch (ABC), overfishing level (OFL), and total allowable catch (TAC) levels is summarized and compared with the time series of aggregate (i.e., all-gear, combined area) commercial catches in `r table_nums("catchABC1",display="cite")`.  Note that, prior to 2014, this time series pertains to the combined BSAI region, so the catch time series differs from that shown in `r table_nums("histcatch",display="cite")`, `r table_nums("catch81_90",display="cite")`, and `r table_nums("Fish_catch",display="cite")`, which pertain to the AI only. Total catch has been less than the OFL in every year since 1993. Instances where catch exceeds TAC can typically be attributed to the fact that the catches listed in `r table_nums("catchABC1",display="cite")` are total catches (i.e., Federal plus State), whereas the TAC applies only to the Federal catch. 

```{r, echo=FALSE, results=FALSE}
yrs=names(table(report$Year))
fedcat=vector()
for (i in 1:length(yrs)){
  fedcat[i]=sum(report$Catch..mt.[which(report$Year==yrs[i]&report$State.Federal.Fishery=="F")])
}
Fish_catch_ABCOFL=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/Fish_catch_ABCOFL.csv",header=TRUE)
ratio=fedcat/as.numeric(gsub(",", "", Fish_catch_ABCOFL$TAC, fixed = TRUE))

```

In the `r (thisyr-2014)+1` years that AI Pacific cod have been managed separately from EBS Pacific cod, the ratio of Federal catch to TAC has ranged from `r round(min(ratio[25:34]),2)` to `r round(max(ratio[25:34]),2)`. The catch/TAC ratio in 2024 (complete through September 22) was `r round(ratio[34],2)`, which is the lowest ratio observed since 2015.

ABCs were first specified in 1980. Prior to separate management of the AI and EBS stocks in 2014, TAC averaged about 83% of ABC, and aggregate commercial catch averaged about 92% of TAC (since 1980). Changes in ABC over time are typically attributable to three factors:  1) changes in resource abundance, 2) changes in management strategy, and 3) changes in the stock assessment model.  Because ABC for all years through 2013 were based on the EBS assessment model (with an expansion factor for the AI), readers are referred to the Eastern Bering Sea Pacific cod stock assessment for a history of changes in that model (Thompson et al. 2018).  During the period of separate AI and EBS management, the assessment of the AI stock has been based on a simple, random effects (Tier 5) model.

_History with Respect to the State Fishery_

Beginning with the 2006 fishery, the State of Alaska managed a fishery for AI Pacific cod inside State waters, with a guideline harvest level (GHL) equal to 3% of the BSAI ABC.  Beginning with the 2014 fishery, this practice was modified by establishing two separate GHL fisheries, one for the AI and one for the EBS.  The table below shows the formulas that have been used to set the State GHL for the AI.

\begin{table}[h]
\centering
\begin{tabular}{ lrr }
\hline
Year & Formula \\
\hline
2014	        &    0.03*(EBS ABC + AI ABC)   \\
2015  	  &   0.03*(EBS ABC + AI ABC)   \\
2016     &   0.27*AI ABC\\
2017    &  0.27*AI ABC\\
2018    &  0.27*AI ABC  \\
2019 & 0.31*AI ABC \\
2020 & 0.35*AI ABC or 6,804 t, whichever is less \\
2021 & 0.39*AI ABC or 6,804 t, whichever is less \\
2022 & 0.39*AI ABC or 6,804 t, whichever is less \\
2023 & 0.35*AI ABC or 6,804 t, whichever is less \\
2024 & 0.35*AI ABC or 6,804 t, whichever is less \\
\hline
\end{tabular}
\end{table}

If 90% of the Aleutian Islands GHL is harvested by November 15 of the preceding year, the GHL increases by 4%. However, the GHL cannot exceed 39% or 6,804 t. If the `r thisyr +1` ABC remains at the value that was specified in `r thisyr` (12,431 t), the above formula would result in a GHL of 5,386 t in 2024, which is the maximum allowed (39%) of the ABC. The total caught in the state fishery through the end of 2023 was 4,511 t (`r table_nums("Fish_catch",display="cite")`). During the period in which a State fishery has existed: 1) TAC has been set so that the sum of the TAC and GHL would not exceed the ABC, 2) catch in the Federal fishery has been kept below TAC, and 3) total catch (Federal+State) has been kept below ABC.

_History with Respect to Steller Sea Lion Protection Measures_

The National Marine Fisheries Service (NMFS) listed the western population segment of Steller sea lions as endangered under the ESA in 1997.  Since then, protection measures designed to protect potential Steller sea lion prey from the potential effects of groundfish fishing have been revised several times.  One such revision was implemented in 2011, remaining in effect through 2014.  This revision prohibited the retention of Pacific cod in Area 543.  The latest revision, implemented in 2015, replaced this prohibition with a “harvest limit” for Area 543 determined by subtracting the State GHL from the AI Pacific cod ABC, then multiplying the result by the proportion of the AI Pacific cod biomass in Area 543 (see “Area Allocation of ABC,” under “Harvest Recommendations,” in the “Results” section).

## Current directed fishery

Presently, the Pacific cod stock is exploited by a multiple-gear fishery, including pot, trawl and longline components (`r figure_nums("catch_gear",display="cite")`). Pacific cod in the Aleutian Islands are exploited in the federal and state fisheries. The management quantities in this document pertain to the federal fishery; however, a proportion of the federal quota is allocated to the state fishery. In `r thisyr`, there was no state catch reported (as of September 22) (`r figure_nums("catch_gear",display="cite")`). In `r thisyr`, the federal fishery consisted of `r round(100*FEDpot[28])`% pot gear, `r round(100*FEDLL[28])`% longline gear, and `r round(100*FEDTRAWL[28])`% trawl gear. 

```{r tar, echo=FALSE,message=FALSE}
#if there is state catch, use this sentence above.
#The state fishery consisted of `r round(100*statepot[28])`% pot gear, `r round(100*stateLL[28])`% longline gear, and `r round(100*stateTRAWL[28])`% trawl gear. 

TARGmat=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/TARGMAT.csv")
TARGmat$Total=rowSums(TARGmat[,2:9])
TARGmat$NonTarget=rowSums(TARGmat[,2:9])-TARGmat[,6]
TARGmatr=TARGmat[,c(2,3,4,7,8,5,11,6,10)]




```
An average of `r 100*round(mean(TARGmat$NonTarget)/mean(TARGmat$Total),2)`% of the total Pacific cod catch in the Aleutian Islands has taken place in non-target fisheries from 1993-2023. Over the past 5 years, the non-target fishery catch of Pacific cod has averaged `r formatC(mean(TARGmat$NonTarget[27:31]),format="d",big.mark=",")` t and the targeted fishery has averaged `r formatC(mean(TARGmat$Pacific.Cod[27:31]),format="d",big.mark=",")` t, out of a total average of `r formatC(mean(TARGmat$Total[27:31]),format="d",big.mark=",")` t (`r table_nums("TARGTAB",display="cite")`).

## Discards

The catches shown in `r table_nums("catch81_90",display="cite")` and `r table_nums("Fish_catch",display="cite")` include estimated discards.  Discard amounts and rates of Pacific cod in the AI Pacific cod fisheries are shown for each year 1993-`r thisyr` in `r table_nums("retdis",display="cite")`.  Amendment 49, which mandated increased retention and utilization of Pacific cod, was implemented in 1998. From 1991-1998, discard rates in the Pacific cod fishery averaged 5.6%.  Since 1998, they have averaged about 1.0%.

# Data

The data used in the age structured models include fishery catch and size compositions, survey biomass and standard error, and age compositions from survey data (`r table_nums("data_inline",display="cite")` and `r figure_nums("dataSS3",display="cite")`). Partial catch information for `r thisyr` was available and was extrapolated to estimate the catch for the full year. On average, `r round(100*prop,1)`% of the annual catch occurs by this date, as estimated by catch statistics for the past 5 full years, `r thisyr-5` - `r thisyr-1`. The full year's estimate of catch of Pacific cod in the Aleutian Islands for `r thisyr` was `r formatC(fish_catch[34,6]/prop,format="d",big.mark=",")` t. Overall, catches have decreased for all gear types since 2020 (`r figure_nums("catch_gear",display="cite")`).

The data used in the Tier 5 Model included biomass estimates and associated error for the NMFS Aleutian Island research surveys, 1991-2022 (`r table_nums("srvbio1",display="cite")`).

## Fishery Data   

```{r, message=FALSE,echo=FALSE, results=FALSE}
report=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/Thompson - PCod Total Catch by Jurisdiction_9_22_24.csv",header=TRUE)
catz=matrix(0,3,12)
grz=names(table(report$Gear))
#row1:HAL
#row2=NPT,PTR,TRW
#row3=POT
for(i in 1:3){
 for(j in 1:12){
  catz[1,j]=sum(report$Catch..mt.[which(report$Year>2019&report$Month==j&report$Gear=="HAL")])
  catz[2,j]=sum(report$Catch..mt.[which(report$Year>2019&report$Month==j&report$Gear%in%c("NPT","PTR","TRW"))])
  catz[3,j]=sum(report$Catch..mt.[which(report$Year>2019&report$Month==j&report$Gear=="POT")])
 }}
catz2=rbind(seq(1,12,1),catz)
catz3=t(catz2)

colnames(catz3)=c("Month","Longline","Trawl","Pot")

sumz=colSums(catz3[,2:4])
relz=sumz/sum(sumz)
round(relz,3)#TOtal year

#Winter
sumzW=colSums(catz3[1:4,2:4])
relzW=sumzW/sum(sumzW)
#round(relzW,3)#TOtal year

#nonWinter
sumzNW=colSums(catz3[5:12,2:4])
relzNW=sumzNW/sum(sumzNW)
#round(relzNW,3)#TOtal year

```

There are three predominant gear types in the Aleutian Islands Pacific cod fishery; pot, trawl, and longline, which are implemented at different times of the year (`r figure_nums("catchbygear",display="cite")`). During spawning season (January - April), mature Pacific cod aggregate for spawning at known locations. During these months, over the past 5 years (January 1, 2020 - September 22, 2024), pot and trawl gear were primarily used (`r 100*round(relzW[2],3)`% trawl, `r 100*round(relzW[3],3)`% pot, `r 100*round(relzW[1],3)`% longline). After spawning, Pacific cod typically disperse for feeding (although some do not); during May through December, cod were primarily caught with trawl gear, followed by longline and pot gear (`r 100*round(relzNW[2],3)`% trawl, `r 100*round(relzNW[1],3)`% longline, `r 100*round(relzNW[3],3)`% pot). While the spawning season is approximately half the time of non-spawning (4 vs. 8 months), the majority, `r 100*round(sum(catz3[1:4,2:4])/sum(catz3[,2:4]),3)`%, of the annual catch (during the time period January 1, 2020 - September 22, 2024) took place during spawning season. Catches have exceeded TAC harvest recommendations in five of the nine years since 2013, but have never exceeded the OFL (`r table_nums("catchABC1",display="cite")`). 

CPUE aggregated over gear types for the number and weight of fish show similar trends, indicating that there has been no large shifts in the weight of individual fish (Spies et al. 2023). Recent declines in CPUE may be attributed to the timing of the fishery relative to spawning season or other factors such as hyperaggregation during spawning in the trawl fishery (Rose and Kulka 1999). Standardized surveys are needed to understand whether declines in fishery CPUE represent declines in Aleutian Islands Pacific cod stock size. Recent declines in CPUE may also be due to less effort in targeted Pacific cod fishing. The amount of targeted Pacific cod fishing has decreased since 2018, but the catch in the atka mackerel and rockfish fisheries has remained the same or increased (`r figure_nums("TARGET",display="cite")`, `r table_nums("TARGTAB",display="cite")`).

Length data taken by fishery observers is used in the model (`r table_nums("data_inline",display="cite")`). The number of hauls from which lengths are taken are somewhat proportional to the magnitude of fishing, and the number of hauls which recorded cod lengths was highest in 2001 (`r table_nums("nlens",display="cite")`). 

## Survey Data

The National Marine Fisheries Service (NMFS) conducts biennial daytime summer trawl surveys in the Aleutian Islands. Survey biomass is estimated by extrapolating the weight from individual trawls with the measured path of the trawl area to the total area surveyed. The net used in the Aleutian Islands survey is a high-rise poly-Noreastern 4 seam bottom trawl (27.2 m headrope, 36.8 m footrope, Nichol et al. 2007). Survey biomass estimates and standard error for Pacific cod are available for the survey years 1991, 1994, 1997, 2000, 2002, 2004, 2006, 2010, 2012, 2014, 2016, 2018, and 2022 (`r table_nums("srvbio",display="cite")`). A survey is currently underway during 2024, and a survey biomass estimate will be available for the November assessment. Aleutian Islands surveys prior to 1991 were not used in the model because they were not standardized to current survey methodology; therefore, data from the 1980, 1983, and 1987 surveys were excluded. Survey data includes NMFS areas 541, 542, and 543. The Aleutian Islands bottom trawl survey does include NMFS areas 518 and 519, but these were not included in data for this model because they are not considered part of the Aleutian Islands management area.

Survey age data is available for each survey, 1991-2022. The number of cod aged from the survey has ranged between 500 and 1,200 and the number of hauls ranges from 76-173 (`r table_nums("nages",display="cite")`). Length composition data from the surveys is also used in the model (`r figure_nums("dataSS3",display="cite")`). All survey data are specified in model as from July, the midpoint of NMFS surveys.

The time series of NMFS bottom trawl survey biomass is shown for Areas 541-543 (Eastern, Central, and Western AI, respectively), together with their respective coefficients of variation, in `r table_nums("srvbio1",display="cite")`. These estimates pertain to the Aleutian management area, and so are smaller than the estimates pertaining to the Aleutian survey area that were reported in BSAI Pacific cod stock assessments prior to 2013. Over the long term, the trawl survey biomass data indicate a decline, and the 2022 estimate of biomass is the lowest in the time series. The total biomass estimate for Pacific cod in the Aleutian Islands declined from over 180,000 t in 1991 to 51,539 t in the current year. Recent declines took place in the eastern Aleutians (>50% decline) and in the central Aleutians (32% decline) from the last survey in 2018 to the current survey in 2022. The western Aleutian Islands stock of Pacific cod increased from 11,425 t to 13,661 t (20% increase) between 2018 and 2022 (`r figure_nums("REMA",display="cite")`, `r figure_nums("Bio_area",display="cite")` and `r table_nums("srvbio1",display="cite")`). 

The most recent longline survey estimate was also the lowest in the time series, but the longline survey data was not incorporated into the Tier 5 model or the age stuctured models (`r table_nums("srvbio",display="cite")`). The longline survey was designed to target sablefish, and how well it documents the abundance of Pacific cod is uncertain. 

# Analytic Approach [Tier 5 model]

_Model 13.4_

Model 13.4 is the Tier 5 random effects model recommended by the Survey Averaging Working Group, which has been accepted by the Plan Team and SSC since the 2013 assessment for the purpose of setting AI Pacific cod harvest specifications. The Tier 5 random effects model is programmed using the ADMB software package (Fournier et al. 2012) as a “random walk” state-space model.  The only parameter in Model 13.4 is the log of the log-scale process error standard deviation. When used to implement the Tier 5 harvest control rules, the Tier 5 models also require an estimate of the natural mortality rate. The Tier 5 random effects model assumes that the observation error variances are equal to the sampling variances estimated from the haul-by-haul survey data. The log-scale process errors and observations are both assumed to be normally distributed. 

Recent estimates of natural mortality indicate that estimates have ranged from 0.20 to 0.96 for Pacific cod (`r table_nums("M_estimates",display="cite")`). A natural mortality estimate of 0.34 been used in the most recent Aleutian Islands Pacific cod assessment, as well as the 2022 and prior BSAI cod assessments (Thompson et al. 2018). This value was based on Equation 7 of Jensen (1996) and an age at maturity of 4.9 years (Stark 2007). The value of 0.34 adopted in 2007 replaced the value of 0.37 that had been used in all BSAI Pacific cod stock assessments from 1993 through 2006. In response to a request from the SSC, the 2008 assessment included a discussion of alternative values and a justification for the value chosen (Thompson et al. 2008). Using the variance for the age at 50% maturity published by Stark (0.0663), the 95% confidence interval for M extends from about 0.30 to 0.38. The value of 0.34 for natural mortality was used for the `r thisyr` Tier 5 Model 13.4, as in previous years. 

Under Tier 5, $F_{OFL}$ is set equal to the natural mortality, $F_{OFL}$ = M, and the fishing mortality rate to achieve the acceptable biological catch is 75% of M, $F_{ABC}$ $\leq$ 0.75×M.

_Model 24.2_

A new base value for natural mortality, 0.417, was introduced as a base model for age structured models. Therefore, M = 0.417 is presented as an alternative calculation for the Tier 5 reference points in Model 24.2, and is preferred over Model 13.4 because of the updated M.

# Analytic Approach [Age structured models]

Stock Synthesis Version 3.30.21 was used to run the the age structured models in this assessment. Stock Synthesis requires that prior distributions and initial values be associated with all internally estimated time-invariant parameters. For age structured models presented in this assessment, all parameters were fit freely without informative priors. 
\
\
The following are features of the age structured models that are consistent with the 2023 models.

* Single sex model, 1:1 male female ratio.
* Survey age and length data were input as conditional age-at-length.
* Recruitment estimated as a mean with lognormally distributed deviations.
* Maturity-at-age was estimated externally using observer data, then input into the model.
* Single-fleet fishery that combines trawl, longline, and pot fishery data, weighted by quarter, gear, and NMFS area, from 1991 - current year (through September 22).
* All parameters were constant over time except for natural mortality after 2015 in Model 24.1.
* Survey and fishery selectivity were modeled as logistic and constant over time. 
* Trawl survey catchability was incorporated analytically.

_Data weighting and model tuning_

Survey length and age input sample sizes generated by bootstrapping the number of hauls from which length and age data were taken using the methodology of Hulson et al. (2023). Fishery length composition input sample sizes were based on the number of hauls, and scaled to the mean survey input sample size (so that the mean fishery length composition input sample size was the same as the survey mean input sample size). This did not result in a change in likelihood, but is preferred over a constant sample size approach to weighting compositional data because it considers the number of hauls in each year and therefore the varying informational content. This approach is consistent among the Aleutian Islands, EBS, and GOA Pacific cod assessment models.

Age and length composition data from survey and length composition data from the fishery were weighted using the methodology of Francis (2011) in an iterative process (`r table_nums("nages",display="cite")`, `r table_nums("nlens",display="cite")`). Input sample sizes for age comps by length bin were equal to the actual number of fish aged in those categories (whole numbers). The marginal compositions were weighted using the bootstrapping method of Hulson et al. (2023).

The parameter for variance in recruitment, sigmaR, was tuned prior to Francis weighting, using a feature in SS3 that provides an estimate based on model output. The value used in Models 24.0, 24.1 and 24.1a was 0.8.


## Parameters Estimated Outside the Assessment Model

_Maturity_ 

The maturity-at-length is modeled by the relationship:

$$Maturity_{length}=\frac{1}{1+e^{-(A+Blength)}},$$ where A and B are parameters in the relationship.

A study based on a collection of 129 female fish in February, 2003, from the Unimak Pass area, NMFS area 509, found that 50% of female fish become mature at approximately 4.88 years ($L_{50\%}$) and 58.0 cm, $A$=-4.7143, $B$=0.9654  (i.e. Tables 2 and 4 in Stark 2007). This maturity ogive is used in the Bering Sea Pacific cod assessment but was not used in this assessment, because the fish in the sample were not from the Aleutian Islands.

Observers routinely collect maturity at length from Pacific cod. An alternative maturity curve was developed based on observer records of maturity from the Aleutian Islands. This parameterization is advantageous because it is based on more records that were taken from Aleutian Islands cod, and is incorporated into age-structured models presented here. Maturity was updated in 2024, resulting in the addition of 16 new visual maturity samples, which did not result in a large change to the maturity curve parameters. There were 1,347 records of visual maturity data from the Aleutian Islands (see table below) during the months January – March since 2008. These were used to estimate a maturity ogive by length using the R package _sizeMat_, which estimates the length of fish at gonad maturity. Maturity was considered a binomial response varable and variables were fitted to the logistic function above for maturity, and the length at which 50% of cod are mature is $L_{50\%}=-A/B$ (`r table_nums("maturity",display="cite")`). Maturity was converted from maturity-at-length to maturity-at-age for `r table_nums("maturity",display="cite")`. This method was accepted by the Plan Team (September 2022) and SSC (October 2022). Maturity data used in the model are based on length, rather than age. Reanalysis in 2024 shifted the age at 50% maturity from 54.9 to 55.5 cm (Confidence intervals = 53.7 - 57.3), A = -8.589, B = 0.155, but this did not result in a change to the maturity-at-age.

``` {r newmat, echo=FALSE, message=FALSE}
#OBS=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/data/Maturity2024.csv")

#library(data.table)
#OBS$MONTH<-as.numeric(OBS$MONTH)##
#	OBS$LENGTH<-as.numeric(OBS$LENGTH)
#	OBS$YEAR<-as.numeric(OBS$YEAR)
#	OBS_AI<-OBS[OBS$NMFS_AREA%in%c(540:543),]

#  dataAI<-OBS_AI[which(!is.na(OBS_AI$MATURITY_DESCRIPTION)),]#They are all females#
#	dataAI4<-dataAI[which(dataAI$YEAR>2007&dataAI$MONTH<=4),]
	
	
#my_ogive_fq = gonad_mature(dataAI4,varNames = c("LENGTH", "MATURITY_DESCRIPTION"), inmName = "Immature",matName = c("Pre Spawn","Spawning","Spent","Developing"), method = "bayes", niter = 999)

#plot(my_ogive_fq, xlab = "Total length (cm.)", ylab = "Proportion mature", col = c("blue", "red"))

#A=median(my_ogive_fq$A_boot)
#B=median(my_ogive_fq$B_boot)
A=-8.58948
B=0.1549561
#M50%=A/B
#slope=-B

```

_Ageing error_

After 2007, there was a shift in our understanding of the first two checks deposited at early ages in Pacific cod. Prior to 2007 they were thought to be true annuli, but subsequently determined not to be. Therefore, geing bias was not incorporated in the model, as all ages used were aged subsequent to 2007, after which time ageing methodology has been consistent and considered non-biased. Ageing error was applied to ages 2-13+. The standard deviation at the first age was 0.57 and 1.16 at the maximum age.

``` {r details, echo=FALSE, message=FALSE}
#* Age at which the estimated pattern begins (just linear below this age), this is the start age. #_The pattern begins at age 3, linear prior to age 3. _
#* Bias at start age (as additive offset from unbiased age). _None._
#* Bias at maximum (as additive offset from unbiased age). _None._
#* Power function coefficient for interpolating between those 2 values (value of 0.0 produces #linear interpolation in the bias). _Zero._
#* Standard deviation at start age. _0.57_
#* Standard deviation at max age. _1.16_
#* Power function coefficient for interpolating between those 2 values. _Zero._

```

There were 10,134 records of aged and lengthed Pacific cod taken from NMFS research surveys between 1991 and 2022 (1991, 1994, 1997, 2000, 2002, 2004, 2006, 2010, 2012, 2014, 2016, 2018, 2022). The maximum age observed was 13.

_Natural mortality_

Natural mortality is notoriously difficult to estimate inside assessment models, because it is typically correlated with other parameters. The SSC and Plan Team have suggested implementing a new methodology for estimating natural mortality externally to the model in 2024, similar to the methodology used in the eastern Bering Sea Pacific cod model since 2023. Implementing a fixed value for natural mortality allows for better estimation of parameters that may be constrained by M. For EBS cod, M was calculated as 0.3866. 

For the base model, natural mortality was fixed at a value calculated using the _Then_lm_ method (http://barefootecologist.com.au/shiny_m.html), which is the same methodology used to calculate natural mortality, M, for the eastern Bering Sea Pacific cod assessment model. 

Research suggests that longevity-based predictions of natural mortality rate are more precise than growth-based estimates (Hamel & Cope, 2022; Then et al., 2015). Longevity can be recorded either as the maximum aged specimen, or the average of the five maximum ages (Sullivan et al., 2022). 

The value of M was recalculated for Aleutian Islands Pacific cod to account for different growth patterns and different maximum ages. For example, the maximum age observed in the eastern Bering Sea was 14, while the maximum age observed in the Aleutian Islands was 13. The age data used for this analysis was the same as in the assessment, from all survey years including the most recent survey in 2022 (Table 1). Based on these values, natural mortality M for Aleutian Islands Pacific cod was 0.417. The difference is likely due to faster growth and lower maximum age observed in Aleutian Islands Pacific cod. 

A natural mortality estimate of 0.34 been used in the most recent Aleutian Islands Pacific cod assessment, as well as the 2022 and prior BSAI cod assessments (Thompson et al. 2018). This value was based on Equation 7 of Jensen (1996) and an age at maturity of 4.9 years (Stark 2007). The value of 0.34 adopted in 2007 replaced the value of 0.37 that had been used in all BSAI Pacific cod stock assessments from 1993 through 2006. In response to a request from the SSC, the 2008 assessment included a discussion of alternative values and a justification for the value chosen (Thompson et al. 2008). Using the variance for the age at 50% maturity published by Stark (0.0663), the 95% confidence interval for M extends from about 0.30 to 0.38. 

## Parameters Estimated Inside the Assessment Model

_Growth_

The Richards growth curve was used to fit growth within the models, but sensitivity to the von Bertalanffy growth curve is shown in Model 24.1a, which uses the von Bertalanffy growth curve (`r table_nums("TableCompare",display="cite")`). The Richards growth curve adds an additional parameter to the logistic growth curve to account for non-symmetrical growth at early ages and maximum ages. Although the Von Bertalanffy growth curve was used in previous iterations of the age structured model, we found that the Richards growth curve improved the fit to the data. The Richards growth curve is also used in the Eastern Bering Sea and the Gulf of Alaska Pacific cod stock assessments. The Richards growth curve improves the fit to the conditional age at length, reducing the CV on age 1 and younger (`r figure_nums("CAAL",display="cite")`). Overall, the conditional age at length provides a good fit to the data.  

None of these models incorporate time-varying growth because it is not justifed by size at age data. Pacific cod growth in the Aleutian Islands has not changed significantly throughout the timeseries; therefore. Timeblocks on growth were explored in 2023, but resulted in minimal changes to parameters. 

_Length at age_

Pacific cod do not exhibit sexually dimorphic growth; males and females grow at the same rate. Therefore, the model did not distinguish between males and females. Growth is rapid at younger ages (`r figure_nums("agerainbow",display="cite")`) and was estimated within the model using the Richards growth curve as described above. Incorporating all length age and length bins were important for fitting growth.

There was no observed change in growth over time. Therefore, only constant growth was used in the age structured models. We explored fixing the length of fish (L1) at early ages (1.0 years) at external estimates, but opted for model estimation of length at L1.

_Selectivity_

Selectivity for the fishery and the survey were fit (separately) using monotonically increasing asymptotic logistic curve (`r figure_nums("Selectivity",display="cite")`). We considered dome-shaped and selectivity in our September report but a better fit to the data was achieved with logistic selectivity curves. 

The fishery is modeled as a single-fleet fishery that combines trawl, longline, and pot fishery data. While these geartypes do catch different distributions of fish, past model explorations for the EBS and Aleutian Islands stock assessment models showed that there is insufficient data to merit three separate fishery selectivity curves.

_Natural mortality_

Model 24.1 used a time block which estimated natural mortality from 2016 - 2024, but it was fixed at M = 0.417 from the beginning of the time series through 2015 (`r table_nums("TableCompare",display="cite")`). The timeblock on natural mortality estimated a single value, rather than a base value with deviations. Estimating a single parameter is advantageous because it reduces the number of parameters estimated and reduces uncertainty.

Model 24.1 (and 24.1a) accounted for changing environmental conditions due to the shift in temperature through the 2016-2024 time block on natural mortality. The year 2016 was selected because this it is two years after the beginning of the documented thermal shift (Xiao and Ren 2022) (`r figure_nums("Regime",display="cite")`). While this thermal shift has not been documented as an ecological regime shift, it is significant and should be considered as a potential factor for changes in abundance or distribution. The time block did not start earlier because it incorporated a ~two year lag for effects of higher temperatures to be observed, the effect of cumulative stress that increased temperatures can incur (e.g. Barbeaux et al. 2018, Laurel and Rogers 2020). Pacific cod are known to respond poorly to temperatures that exceed their preferred thermal range; therefore, increased natural mortality due to the thermal shift may be the optimal model configuration. The lag also corresponded to the time required for cod to grow to a size/age at first survey selectivity. The assumption that increased natural mortality due to higher temperatures is supported by evidence in the laboratory and in situ, and includes all life stages (Laurel et al. 2008, Barbeaux et al. 2020). Accordingly, the time block on natural mortality in Model 24.1 indicated higher natural mortality which was freely esetimated and not on a bound (`r round(M24_1$Natural_Mortality[34,13],2)`). As of August 2024, heatwave conditions in the Aleutian Islands appear to be less extreme than in past years during all months with the exception of August in the western Aleutians, but periods of moderate heatwave conditions have occurred (`r figure_nums("Heatwave_2024",display="cite")`). The timeblock on natural mortality in Models 24.1 and 24.0A improves the fit to survey index of abundance, particularly later years in the survey (`r figure_nums("Index",display="cite")`). The survey index declined from 2018 - 2022; therefore, the spawning stock size relative to unfished is lowest for the models that include the natural mortality timeblock (`r figure_nums("FracUnfished",display="cite")`). 

_Catchability_

Experiments have been performed that measure whole-gear efficiency (the proportion of fish passing between the otter doors of a bottom trawl that are subsequently captured), and these studies can inform catchability. Somerton (2004) used an 83-112 Eastern Trawl trawl net in the eastern Bering Sea and found no evidence that Pacific cod were herded into the net (Somerton et al. 2004). Another study estimated 47.3% of cod in the water column to be available to the trawl used on the eastern Bering Sea trawl survey and 91.6% are available to the trawl used on the Gulf of Alaska and Aleutian Islands surveys (Nichol et al. 2007). This study was based on results showing that 95% of cod were found within 10 m of the seafloor, based on 286 archival tagged cod off Kodiak Island in the Gulf of Alaska and off Unimak Pass in the eastern Bering Sea, Alaska  (Nichol et al. 2007). More recently Rand et al. (2022) found no evidence for difference in mean size of Pacific cod caught by the survey and the fishery in the eastern Bering Sea. However, the Aleutian Islands presents a more complex environment, with rocky and steep habitat that is not consistently trawlable (Logerwell et al. 2005). It is plausible that this habitat could lead to survey catchability <1 in the Aleutian Islands.

_Recruitment_

Recruitment was estimated for all years in the model, excluding the final 3 years, as there was little data to inform it (1991 - 2021). Recruitment after 2021 through the forecast period 2039 used the mean 1991 - 2021 age-0 recruitment. During the 13 years prior to 1991, initial age comps are estimated, and during that period the model uses the MLE of initial/virgin recruitment. 

_Other parameters_

The total likelihood and the number of parameters estimated for each model is shown in `r table_nums("TableCompare",display="cite")`. In this table, we calculated a rough estimate of AIC, using AIC = 2 x the number of parameters + 2 x total log likelihood. There is one additional parameter in Model 24.1 than 24.0 due to the estimation of natural mortality during 2016 - 2024. There is also one fewer parameter in Model 24.1a due to the von Bertalanffy growth curve which uses 3 parameters rather than 4 in the Richards growth curve. Parameters such as recruitment deviations are effectively less than one parameter; however, growth parameters and natural mortality are full parameters. Given that the AIC is a rough estimate, we provide information on the total likelihood and other estimates when discussing model fit to the data.

## Description of Alternative Models

Model 24.1 is the preferred age structured model, and Models 24.1a and 24.0 are "sensitivity models", presented to show sensitivity to the Richards growth curve and the timeblock on natural mortality, respectively. 

# Age Structured Model Results (Tier 3)

Model 24.1 includes the addition of a time block on natural mortality with a break in 2015, corresponding with the thermal shift in 2013/2014 with a 2-year lag (Xiao and Ren 2022). This results in the best fit to the data and the lowest AIC, and is a significant improvement over Model 24.0, which does not include this timeblock parameter (`r table_nums("TableCompare",display="cite")`). In Pacific cod, a sustained increase in foraging demand has been documented during marine heat waves (Barbeaux et al. 2020). Winter heatwaves compound this effect by increasing demand for prey resources during months when demand typically declines. The availability of sufficient prey has been shown to be limiting through bottom-up availability in quality and quantity of lower-trophic prey, and competition with other higher level predators seeking similar prey resources (Barbeaux et al. 2020). Length-weight residuals, also referred to as fish condition, are a measure of fish weight vs. body length, and are an indicator of somatic growth. Poor fish condition is expected to be correlated with bioenergetic estimates for high metabolic demand, and Pacific cod length-weight residuals have been below average in all regions of the Aleutian Islands since the 2012 survey, for over a decade (`r figure_nums("FishCondition",display="cite")`). This period of low fish condition corresponds with the recent heatwave in the Aleutian Islands. Natural mortality was estimated higher in the timeblock as expected due to thermal stress `r round(as.numeric(M24_1$Natural_Mortality[37,23]),3)`, over M = `r round(as.numeric(M24_1$Natural_Mortality[25,23]),3)`. The incorporation of the natural mortality time block in Model 24.1 improved the fit to the survey index and the overall likelihood compared with Model 24.0, which did not incorporate the natural mortality timeblock (`r table_nums("TableCompare", display="cite")`).

Model 24.1 also represents an improvement over Model 24.1a, with the incorporation of the Richards growth curve. The Richards growth curve improved the fit to the data, specifically the survey, recruitment, and length composition likelihood components (`r table_nums("TableCompare",display="cite")`) over Model 24.1a that used the von Bertalanffy growth curve. The Richards growth curve allows for an inflection point between younger (age 2) and older cod (age 4+) that is not available in the von Bertalanffy. This provides a different slope, which is straighter, matches cod growth data better, and provides a smooth transition from age 1, in which growth is linear to age 2. It also results in a smaller maximum size and tighter confidence intervals (`r figure_nums("Growth",display="cite")`). We note that the Richards growth curve improved the likelihood (Model 24.1a vs. 24.1) from 547 to 534, with the addition of a single parameter. which is a reasonable improvement. In the September report (https://meetings.npfmc.org/Meeting/Details/3056), there were seven features, removed sequentially in a sensitivity analysis. These features included the choice of maximum length, natural mortality timeblock, CV on young growth, shape of the survey selectivity curve, maximum age, initial fishing mortality, fixed vs. estimated natural mortality, and the shape of the growth curve. As each model progressed, better model options were replaced with the less desirable alternative. In the final model, the Richards growth curve was replaced by the von Bertalanffy growth curve. A drastic decline in model fit appeared in the model with the the von Bertalanffy growth curve vs. the Richards growth curve. However, this decline was partially due to the method by which the sensitivity analysis was performed. Because seven desirable features were removed from the final model, it showed a considerably poorer fit to the data, as the cumulative effect of multiple less desirable features. In the current assessment, bridging is performed with no more than one feature removed from Model 24.1.

AIC and total likelihood were lowest for Model 24.1 (`r table_nums("TableCompare", display="cite")`), indicating a better fit to the data despite the addition of an estimated timeblock parameter and the additional Richards growth parameter. Improvements in Model 24.1 over Model 24.0 are notable in the survey and length composition likelihood components, as well as the fit to the survey index and AIC (`r  figure_nums("survey_index", display="cite")`, `r  table_nums("TableCompare", display="cite")`). Model 24.1 provided better fits to survey, length composition, age composition, and recruitment likelihoods than Model 24.1a. Given the improvements in the model specficiation relative to fits to the data and improved assumptions, Model 24.1 is the recommended candidate for management. The timeseries of total biomass (ages 1 and above) for Models 24.1, 24.1a, and 24.0 is provided in `r table_nums("TotBio_est",display="cite")`, the timeseries of estimated spawning biomass with 95% confidence intervals in `r table_nums("SSB_est",display="cite")`, and age-0 recruitment (1000s of fish) with confidence intervals in `r table_nums("rec_est",display="cite")`. Parameters estimated by each model are provided in `r table_nums("parsM24_1", display="cite")` and `r table_nums("parsM24_1a",display="cite")`.

Based on Model 24.1, total biomass declined from approximately `r formatC(M24_1$timeseries$Bio_all[4],big.mark=",",format="d")` t in 1992 to a timeseries low of `r formatC(min(M24_1$timeseries$Bio_all),big.mark=",",format="d")` t in `r seq(1989,2039,1)[which(M24_1$timeseries$Bio_all==min(M24_1$timeseries$Bio_all))]` (`r table_nums("TotBio_est",display="cite")`). The trawl survey estimate of biomass was `r formatC(M24_1$cpue$Obs[34],big.mark=",",format="d")` t in `r thisyr-1`, similar to but lower than the 2022 estimate, which was the previous lowest estimate in the timeseries. Female spawning biomass has followed a similar overall declining trend as total biomass in all models (`r table_nums("SSB_est",display="cite")`, with the peak spawning biomass occurring by the early 1990s for all models. For Model 24.1, spawning biomass reached its lowest point of `r formatC(min(M24_1$timeseries$SpawnBio),big.mark=",",format="d")` t in `r seq(1989,2039,1)[which(M24_1$timeseries$SpawnBio==min(M24_1$timeseries$SpawnBio))]`.

In Models 24.1 and 24.1a, an increase occurs in recruitment deviations during 2015 - 2020, corresponding with the natural mortality timeblock (`r figure_nums("Recruitment",display="cite")`). In contrast, Model 24.0 accounted for the decrease in survey indices of abundance by reduced recruitment during 2010 - 2020. Presumably, the increase in recruitment in Model 24.1 was a response to the higher natural mortality. There is little evidence that Pacific cod recruitment increases during heatwave conditions, although recent research indicates that warm conditions increase mortality in overwintering age-0 cod, which may be consistent with model results indicating that mortality occurs during later stages of development (Spencer et al. Submitted). In the Gulf of Alaska, abundance declined with increasing ocean temperature during egg and larval phases, and the age-0 cod were larger (Abookire et al. 2022). The Gulf of Alaska is a broad, shallow shelf, quite different from the Aleutian Islands archipelago, a partially submerged mountain range in which cod habitat consists of narrow and steep terrain (Logerwell et al. 2005). Some life history stages of Pacific cod in the Aleutian Islands may escape warm temperatures by moving to deeper water, unlike cod in the Gulf of Alaska. Survey length compositions, which include age 1 and 2 cod, indicate that the proportion of age 1 cod decreased in 2002 and 2004 from previous years, and then increased to a stable proportion over the remainder of years through 2022 (`r figure_nums("comp_lenfit",display="cite")`). This is consistent with a decrease in recruitment during the early 2000s, and a subsequent increase (`r figure_nums("Recruitment",display="cite")`). Aggregated over time, the fit to fishery and survey length frequencies is not affected by the growth curve or natural mortality time block (`r figure_nums("LengthFit",display="cite")`).

Finally, the stock appears to have declined to close to $B_{20}\%$, despite relatively light fishing (`r figure_nums("phase24_1",display="cite")`, `r figure_nums("phase24_0",display="cite")`). Models 24.0 and 24.1 both indicate a period of high recruitment during the 1990s, followed by lower recruitment from 2001 - 2010 (`r figure_nums("plotREC",display="cite")`). This lower recruitment during 2001 - 2010 may have resulted in declining stock sizes. For Model 24.1, the mean age-0 recruitment during 2001 - 2010 was 42,836 (x1,000) and the mean used for forecasting, was 61,854 (x1,000). Model 24.0 used a somewhat lower estimate for forecasting, 59,243 (x1,000). A second reason for the low stock sizes despite low fishing mortality is climate-related mortality. While Model 24.1 provides a link between the modeled population dynamics and climate, future research may develop more complex understanding of the relationship between climate stress and stock size.

# Evaluation of Models and Associated Uncertainty

The fit to the length composition data using one-step-ahead (OSA) residuals generally showed no large-scale pattern of over or under-fitting and appear to support the assumption of randomly distributed error (`r figure_nums("OSA24_1",display="cite")`, `r figure_nums("OSA24_1a",display="cite")`, `r figure_nums("OSA24_0",display="cite")`). The OSA residuals are similar among the three models (24.1, 24.1a, and 24.0), with some residual patterns that are small in magnitude. For the fishery length compositions, patterns of positive residuals are present that indicate slightly heavy tails, characterized by positive residuals around 50 cm and 100 cm. There are few outliers which do not show a strongly discernable pattern. Survey residuals generally fit, although a pattern of small positive residuals are present around 50 cm and at approximately 20 cm representing 1 year old fish. There are some outlier residuals at age 1 length frequencies, but these outliers are present in fewer than half the years. The standardized deviation of normalized residuals (SDNR) is approximately 1 for all models. In general SDNR much greater than 1 is not consistent with a good fit to the data while SDNR less than 1 indicates that the data was fitted better than expected, and is not a cause for concern (Francis 2011). The model fit to observed length frequency distributions for survey and fishery show a similar fit for all models (`r figure_nums("OSA24_1",display="cite")`, `r figure_nums("OSA24_1a",display="cite")`, `r figure_nums("OSA24_0",display="cite")`).

Observed and expected length frequencies for the fishery (`r figure_nums("yrclass",display="cite")`) and the survey (`r figure_nums("yrclassSrv",display="cite")`), and observed age frequencies (`r figure_nums("yrclassAge",display="cite")`) provide insight into the data and model fits. The fishery and survey length compositions show that predominant sizes were larger in 2024 than in 2022, indicating an ageing population, while indication of incoming age 1 and 2 year classes. The 2022 age frequencies indicate more 3 year olds than in 2018, and have not yet been read for 2024 (`r figure_nums("yrclassAge",display="cite")`). 

# Sensitivity to Model Specification

Sensitivity model runs were performed in the September Pacific cod assessment. Models provided information on sensitivity to the following features: the choice of maximum length, natural mortality timeblock, CV on young growth, shape of the survey selectivity curve, maximum age, initial fishing mortality, fixed vs. estimated natural mortality, and the shape of the growth curve.

# Convergence Status and Criteria

Convergence was determined by successful inversion of the Hessian matrix and a maximum gradient component of less than 1e-4 (this value was 9.3e-05 for Model 24.1, 1.2e-05 for Model 24.0, and 1.1e-05 for Model 24.1a). A jitter analysis revealed that the proposed Model 24.1 and sensitivity models 24.1a and 24.0 are insensitive to perturbations of parameter start values on the order of 10% (`r figure_nums("jitter",display="cite")`, `r figure_nums("jitter24_1a",display="cite")`, `r figure_nums("jitter24_0",display="cite")`). All of the jittered model runs went to the maximum likelihood estimate and all parameters were estimated within their pre-specified bounds. All model runs implemented within the jitter test had total negative log likelihood values equal to or less than the base model. Therefore, based on the jitter test, there was no evidence to reject the hypothesis that the base model successfully converged.

# Likelihood Profiles on Key Parameters
 
Likelihood profiles have been performed on several key parameters: catchability (`r figure_nums("lnQ",display="cite")`) and initial recruitment (R0, `r figure_nums("R0",display="cite")`). The likelihood profile for initial recruitment (R0) used values between 10.4 and 12.0 in increments of 0.2 (`r figure_nums("R0",display="cite")`). The profile suggests that the global MLE for R0 is approximately 11.4, and the range of R0 with likelihood values <1.96, indicating an acceptable range, is 11.0 - 11.5. Index data indicates that initial recruitment is somewhat higher, while length data indicates R0 is somewhat lower. The global estimate is between these two and similar to the model estimate of recruitment, `r round(as.numeric(TableCompare[21,2]),1)`. Maximum likelihood estimates of R0 were between 11.0 and 11.5 for all three models (`r table_nums("parsM24_1",display="cite")` `r table_nums("parsM24_1a",display="cite")`).

A likelihood profile on survey catchability (q) was performed, and results are presented as the logarithm of catchability followed by the actual catchability (not logarithm) in parentheses. Log(q) was profiled from -0.8 to 0.8 (q = 0.45 to 2.22) in increments of 0.1, and showed conflict among data sources (`r figure_nums("lnQ",display="cite")`). Index data suggested that survey catchability was -0.5 (q = 0.6), and recruitment suggested even lower. In contrast, F ballpark indicated higher ln(q), at 0.8 (q = 2.22). The age and length data were both close to zero, and the MLE was approximately -0.2 (q = 0.82). We estimated catchability in all models analytically, as the exploitable biomass estimated by the model divided by the survey estimate of biomass. This results in the same estimate as estimating catchability with a uniform prior, but provides a more parsimonious alternative. The calculated catchability for Model 24.1 was `r round(M24_1$cpue$Calc_Q[1],3)` (`r table_nums("TableCompare",display="cite")`). 

# Parameters used in forecasting

Due to the time-block on natural mortality in Model 24.1, forecasts were performed using the base value of natural mortality used in Model 24.1 from 1991-2015, M = 0.417. We justify this choice as follows. As populations experience the stress of climate change, natural mortality will increase in response, as in Model 24.1 (Szuwalski et al. 2023). Incorporating increased natural mortality in harvest rates implies a ‘double-threat’, because adapting the harvest control rule to changing environmental conditions would result in both lower biomass targets and higher harvest rates (Szuwalski et al. 2023). Populations that have declined as a result of climate stress (or fishing mortality) are subject to natural mortality _plus_ the added mortality due to either fishing or climate or both. Therefore, projections are based on the time period in which natural mortality is fixed at the base value of 0.417, and do not incorporate an increase due to climate change. Further, the heatblock M has a big impact on advice and would confer large changes if future versions of the models change the size of the estimate.

While other parameters, such as recruitment, can be affected by climate change, we used mean recruitment over all years it was estimated (1991 - 2021). In addition to simplifying assumptions in projections, this choice did not change results notably. For all other parameters (except M) the entire time series was used in projections. Recruitment from 2022 - 2024 was not estimated, as there was little data to inform it.

# Retrospective analysis 

A ten-year retrospective analysis was conducted by sequential removal of all data annually beginning with 2024 and ending with 2014, for Models 24.0 and 24.1 (`r figure_nums("retro_M2",display="cite")`). By age ten, a cohort of fish have reached their asymptotic length, are considered 100% mature and fully selected by the survey. We note that the timeblock challenges the assumptions of the retrospective analysis because model parameterization is not constant over time. Therefore, a retrospective analysis may be inappropriate for Model 24.1; however, we have included it here to illustrate the improvement of Model 24.1 over Model 24.0. For Model 24.1, the mean terminal spawning biomass estimate from each of these retrospective models was within the 95% confidence interval of the current base model for the first five years, but deviated as the number of years in the 2016-2024 timeblock was reduced. Retrospective plots for the two age structured models indicate an improved pattern for Model 24.1 over Model 24.0 (`r figure_nums("retro_M2",display="cite")`). 

Model 24.0 produced Mohn's rho value of `r Rho_24_0`, while Model 24.1 produced a smaller retrospective pattern, `r Rho_24_1` (`r figure_nums("retro_M2",display="cite")`). Hurtado-Ferro (2015) provides some guidance on the range of acceptable values for Mohn's rho. For a flatfish-like species with M = 0.2, the lower and upper bounds were given as -0.15 and 0.2. For a sardine-like species with M = 0.34, the lower and upper bounds were given to be -0.22 and 0.3. If Mohn's rho were entirely dependent on M (likely an oversimplification), then an equation for the lower and upper limits could be developed from these guidelines as follows: 
$Rho_{lowerbound} = -0.08-0.35*M$ and
$Rho_{upperbound} = 0.10+0.50*M$. Using these guidelines, and noting that the base value of natural mortality for Models 24.0 and 24.1 was 0.417, lower and upper bounds would be -0.23 and 0.31. Given these guidelines, the Mohn's rho is within acceptable bounds for Model 24.1 but is outside of acceptable values for Model 24.0 (`r figure_nums("retro_M2",display="cite")`).

Without incorporating a timeblock on natural mortality, the model cannot fit recent declines in total biomass documented by the Aleutian Islands survey, as illustrated by the poor retrospective pattern in Model 24.0 (e.g. Mohn's rho >0.4, `r figure_nums("retro_M2",display="cite")`). 

# Historical retrospectives (between models)

This was not performed, no previous Tier 3 assessment models have been used for this stock.

# Tier 5 Model Results

The Tier 5 ABCs and OFLs for `r thisyr+1` and `r thisyr+2` for Models 13.4 and 24.2 incorporate random effect estimates of Aleutian Island Pacific cod biomass from 1991 - 2024 (`r table_nums("srvbio1",display="cite")`). The 2024 survey biomass estimate was `r Srv_thisF` t, `r 100*round(1-(Srv_this/Srv_last),2)`% lower than the 2022 estimate. The random effects model estimated an exploitable biomass of `r formatC(round(REMA$Biomass[34]),format="d",big.mark=",")` t, which resulted in OFLs (`r T5OFL_thisF` t) and ABCs (`r T5ABC_thisF` t) for `r thisyr+1` and `r thisyr+2`. Model 24.2 calculated Tier 5 ABCs and OFLs with the previously described estimate of natural mortality (M = 0.417), which resulted in higher OFLs (`r T5OFL_thisF24_2` t) and ABCs (`r T5ABC_thisF24_2` t) for `r thisyr+1` and `r thisyr+2` (`r table_nums("Tier5_comparison",display="cite")`). We recommend the use of age structured Model 24.1 for reference point setting and harvest quantities. 

# Projections and Harvest Recommendations

A Kobe plot demonstrating the stock status uncertainty over $SSB/SSB_{MSY}$ and $F/F_{MSY}$ indicates that the stock has shifted from >$SSB_{MSY}$ in 1991 to a lower stock status <$SSB_{MSY}$. Uncertainty was generated by drawning 1,000 points from a joint multivariate normal posterior for SSB and F. The results (for Model 24.1) indicate a 96.4% probability that the stock status is between ~$SSB_{12\%}$ and $SSB_{35\%}$, and that the fishing mortality rate is below $F_{40}$ (`r figure_nums("Kobe24_1",display="cite")`). Stock status is similar for Model 24.1a (`r figure_nums("Kobe24_1a",display="cite")`), but higher for Model 24.0, which indicates a 52% chance the stock status is greater than $SSB_{35\%}$ (`r figure_nums("Kobe24_0",display="cite")`). Phase plane plots indicate a similar stock status for current and future years (`r figure_nums("phase24_1",display="cite")`); stock status is above $B_{20\%}$ and is predicted to remain so through 2026. Model 24.0 indicates a higher stock status than Model 24.1 ((`r figure_nums("phase24_0",display="cite")`)).

This is consistent with other analyses: `r table_nums("TableCompare",display="cite")` indicates that the proportion of unfished biomass is `r 100*round(as.numeric(TableCompare[18,2]),2)`%, and fishing mortality has been low relative to management quantities (`r table_nums("catchABC1",display="cite")`, `r figure_nums("catch_gear",display="cite")`).

# Amendment 56 Reference Points

Amendment 56 to the BSAI Groundfish Fishery Management Plan (FMP) defines the “overfishing level” (OFL), the fishing mortality rate used to set OFL ($F_{OFL}$), the maximum permissible ABC, and the fishing mortality rate used to set the maximum permissible ABC.  The fishing mortality rate used to set ABC ($F_{ABC}$) may be less than this maximum permissible level, but not greater.  

Under Tier 5, $F_{OFL}$ is set equal to the natural mortality, $F_{OFL}$ = M, and the fishing mortality rate to achieve the acceptable biological catch is 75% of M, $F_{ABC}$ $\leq$ 0.75×M.

`r table_nums("Tier5_comparison",display="cite")` includes estimates of harvest specifications, OFL, maximum permissible ABC, and the associated fishing mortality rates for `r thisyr+1` and `r thisyr+2` for the Tier 5 reference points for Tier 5 Models 13.4 and 24.2. Note that the 95% confidence intervals for the Tier 5 estimates of biomass are shown in `r table_nums("T5Bio",display="cite")`. Model 24.2 is the preferred Tier 5 model for 2024 because it is consistent with Plan Team and SSC recommendations for an externally estimated estimate of natural mortality.

# Specification of OFL and Maximum Permissible ABC

The Tier 3 models and projections were conducted using the Stock Synthesis program. Models 24.1, 24.1a, and 24.0 indicate that the Aleutian Islands Pacific cod stock is not subjected to overfishing and not overfished. If fishing continues at its average rate for the past 5 years, female spawning biomass is predicted to be above $B_{35\%}$ by 2029 (`r table_nums("Scenarios_M24_1",display="cite")`, `r table_nums("Scenarios_M24_1a",display="cite")`, `r table_nums("Scenarios_M24_0",display="cite")`). Projections of catch for Aleutian Islands Pacific cod 2025 through 2037 based on the seven harvest scenarios are shown as figures for Models 24.1, 24.1a, and 24.0 
Projections for Aleutian Islands Pacific cod 2025 through 2037 are presented for Models 24.1, 24.1a and 24.0 for catches (`r figure_nums("Catch_scenarios24_1",display="cite")`, `r figure_nums("Catch_scenarios24_1a",display="cite")`, `r figure_nums("Catch_scenarios24_0",display="cite")`) and stock status (`r figure_nums("Projection_scenarios24_1",display="cite")`, `r figure_nums("Projection_scenarios24_1a",display="cite")`, `r figure_nums("Projection_scenarios24_0",display="cite")`).

For stocks in Tiers 4-6, no determination can be made of overfished status or approaching an overfished condition as information is insufficient to estimate the MSY stock level. Under Model 13.4, ABC and OFL for `r thisyr+1` and `r thisyr+2` are `r T5ABC_thisF` t and `r T5OFL_thisF` t. Under Model 24.2, ABC and OFL for `r thisyr+1` and `r thisyr+2` are `r T5ABC_thisF24_2` t and `r T5OFL_thisF24_2` t. 

Under Model 24.1, ABCs for `r thisyr+1` and `r thisyr+2` are `r formatC(Two_year$C_ABC[1],big.mark=",",format="d")` t and `r formatC(Two_year$C_ABC[2],big.mark=",",format="d")` t, and OFLs are `r formatC(Two_year$C_OFL[1],big.mark=",",format="d")` t and `r formatC(Two_year$C_OFL[2],big.mark=",",format="d")` t (`r table_nums("M24_0_1_comparison",display="cite")`). 

## Standard Harvest Scenarios and Projection Methodology

A standard set of projections is required for each stock managed under Tiers 1, 2, or 3 of Amendment 56.  This set of projections encompasses seven harvest scenarios designed to satisfy the requirements of Amendment 56, the National Environmental Protection Act, and the Magnuson-Stevens Fishery Conservation and Management Act (MSFCMA).

For all models, the year-end catch for 2024 was estimated to be 4,676 t, as described above. In the event that catch is likely to be less than the recommended ABC in either of the first two projection years, Scenario 2 must be conducted, using the best estimates of catch in those two years (otherwise, Scenario 2 can be omitted if the author’s
recommended ABCs for the next two years are equal to the maximum permissible ABCs). 

Five of the seven standard scenarios support the alternative harvest strategies analyzed in the Alaska Groundfish Harvest Specifications Final Environmental Impact Statement. These five scenarios, which are designed to provide a range of harvest alternatives that are likely to bracket the final TAC for `r thisyr+1`, are as follows (max $F_{ABC}$ refers to the maximum permissible value of $F_{ABC}$ under Amendment 56):

* Scenario 1:  In all future years, F is set equal to max $F_{ABC}$.  (Rationale:  Historically, TAC has been constrained by ABC, so this scenario provides a likely upper limit on future TACs.)
* Scenario 2:  In all future years, F is set equal to a constant fraction of max $F_{ABC}$, where this fraction is equal to the ratio of the $F_{ABC}$  value for the assessment two years ago recommended in the assessment to the max $F_{ABC}$ for the current year.  (Rationale:  When $F_{ABC}$ is set at a value below max $F_{ABC}$, it is often set at the value recommended in the stock assessment.) 

Aleutian Islands Pacific cod was not managed with a Tier 3 model two years ago; therefore, we used the value 0.4 as a multiplier on $F_{ABC}$ for all future years, which yields F = 0.201 for all future years for Model 24.1, F = 0.288 for Model 24.1a, and F = 0.333 for Model 24.0. This is much than the 2031 - 2039 $F_{ABC}$ = 0.804 in Scenario 1 for Model 24.1, $F_{ABC}$ = 1.156 for Model 24.1a, and $F_{ABC}$ = 0.813 for Model 24.0.

* Scenario 3: In all future years, F is set equal to the average of the five most recent years. (Rationale: For some stocks, TAC can be well below ABC, and recent average F may provide a better indicator of $F_{TAC}$ than $F_{ABC}$.) 
* Scenario 4: In all future years, the upper bound on $F_{ABC}$ is set at $F_{75\%}$. (Rationale: This scenario provides a likely lower bound on $F_{ABC}$ that still allows future harvest rates to be adjusted downward when stocks fall below reference levels.)
* Scenario 5:  In all future years, F is set equal to zero.  (Rationale:  In extreme cases, TAC may be set at a level close to zero.)

Two other scenarios are needed to satisfy the MSFCMA’s requirement to determine whether a stock is currently in an overfished condition or is approaching an overfished condition.  These two scenarios are as follow (for Tier 3 stocks, the MSY level is defined as $B_{35\%}$):

* Scenario 6:  In all future years, F is set equal to $F_{OFL}$.  (Rationale:  This scenario determines whether a stock is overfished.  If the stock is 1) above its MSY level in `r thisyr` or 2) above 1/2 of its MSY level in `r thisyr` and expected to be above its MSY level in `r thisyr+10` under this scenario, then the stock is not overfished.)
* Scenario 7: In `r thisyr+1` , F is set equal to max $F_{ABC}$, and in all subsequent years, F is set equal to $F_{OFL}$. (Rationale: This scenario determines whether a stock is approaching an overfished condition. If the stock is 1) above its MSY level in `r thisyr+2` or 2) above 1/2 of its MSY level in `r thisyr+2` and expected to be above its 
MSY level in `r thisyr+12` under this scenario, then the stock is not approaching an overfished condition.)  

# $F_{limit}$

``` {r FLIMIT, echo=FALSE,message=FALSE }

#install.packages("devtools")
#devtools::install_github("BenWilliams-NOAA/groundfishr")
#library(groundfishr)

#2023 FOFL was 0.544,2024 OFL =12732
#last_f=0.544


#oneone <- data.frame(age = 1:13, naa = 2*as.numeric(M24_1$natage[71,14:26])/1000, waa = as.numeric(M24_1$wtatage[168,8:20]), saa = as.numeric(M24_1$ageselex[216,8:20]))

#Wants OFL in kt, waa in kg, naa in millions

#best_f(m = 0.417, last_ofl = 12.732, data = oneone, type = 1,last_f=0.544)

#M24_1$ageselex[219,8:20] #survey 

#Steve B recommends just running it through SS.
#


```

The fishing mortality rate (based on Model 24.1) that would have produced a catch for last year equal to last year’s OFL of 12,732 t is 0.578. This was produced with the full model, with higher natural mortality during the final nine years, and therefore is higher than the reference table may indicate. This value will be reported in the Aleutian Islands Pacific cod SARA file as the F_LIMIT and included in the species information system (SIS) output. 

# Risk Table

## Assessment Considerations

To date the SSC has categorized this stock in Tier 5 where a reliable biomass estimate and estimate of M is available. The biomass estimate has come from a smoother of survey estimates (using REMA).  This guides the Council in setting the ABC and OFL. As noted above, we prefer the SSC adopt the age-structured assessment model (24.1) for advice using Tier 3 because of the available estimates of recruitment, biomass, and SPR rates that this model provides. A benefit of a Tier 3 categorization is that the Council can be informed on the status determination of the stock (under Tier 5 such information is unavailable). Therefore, given our assessment refinements incorporating ecosystem and biological sampling, our assessment concerns are level 1, no concern.

## Population Dynamics Considerations

The long-term (1991-2024) trawl survey biomass trend is downward and the 2024 index is the lowest of the time series (`r figure_nums("REMA",display="cite")`). Research indicates that Aleutian Islands cod have do not move out of the region and it is unlikely that the decline is due to emigration. 

We consider the current stock size atypical, considering the survey biomass time series. Therefore, population dynamics considerations were rated as level 2. 


## Environmental/Ecosystem Considerations

Environment:   The average bottom temperature from the Aleutian Islands bottom trawl survey (165$^{\circ}$W – 172$^{\circ}$E, 30-500 m) was close to the 20-year mean (1991–2012) for all subareas but still above the long term mean. This is in contrast with the four survey years prior, which were generally warmer than average for bottom temperatures. The bottom temperature means are similar across all four regions (Howard and Laman, 2024) and values close to the long term mean is considered a positive indicator. Satellite sea surface temperatures show a step increase in 2014 with higher temperatures both in summer and winter (Xiao and Ren 2023). Sea surface temperatures were above the mean through winter across all subregions. Over the eastern Aleutian Islands, there were few days of marine heatwave (MHW) status relative to the mean over the last decade, which was also the case in 2021 and 2022. At times during late summer, over 75%  of the western Aleutians were in MHW status. While there were also warm anomalies and MHWs over 25% of the central and eastern Aleutians in summer, these were not sufficient to register in the spatial mean (Lemagie and Callahan, 2024). 

Pacific cod are typically found between 3.5–5.7$^{\circ}$C (range 2.8 to 6.9$^{\circ}$C) and an average depth of 164 m (range 22 – 435) in the Aleutian Islands based on data from the bottom trawl survey.  In general, higher ambient temperatures incur bioenergetic costs for ectothermic fish. However, Holsman and Aydin (2015) found  adult Pacific cod consumption in the Aleutian Islands increases up to 4$^{\circ}$C and decreases past 5$^{\circ}$C. While the continued decrease in average bottom temperature in 2024 is favorable, the fact that the sea surface temperatures continue to be above the long term average in winter and summer is considered to have a negative effect. Pacific cod are particularly sensitive to the impacts of increased temperatures due to a combination of their energetic demands, diet, as seen in the Gulf of Alaska during the 2014–2016 heatwave (Barbeaux et al. 2020). In addition, Pacific cod eggs adhere to the seafloor and with demersal egg hatching success above 20% restricted to temperatures between 3-6$^{\circ}$C and they spawn between January and May. Warmer winter temperatures may impact their overwinter survival.

Prey: Atka mackerel and pollock are primarily planktivorous fish, feeding largely on copepods and euphausiids followed by pelagic amphipods and pelagic gelatinous filter feeders. While Atka mackerel and pollock >250 mm showed a fish condition below the long term average, which decreases their nutritional value and quality as prey (Howard and Laman 2024, Ortiz 2024), pollock 100-250 mm had a high fish condition 2 std. dev. above the long term mean. Pacific cod stomachs collected in the bottom trawl survey in the western and central Aleutians (areas 543, 542) have shown decreases of Atka mackerel in its diet, previously one of their primary prey items, over the past few years (Ortiz and Zador 2023), potentially reflecting scarcer and lower quality prey available for Pacific cod. The reverse has happened in area 541, where Atka mackerel was not generally a common prey in earlier years, but has now increased to over 20% of Pacific cod diet by biomass since 2014, replacing sculpins. The improved quality of small pollock as prey may have prevented a further decrease in the fish condition of Pacific cod, as it stayed similar to that observed in 2022 which had improved compared to the previous three survey years. Still the fish condition of Pacific cod has remained below average since 2012. 

As a generalist, Pacific cod is able to compensate for the lower availability of any one type of prey, having the ability to easily switch between fish and benthic crustacean prey. The increase in prey quantity and quality may be offset by the dominance of rockfish (POP and Northern Rockfish) within the pelagic foragers guild, previously comprising a larger proportion of pollock and Atka mackerel in the early 1990s.  This year, piscivorous and planktivorous seabirds had mostly below average reproductive success, around Buldir signaling foraging conditions that did not allow them to rear chicks successfully, which may in turn signal suboptimal foraging conditions for groundfish as well. In contrast,  planktivorous and piscivorous seabirds at Aiktak (eastern Aleutians) had above average reproductive success, signaling a potential gradient in foraging conditions with better conditions towards the east (Rojek et al. 2024).
Competitors and predators: Among the fish apex predators, piscivores and invertivores (e.g. arrowtooth flounder, Pacific halibut) continue declining except for sculpins and sablefish (Ortiz, 2024). As of 2023, Steller sea lions were declining in the western Aleutians offset by increases in the east (Sweeney and Gelatt, 2024), and harbor seals are also declining (London et al. 2018). Piscivorous and planktivorous seabirds at Aiktak are reproducing successfully and puffins are feeding on pollock, but their abundance trend is unknown as is that of common murres (Rojek et al 2024).

Despite the cooler temperatures, and good quality small pollock, the fish condition of cod stayed similar to that in 2022 as opposed to continue to improve. This together with continued warm winter and potentially suboptimal foraging conditions in the western Aleutians suggests there still remain some adverse signals relevant to the stock, but the pattern is not consistent across all areas. 

Environmental/ ecosystem considerations were rated as level 2 (multiple indicators showing consistent adverse signals a) across the same trophic level, and/or b) up or down trophic levels (i.e., predators and prey).


## Fishery Performance Considerations

Trends in CPUE can be examined for evidence of population trends, although other factors can affect CPUE besides population dynamics. Trends in catch are available from fishery data through `r thisyr`, and consistently indicate a downward trend (`r figure_nums("catch_gear",display="cite")`). 

The fishery reports that lack of catcher-vessel (CV) trawl effort in the Aleutian Islands is not due to lack of interest. The Aleutian Islands fishery often gets pre-empted by the Bering Sea fishery given the later timing of aggregation in the Aleutians and the lack of an Aleutian set-aside of the CV sector appointment. For the trawl CVs, the early part of the A season catch rates in the Bering Sea are often better. By March, CPUE for trawl CVs is generally better in the Aleutian Islands. Unfortunately the CV trawl cod fishery in the Aleutian Islands is often closed by then.

In some years (e.g. 2020) the BSAI CV trawl fleet took a large portion or all of their A season quota in the Bering Sea before the Aleutian Islands cod aggregate (for spawning). The Adak processor was closed in 2020 through 2023, and is unlikely to open for the 2024 A season, so no local processing plant is available. This results in fewer smaller pot and hook-and-line vessels unless a floating processor or tender is available to assist. 

Fishery performance considerations were rated as level 1.

## Risk Summary

The ratings of the four categories are summarized below:

\begin{table}[ht]
\centering
\begin{tabular}{ p{3cm}p{3cm}p{3cm}p{3cm} }
\hline
 Assessment \newline consideration & Population \newline dynamics & Environmental \newline ecosystem & Fishery \newline performance  \\
\hline
 Level 1: Normal  & Level 2: Increased concerns & Level 2: Increased concerns & Level 1: Normal   \\ 
\hline
\end{tabular}
\end{table}

Based on this risk table, we recommend a reduction in ABC for the Tier 5 Model 24.2. We recommend the Tier 3 Model 24.1 ABC be used in place of the Model 24.2 ABC. 


_Area Allocation of ABC_

```{r WAI,echo=FALSE,message=FALSE}

reALL <- read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/REMA/AI_REMA.csv")
reWAI <- read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/REMA/AI_REMA_WAI.csv")


WAIprop=reWAI$Biomass[34]/reALL$Biomass[34]

```

As noted in the “Management History” subsection of the “Fishery” section, the current Steller sea lion protection measures require an estimate of the proportion of the Aleutian Islands Pacific cod stock residing in Area 543, which will be used to set the harvest limit in 543 after subtraction of the State GHL from the overall AI ABC.  Since 2018, the Area 543 proportion has been calculated in Tier 5 model based on the most recent estimate of biomass in Area 543 relative to the estimate from the total area. Using Aleutian Islands trawl survey data from 1991 - 2024, this proportion is `r 100*round(WAIprop,2)`%. This represents an increase, as 15.7% was used between 2018 and 2022. The proportion allocated to the western Aleutian Islands region 543 would remain `r 100*round(WAIprop,2)`% after the State GHL is subtracted from the overall AI ABC whether a Tier 5 or Tier 3b harvest strategy is adopted.

_Status Determination_

Under the MSFCMA, the Secretary of Commerce is required to report on the status of each U.S. fishery with respect to overfishing.  This report involves the answers to three questions:  1) Is the stock being subjected to overfishing?  2) Is the stock currently overfished?  3) Is the stock approaching an overfished condition? The official AI catch estimate for the most recent complete year (`r thisyr-1`) is `r formatC(fish_catch[33,6],big.mark=",",format="d")` t.  This is less than the `r thisyr-2` AI OFL of `r formatC(Fish_catch_ABCOFL[31,5],format="d",big.mark=",")` t and also the AI ABC of `r formatC(Fish_catch_ABCOFL[32,3],format="d",big.mark=",")`.  Therefore, the AI Pacific cod stock is not being subjected to overfishing. Because this stock is managed under Tier 5, no determination can be made with respect to overfished status. If the status changes to Tier 3, it would not be considered subjected to overfishing.

# Ecosystem Considerations

## Ecosystem effects on the stock

A primary ecosystem phenomenon affecting the Pacific cod stock seems to be the occurrence of periodic “regime shifts,” in which central tendencies of key variables in the physical environment change on a scale spanning several years to a few decades (Zador, 2011).  One well-documented example of such a regime shift occurred in 1977, and shifts occurring in 1989 and 1999 have also been suggested (e.g., Hare and Mantua 2000).  Because the data time series in the models presented in this assessment do not begin until 1991, the 1977 regime shift should not be a factor in any of the quantities presented here, although it may indeed have had an impact on the stock.

The prey and predators of Pacific cod have been described or reviewed by Albers and Anderson (1985), Livingston (1989, 1991), Lang et al. (2003), Westrheim (1996), and Yang (2004). The composition of Pacific cod prey varies to some extent by time and area.  In terms of percent occurrence, some of the most common items in the diet of Pacific cod in the BSAI and GOA have been polychaetes, amphipods, and crangonid shrimp.  In terms of numbers of individual organisms consumed, some of the most common dietary items have been euphausids, miscellaneous fishes, and amphipods.  In terms of weight of organisms consumed, common dietary items include walleye pollock, fishery offal, yellowfin sole, and crustaceans. Small Pacific cod feed mostly on invertebrates, while large Pacific cod are mainly piscivorous. Predators of Pacific cod include Pacific cod, halibut, salmon shark, northern fur seals, Steller sea lions, harbor porpoises, various whale species, and tufted puffin.  Major trends in the most important prey or predator species could be expected to affect the dynamics of Pacific cod to some extent.

## Fishery Effects on the Ecosystem

Potentially, fisheries for Pacific cod can have effects on other species in the ecosystem through a variety of mechanisms, for example by relieving predation pressure on shared prey species (i.e., species which serve as prey for both Pacific cod and other species), by reducing prey availability for predators of Pacific cod, by altering habitat, by imposing bycatch mortality, or by “ghost fishing” caused by lost fishing gear.

_Incidental Catch Taken in the Pacific Cod Fisheries_

Incidental catches taken in the Pacific cod target fisheries, expressed as proportions of total incidental EBS catches (i.e., across all targets) for the respective species, are summarized in several tables.  For the purpose of generating these tables, Pacific cod targets were those identified as such in the AKFIN database (https://akfin.psmfc.org/).  Catches for `r thisyr` in each of these tables are incomplete, through the end of October `r thisyr`.  The Pacific cod fishery using trawl gear (`r table_nums("bycatchFMP",display="cite")`) and fixed gear `r table_nums("bycatchFMP_HAL",display="cite")` take a small proportion of the incidental catch of FMP species (1991-`r thisyr`). FMP species are more commonly caught in the Pacific cod trawl fishery than with longline gear (`r table_nums("bycatchFMP",display="cite")`, `r table_nums("bycatchFMP_HAL",display="cite")`). In the Pacific cod trawl fishery, flatfish are most commonly caught FMP species, followed by Atka mackerel and occasionally sharks and skates. During some years from 1991-`r thisyr`, the proportional catch of octopus and longnose skate was high in the Pacific cod trawl (`r table_nums("OtherSP",display="cite")`) and longline (`r table_nums("OtherSP_HAL",display="cite")`) fisheries, although incidental catch of squid and members of the former “other species” complex taken by trawl gear was lower.  Similarly, the Pacific cod fishery accounts for a large proportion of several crab species bycatch (`r table_nums("prohibited",display="cite")`). Discard mortality of halibut taken in the Pacific cod fishery from 1991-`r thisyr`, aggregated across gear types, has declined during this time period. The proportion of incidental catch of non-target species groups taken from 2003-`r thisyr`, excluding bird species, aggregated across gear types `r table_nums("OtherSP2",display="cite")` varies from very little to almost all of the bycatch in a given year.  

_Steller Sea Lions_

Pacific cod is one of the four most important prey items of Steller sea lions in terms of frequency of occurrence averaged over years, seasons, and sites, and is especially important in winter in the GOA and BSAI (Pitcher 1981, Calkins 1998, Sinclair and Zeppelin 2002).  The size ranges of Pacific cod harvested by the fisheries and consumed by Steller sea lions overlap, and the fishery operates to some extent in the same geographic areas used by Steller sea lion as foraging grounds (Livingston (ed.), 2002). A study conducted in 2002-2005 using pot fishing gear demonstrated that the local concentration of cod in the Unimak Pass area is very dynamic, so that fishery removals did not create a measurable decline in fish abundance (Conners and Munro 2008).  A preliminary tagging study in 2003–2004 showed some cod remaining in the vicinity of the release area in the southeast Bering Sea for several months, while other fish moved distances of 150 km or more north-northwest along the shelf, some within two weeks (Rand et al. 2015).

_Seabirds_

In the BSAI and GOA, the northern fulmar (_Fulmarus glacialis_) comprises the majority of seabird bycatch, primarily in the longline fisheries, including the fixed gear fishery for Pacific cod (Livingston (ed.) 2002).  Shearwater (_Puffinus_ spp.) distribution overlaps with the Pacific cod longline fishery in the Bering Sea, and with trawl fisheries in general in both the Bering Sea and GOA. Black-footed albatross (_Phoebastria nigripes_) is taken in much greater numbers in the GOA longline fisheries than the Bering Sea longline fisheries, but is not taken in the trawl fisheries. The distribution of Laysan albatross (_Phoebastria immutabilis_) appears to overlap with the longline fisheries in the central and western Aleutians. The distribution of short-tailed albatross (_Phoebastria albatrus_) also overlaps with the Pacific cod longline fishery along the Aleutian chain, although the majority of the bycatch has taken place along the northern portion of the Bering Sea shelf edge; in contrast, only two have been recorded in the GOA. Some success has been obtained in devising measures to mitigate fishery-seabird interactions.  For example, on vessels larger than 60 ft. LOA, paired streamer lines of specified performance and material standards have been found to significantly reduce seabird incidental take. Typically bycatch of bird species in the Pacific cod trawl and longline fisheries is low, although in some years a large proportion of certain species were taken in the Pacific cod fisheries (`r table_nums("bird",display="cite")`.

# Data Gaps and Research Priorities

Longer-term research needs include improved understanding of: 1) the spatial dynamics, trophic and other interspecific relationships, and the relationship between climate and recruitment, 2) ecology of species that interact with Pacific cod, including estimation of interaction strengths, biomass, carrying capacity, and resilience, and 3) the physiological response of cod to thermal stress using gene expression studies in an experimental setting. 

An age-based histological maturity curve would be ideal and is a research priority for the model.

# Acknowledgements

We thank the RACE Longline and AFSC survey personnel for data collection and the AFSC age-and-growth department processing the samples used in this assessment. Thank you to Jim Ianelli for reviewing the document and SSMA/MESA for providing helpful comments.

\pagebreak

# Literature Cited

Abookire, A. A., J. F. Piatt, B. L. Norcross.  2001.  Juvenile groundfish habitat in Kachemak Bay, Alaska, during late summer.  Alaska Fishery Research Bulletin 8(1).

Abookire, A. A., J. T. Duffy-Anderson, C. M. Jump.  2007.  Habitat associations and diet of young-of-the-year Pacific cod (_Gadus macrocephalus_) near Kodiak, Alaska.  Marine Biology 150:713-726.

Abookire, A., Litzow, M., Malick, M., Laurel, B. 2022. Post-settlement abundance, condition, and survival in a climate-stressed population of Pacific cod. CJFAS, Can. J. Fish. Aquat. Sci. 79: 958–968 (2022) dx.doi.org/10.1139/cjfas-2021-0224.

Albers, W. D., and P. J. Anderson.  1985.  Diet of Pacific cod, _Gadus macrocephalus_, and predation on the northern pink shrimp, _Pandalus borealis_, in Pavlof Bay, Alaska.  Fish. Bull., U.S. 83:601-610.

Bakkala, R. G., S. Westrheim, S. Mishima, C. Zhang, E. Brown.  1984.  Distribution of Pacific cod (_Gadus macrocephalus_) in the North Pacific Ocean.  International North Pacific Fisheries Commission Bulletin 42:111-115.

Barbeaux, S., K. Aydin, B. Fissel, K. Holsman, B. Laurel, W. Palsson, K. Shotwell, Q. Yang, and S. Zador.  2018.  Assessment of the Pacific cod stock in the Gulf of Alaska.  In Plan Team for Groundfish Fisheries of the Gulf of Alaska (compiler), Stock assessment and fishery evaluation report for the groundfish resources of the Gulf of Alaska, chapter 2, p. 1-160.  North Pacific Fishery Management Council, 605 W. 4th Avenue Suite 306, Anchorage, AK 99501.

Barbeaux, S.J., Holsman, K. and Zador, S., 2020. Marine heatwave stress test of ecosystem-based fisheries management in the Gulf of Alaska Pacific Cod Fishery. Frontiers in Marine Science, 7, p.703.

Bian, X., Zhang, X., Sakurai, Y., Jin, X., Gao, T., Wan, R. and Yamamoto, J., 2014. Temperature‐mediated survival, development and hatching variation of Pacific cod _Gadus macrocephalus_ eggs. Journal of Fish Biology, 84(1), pp.85-105.

Bian, X., Zhang, X., Sakurai, Y., Jin, X., Wan, R., Gao, T. and Yamamoto, J., 2016. Interactive effects of incubation temperature and salinity on the early life stages of pacific cod _Gadus macrocephalus_. Deep Sea Research Part II: Topical Studies in Oceanography, 124, pp.117-128.

Blackburn, J.E. and Jackson, P.B., 1982. Seasonal composition and abundance of juvenile and adult marine finfish and crab species in the nearshore zone of Kodiak Island's eastside during April 1978 through March 1979. Outer Continental Shelf Environmental Assessment Program, Final Reports of Principal Investigators, 54, pp.377-570.

Bryan, D. R., McDermott, S. F., Nielsen, J. K., Fraser, D., & Rand, K. M. 2021. Seasonal migratory patterns of Pacific cod (_Gadus macrocephalus_) in the Aleutian Islands. Animal Biotelemetry, 9(24), 1–18. https://doi.org/10.1186/s40317-021-00250-2.

Calkins, D. G.  1998.  Prey of Steller sea lions in the Bering Sea.  Biosphere Conservation 1:33-44.

Canino, M.F., Spies, I.B., Cunningham, K.M., Hauser, L. and Grant, W.S., 2010. Multiple ice‐age refugia in Pacific cod, _Gadus macrocephalus_. Molecular Ecology, 19(19), pp.4339-4351.

Conners, M. E., and P. Munro.  2008.  Effects of commercial fishing on local abundance of Pacific cod (_Gadus macrocephalus_) in the Bering Sea.  Fishery Bulletin 106:281-292.

Cooper, D.W., Cieciel, K., Copeman, L., Emelin, P.O., Logerwell, E., Ferm, N., Lamb, J., Levine, R., Axler, K., Woodgate, R.A. and Britt, L., 2023. Pacific cod or tikhookeanskaya treska (_Gadus macrocephalus_) in the Chukchi Sea during recent warm years: Distribution by life stage and age-0 diet and condition. Deep Sea Research Part II: Topical Studies in Oceanography, 208, p.105241.

Copeman, L. A., Stowell, M. A., Salant, C. D., Ottmar, M. L., Spencer, M. L., Iseri, P. J., et al. 2022. The role of temperature on overwinter survival, condition metrics and lipid loss in juvenile polar cod (Boreogadus saida): A laboratory experiment. Deep Sea Res. Part 2 Top. Stud. Oceanogr. 205, 105177.

Cunningham, K.M., Canino, M.F., Spies, I.B. and Hauser, L., 2009. Genetic isolation by distance and localized fjord population structure in Pacific cod (_Gadus macrocephalus_): limited effective dispersal in the northeastern Pacific Ocean. Canadian Journal of Fisheries and Aquatic Sciences, 66(1), pp.153-166.

Doyle, M. J., S. J. Picquelle, K. L. Mier, M. C. Spillane, N. A. Bond.  2009.  Larval fish abundance and physical forcing in the Gulf of Alaska, 1981-2003.  Progress in Oceanography 80:163-187.

Drinan, D.P., Gruenthal, K.M., Canino, M.F., Lowry, D., Fisher, M.C. and Hauser, L., 2018. Population assignment and local adaptation along an isolation‐by‐distance gradient in Pacific cod (_Gadus macrocephalus_). Evolutionary Applications, 11(8), pp.1448-1464.

Fournier, D. A., H. J. Skaug, J. Ancheta, J. Ianelli, A. Magnusson, M. N. Maunder, A. Nielsen, and J. Sibert.  2012.  AD Model Builder: using automatic differentiation for statistical inference of highly parameterized complex nonlinear models.  Optimization Methods and Software 27:233-249.

Gotceitas, V., S. Fraser, J. A. Brown.  1997.  Use of eelgrass beds (_Zostera marina_) by juvenile Atlantic cod (_Gadus morhua_).  Canadian Journal of Fisheries and Aquatic Sciences 54:1306-1319.

Greer-Walker, M.  1970.  Growth and development of the skeletal muscle fibres of the cod (_Gadus morhua_ L.).  Journal du Conseil 33:228-244.

Handegard, N.O., and D. Tjøstheim.  2005.  When fish meet a trawling vessel: examining the behaviour of gadoids using a free-floating buoy and acoustic split-beam tracking.  Canadian Journal of Fisheries and Aquatic Sciences 62:2409–2422.

Hare, S. R., and N. J. Mantua.  2000.  Empirical evidence for North Pacific regime shifts in 1977 and 1989.  Progress in Oceanography 47:103-146.

Holsman K. and K. Aydin. 2015. Comparative methods for evaluating climate change impacts on the foraging ecology of Alaskan groundfish. Mar. ecol. Prog. Ser. 521:217-235. doi: 10.3354/meps11102.

Howard, R., and E. Laman, 2024. Bottom Trawl Survey Temperature Analysis. In: Ortiz, I. and S. Zador. 2024. Ecosystem Status Report 2024: Aleutian Islands, Stock Assessment and Fishery Evaluation Report, North Pacific Fishery Management Council, 1007 West 3rd Ave., Suite 400, Anchorage, Alaska 99501.

Hulson, P-J. F., B. C. Williams, M. R. Siskey, M. D. Bryan, and J. Conner. 2023. Bottom trawl survey age and length composition input sample sizes for stocks assessed with statistical catch-at-age as-sessment models at the Alaska Fisheries Science Center. U.S. Dep. Commer., NOAA Tech. Memo. NMFS-AFSC-470, 38 p.

Hurst, T. P, D. W. Cooper, J. S. Scheingross, E. M. Seale, B. J. Laurel, M. L. Spencer.  2009.  Effects of ontogeny, temperature, and light on vertical movements of larval Pacific cod (_Gadus macrocephalus_).  Fisheries Oceanography 18:301-311.

Hurst, T. P., B. J. Laurel, L. Ciannelli.  2010.  Ontogenetic patterns and temperature-dependent growth rates in early life stages of Pacific cod (_Gadus macrocephalus_).  Fishery Bulletin, U.S. 108:382-392.

Hurst, T. P., J. H. Moss, and J. A. Miller.  2012.  Distributional patterns of 0-group Pacific cod (_Gadus macrocephalus_) in the eastern Bering Sea under variable recruitment and thermal conditions.  ICES Journal of Marine Science 69:163-174.

Hurst, T. P., D. W. Cooper, J. T. Duffy-Anderson, E. V. Farley.  2015.  Contrasting coastal and shelf nursery habitats of Pacific cod in the southeastern Bering Sea.  ICES Journal of Marine Science 72:515-527.

Hurtado-Ferro, F., C. S. Szuwalski, J. L. Valero, S. C. Anderson, C. J. Cunningham, K. F. Johnson, R. Licandeo, C. R. McGilliard, C. C. Monnahan, M. L. Muradian, K. Ono, K. A. Vert-Pre, A. R. Whitten, and A. E. Punt.  2015.  Looking in the rear-view mirror: bias and retrospective patterns in integrated, age-structured stock assessment models.  ICES Journal of Marine Science 72:99-110.

Jensen, A. L.  1996.  Beverton and Holt life history invariants result from optimal trade-off of reproduction and survival.  Can. J. Fish. Aquat. Sci. 53:820-822.

Ketchen, K. S.  1961.  Observations on the ecology of the Pacific cod (_Gadus macrocephalus_) in Canadian waters.  Journal of the Fisheries Research Board of Canada 18:513-558.

Lang, G. M., C. W. Derrah, and P. A. Livingston.  2003.  Groundfish food habits and predation on commercially important prey species in the Eastern Bering Sea from 1993 through 1996.  Alaska Fisheries Science Center Processed Report 2003-04.  Alaska Fisheries Science Center, 7600 Sand Point Way NE., Seattle, WA 98115-6349.  351 p.

Laurel, B.J., Stoner, A.W., Ryer, C.H., Hurst, T.P. and Abookire, A.A., 2007. Comparative habitat associations in juvenile Pacific cod and other gadids using seines, baited cameras and laboratory techniques. Journal of Experimental Marine Biology and Ecology, 351(1-2), pp.42-55.

Laurel, B.J., Hurst, T.P., Copeman, L.A. and Davis, M.W., 2008. The role of temperature on the growth and survival of early and late hatching Pacific cod larvae (_Gadus macrocephalus_). Journal of Plankton Research, 30(9), pp.1051-1060.

Laurel, B.J., Hurst, T.P. and Ciannelli, L., 2011. An experimental examination of temperature interactions in the match–mismatch hypothesis for Pacific cod larvae. Canadian Journal of Fisheries and Aquatic Sciences, 68(1), pp.51-61.

Laurel, B.J., Copeman, L.A. and Parrish, C.C., 2012. Role of temperature on lipid/fatty acid composition in Pacific cod (_Gadus macrocephalus_) eggs and unfed larvae. Marine Biology, 159, pp.2025-2034.

Laurel, B.J., and L. A. Rogers. 2020. Loss of spawning habitat and prerecruits of Pacific cod during a Gulf of Alaska heatwave. Canadian Journal of Fisheries and Aquatic Sciences. 77(4): 644-650. https://doi.org/10.1139/cjfas-2019-0238.

Laurel, B. J., Abookire, A., Barbeaux, S. J., Almeida, L. Z., Copeman, L. A., Duffy-Anderson, J., et al. 2023. Pacific cod in the Anthropocene: An early life history perspective under changing thermal habitats. Fish Fish . doi: 10.1111/faf.12779.

Lemagie, E. and M. Callahan. 2024. Regional Sea Surface Temperature and Marine Heatwaves. In:  Ortiz, I. and S. Zador. 2024. Ecosystem Status Report 2024: Aleutian Islands, Stock Assessment and Fishery Evaluation Report, North Pacific Fishery Management Council, 1007 West 3rd Ave., Suite 400, Anchorage, Alaska 99501.

Livingston, P. A.  1989.  Interannual trends in Pacific cod, _Gadus macrocephalus_, predation on three commercially important crab species in the eastern Bering Sea.  Fish. Bull., U.S. 87:807-827.

Livingston, P. A.  1991.  Pacific cod.  In P. A. Livingston (editor), Groundfish food habits and predation on commercially important prey species in the eastern Bering Sea from 1984 to 1986, p. 31-88.  U.S. Dept. Commer., NOAA Tech. Memo. NMFS F/NWC-207.

Livingston, P. A. (editor).  2002.  Ecosystem Considerations for 2003.  North Pacific Fishery Management Council, 605 West 4th Ave., Suite 306, Anchorage, AK 99501.

Loher, T. 2022. Dispersal and seasonal movements of Pacific halibut (_Hippoglossus stenolepis_) in the eastern Bering Sea and Aleutian Islands, as inferred from satellite-transmitting archival tags. Animal Biotelemetry, 10(18), 1–21. https://doi.org/10.1186/s40317-022-00288-w.

Logerwell, E.A., Aydin, K., Barbeaux, S., Brown, E., Conners, M.E., Lowe, S., Orr, J.W., Ortiz, I., Reuter, R. and Spencer, P., 2005. Geographic patterns in the demersal ichthyofauna of the Aleutian Islands. Fisheries Oceanography, 14, pp.93-112.

London, J., P. Boveng, S. Dahle, H. Ziel, C. Christman, J. Ver Hoef. 2022. Harbor seals in the Aleutian Islands. In Ortiz, I. and S. Zador, 2022. Ecosystem Status Report 2022: Aleutian Islands, Stock Assessment and Fishery Evaluation Report, North Pacific Fishery Management Council, 1007 West Third, Suite 400, Anchorage, Alaska 99501.

Neidetcher, S. K., Hurst, T. P., Ciannelli, L., Logerwell, E. A.  2014.  Spawning phenology and geography of Aleutian Islands and eastern Bering Sea Pacific cod (_Gadus macrocephalus_). Deep-Sea Research II: Topical Studies in Oceanography 109:204-214.

Nichol, D. G., T. Honkalehto, G. G. Thompson.  2007.  Proximity of Pacific cod to the sea floor: using archival tags to estimate fish availability to research bottom trawls.  Fisheries Research 86:129-135.

Nichol, D.G., Kotwicki, S. and Zimmermann, M., 2013. Diel vertical migration of adult Pacific cod _Gadus macrocephalus_ in Alaska. Journal of Fish Biology, 83(1), pp.170-189.

Ona, E., and O. R. Godø.  1990.  Fish reaction to trawling noise: the significance for trawl sampling. Rapports et Procès-Verbaux des Réunions du Conseil International pour l’Exploration de la Mer 189: 159–166.

Ortiz, I. 2024. Aleutian Islands Report Card. In: Ortiz, I. and S. Zador, 2024. Ecosystem Status Report 2024: Aleutian Islands, Stock Assessment and Fishery Evaluation Report, North Pacific Fishery Management Council, 1007 West Third, Suite 400, Anchorage, Alaska 99501.

Parker-Stetter, S. L., J. K. Horne, E. V. Farley, D. H. Barbee, A. G. Andrews III, L. B. Eisner, J. M. Nomura.  2013.  Summer distributions of forage fish in the eastern Bering Sea.  Deep-Sea Research II 92:211-230.

Pitcher, K. W.  1981.  Prey of the Steller sea lion, _Eumetopias jubatus_, in the Gulf of Alaska.  U.S. Natl. Mar. Fish. Serv., Fish. Bull. 79:467-472.

Rand, K.M., McDermott, S.F., Bryan, D., Nielsen, J.K., Spies, I.B., Barbeaux, S.J., Loomis, T. and Gauvin, J., 2022. Non-random fishery data can validate research survey observations of Pacific cod (_Gadus macrocephalus_) size in the Bering Sea. Polar Biology, pp.1-10.

Rojek, N., Lindsey, J., Arumitsu, M., Drummond, B., Renner, H., Kaler, R. Ortiz, I. and S. Zador. 2024. Integrated Seabird Information. In: Ortiz, I. and S. Zador, 2024. Ecosystem Status Report 2024: Aleutian Islands, Stock Assessment and Fishery Evaluation Report, North Pacific Fishery Management Council, 1007 West Third, Suite 400, Anchorage, Alaska 99501.

Rose, G.A. and Kulka, D.W., 1999. Hyperaggregation of fish and fisheries: how catch-per-unit-effort increased as the northern cod (_Gadus morhua_) declined. Canadian Journal of Fisheries and Aquatic Sciences, 56(S1), pp.118-127.

Rugen, W.C., and Matarese, A.C. 1988. Spatial and temporal distribution and relative abundance of Pacific cod (_Gadus macrocephalus_) larvae in the western Gulf of Alaska. Vol. 88, no. 18. Resource Assessment and Conservation Engineering Division, Northwest and Alaska Fisheries Center, National Marine Fisheries Service, National Oceanic and Atmospheric Administration.

Sakurai, Y., T. Hattori.  1996.  Reproductive behavior of Pacific cod in captivity.  Fisheries Science 62:222-228.
Savin, A. B.  2008.  Seasonal distribution and Migrations of Pacific cod _Gadus macrocephalus_ (Gadidae) in Anadyr Bay and adjacent waters.  Journal of Ichythyology 48:610-621.

Schaal, S., Larson, W., Vollenweider, J., Miller, K., Klenz, T., Maselko, J., Neff, D. Tobin, C., McDermott, S., and Spies, I. Submitted 2024. 
Genetic data reveal non-local juvenile recruitment and variable seasonal movement of a highly mobile marine fish across Alaska. Evolutionary Applications.

Shimada, A. M., and D. K. Kimura. 1994. Seasonal movements of Pacific cod (_Gadus macrocephalus_) in the eastern Bering Sea and adjacent waters based on tag-recapture data. U.S. Natl. Mar. Fish. Serv., Fish. Bull. 92:800-816.

Sinclair, E. S. and T. K. Zeppelin.  2002.   Seasonal and spatial differences in diet in the western stock of Steller sea lions (_Eumetopias jubatus_).  Journal of Mammalogy 83(4).

Slesinger, E., Mundorff, S., Laurel, B. J., and Hurst, T. P. 2024. The combined effects of ocean warming and ocean acidification on Pacific cod (_Gadus macrocephalus_) early life stages. Mar. Biol. 171. doi: 10.1007/s00227-024-04439-w

Somerton, D.A., 2004. Do Pacific cod (_Gadus macrocephalus_) and walleye pollock (_Theregra chalcogramma_) lack a herding response to the doors, bridles, and mudclouds of survey trawls?. ICES Journal of Marine Science, 61(7), pp.1186-1189.

Spencer, L., Slesinger, E,. Spies, I., Laurel, B., Hurst, T.. Submitted (2024) Molecular indicators of warming and other climate stressors in larval Pacific cod. CJFAS.

Spies, I., 2012. Landscape genetics reveals population subdivision in Bering Sea and Aleutian Islands Pacific cod. Transactions of the American Fisheries Society, 141(6), pp.1557-1573.

Spies, I., Gruenthal, K.M., Drinan, D.P., Hollowed, A.B., Stevenson, D.E., Tarpey, C.M. and Hauser, L., 2020. Genetic evidence of a northward range expansion in the eastern Bering Sea stock of Pacific cod. Evolutionary Applications, 13(2), pp.362-375.

Spies, I., Tarpey, C., Kristiansen, T., Fisher, M., Rohan, S. and Hauser, L., 2022. Genomic differentiation in Pacific cod using P ool‐S eq. Evolutionary Applications, 15(11), pp.1907-1924.

Stark, J. W.  2007.  Geographic and seasonal variations in maturation and growth of female Pacific cod (_Gadus macrocephalus_) in the Gulf of Alaska and Bering Sea.  Fish. Bull. 105:396-407.

Stevenson, D.E. and Lauth, R.R., 2019. Bottom trawl surveys in the northern Bering Sea indicate recent shifts in the distribution of marine species. Polar Biology, 42(2), pp.407-421.

Sweeney, K. and T. Gelatt. 2024. Steller sea Lions in the Aleutian Islands. In: Ortiz, I. and S. Zador, 2024. Ecosystem Status Report 2024: Aleutian Islands, Stock Assessment and Fishery Evaluation Report, North Pacific Fishery Management Council, 1007 West Third, Suite 400, Anchorage, Alaska 99501.

Szuwalski, C.S., Hollowed, A.B., Holsman, K.K., Ianelli, J.N., Legault, C.M., Melnychuk, M.C., Ovando, D. and Punt, A.E., 2023. Unintended consequences of climate‐adaptive fisheries management targets. Fish and Fisheries, 24(3), pp.439-453.

Thompson, G., J. Ianelli, R. Lauth, S. Gaichas, and K. Aydin.  2008.  Assessment of the Pacific cod stock in the Eastern Bering Sea and Aleutian Islands Area.  In Plan Team for Groundfish Fisheries of the Bering Sea/Aleutian Islands (compiler), Stock assessment and fishery evaluation report for the groundfish resources of the Bering Sea/Aleutian Islands regions, p. 221-401.  North Pacific Fishery Management Council, 605 W. 4th Avenue Suite 306, Anchorage, AK 99501.

Thompson, G. G., and W. A. Palsson.  2018.  Assessment of the Pacific cod stock in the Aleutian Islands.  In Plan Team for Groundfish Fisheries of the Bering Sea/Aleutian Islands (compiler), Stock assessment and fishery evaluation report for the groundfish resources of the Bering Sea/Aleutian Islands regions, chapter 2, p. 1-48.  North Pacific Fishery Management Council, 605 W. 4th Avenue Suite 306, Anchorage, AK 99501.

Thomson, J. A.  1963.  On the demersal quality of the fertilized eggs of Pacific cod, _Gadus macrocephalus_ Tilesius.  Journal of the Fisheries Research Board of Canada 20:1087-1088.

Ueda, Y., Y. Narimatsu, T. Hattori, M. Ito, D. Kitagawa, N. Tomikawa, and T. Matsuishi.  2006.  Fishing efficiency estimated based on the abundance from virtual population analysis and bottom-trawl surveys of Pacific cod (_Gadus macrocephalus_) in the waters off the Pacific coast of northern Honshu, Japan.  Nippon Suisan Gakkaishi 72:201-209.

Westrheim, S. J.  1996.  On the Pacific cod (_Gadus macrocephalus_) in British Columbia waters, and a comparison with Pacific cod elsewhere, and Atlantic cod (_G. morhua_).  Can. Tech. Rep. Fish. Aquat. Sci. 2092.  390 p.

Xiao, D., and Ren, H.L. 2022. A regime shift in North Pacific annual mean sea surface temperature in 2013/14. Front. Earth Sci. doi.org/10.3389/feart.2022.987349.

Yang, M-S.  2004.  Diet changes of Pacific cod (_Gadus macrocephalus_) in Pavlof Bay associated with climate changes in the Gulf of Alaska between 1980 and 1995.  U.S. Natl. Mar. Fish. Serv., Fish. Bull. 102:400-405.

Zador, S. (editor).  2011.  Ecosystem considerations for 2012.  North Pacific Fishery Management Council, 605 W. 4th Avenue Suite 306, Anchorage, AK 99501.


\pagebreak
 
# Tables



`r table_nums(name = "M24_0_1_comparison", caption = " Summary table with a comparison of Models 24.1, 24.1a, and 24.0. *Asterisk denotes natural mortality estimated in the timeblock 2016-2024. The 2025 and 2026 ABCs for Model 24.2 are based on Model 24.1, and are used in place of the maxABC for Model 24.2.")`

``` {r M24_0_1_comparison, echo=FALSE, message=FALSE}


```

\begin{table}[ht]
\centering
\begin{tabular}{lrr|rr|rr}
  \hline
       & \multicolumn{2}{c|}{Model 24.1 } & \multicolumn{2}{c|}{Model 24.1a } & \multicolumn{2}{c}{Model 24.0 }  \\
        Quantity & `r thisyr+1 `      &`r thisyr+2`   & `r thisyr+1`      &`r thisyr+2` & `r thisyr+1`      &`r thisyr+2` \\ 
  \hline
$M$ (natural mortality rate)          &  `r round(as.numeric(M24_1$Natural_Mortality[25,23]),2)`, `r round(as.numeric(M24_1$Natural_Mortality[37,23]),2)`*   
                                      &  `r round(as.numeric(M24_1$Natural_Mortality[25,23]),2)`, `r round(as.numeric(M24_1$Natural_Mortality[37,23]),2)`*   
                                      & `r round(as.numeric(M24_1a$Natural_Mortality[25,23]),2)`, `r round(as.numeric(M24_1a$Natural_Mortality[37,23]),2)`*   
                                      &  `r round(as.numeric(M24_1a$Natural_Mortality[25,23]),2)`, `r round(as.numeric(M24_1a$Natural_Mortality[37,23]),2)`*  
                                      &  `r round(as.numeric(M24_0$Natural_Mortality[25,23]),2)`  & `r round(as.numeric(M24_0$Natural_Mortality[25,23]),2)`   \\
Tier                                  &  3   &  3   &  3   & 3 &  3   & 3   \\
Projected total (age 1+) biomass (t)  &  `r formatC(M24_1$timeseries$Bio_smry[which(M24_1$timeseries$Yr==thisyr+1)],big.mark=",",format="d")`  
                                      &  `r formatC(M24_1$timeseries$Bio_smry[which(M24_1$timeseries$Yr==thisyr+2)],big.mark=",",format="d")` 
                                      &  `r formatC(M24_1a$timeseries$Bio_smry[which(M24_1a$timeseries$Yr==thisyr+1)],big.mark=",",format="d")`  
                                      &  `r formatC(M24_1a$timeseries$Bio_smry[which(M24_1a$timeseries$Yr==thisyr+2)],big.mark=",",format="d")`  
                                      &  `r formatC(M24_0$timeseries$Bio_smry[which(M24_1$timeseries$Yr==thisyr+1)],big.mark=",",format="d")` 
                                      & `r formatC(M24_0$timeseries$Bio_smry[which(M24_1$timeseries$Yr==thisyr+2)],big.mark=",",format="d")` \\
Projected female spawning biomass (t) &  `r formatC(Two_year$SSB[1],big.mark=",",format="d")`  &  `r formatC(Two_year$SSB[2],big.mark=",",format="d")`  
                                      & `r formatC(Two_year24_1a$SSB[1],big.mark=",",format="d")`  &  `r formatC(Two_year24_1a$SSB[2],big.mark=",",format="d")` 
                                      &  `r formatC(Two_year24_0$SSB[1],big.mark=",",format="d")` & `r formatC(Two_year24_0$SSB[2],big.mark=",",format="d")` \\
$\:\:\:\:\:\:B_{100\%}$               &  `r formatC(Two_year$SB100[1],big.mark=",",format="d")`  &  `r formatC(Two_year$SB100[2],big.mark=",",format="d")` 
                                      &  `r formatC(Two_year24_1a$SB100[1],big.mark=",",format="d")`  &  `r formatC(Two_year24_1a$SB100[2],big.mark=",",format="d")`  
                                      &  `r formatC(Two_year24_0$SB100[1],big.mark=",",format="d")` & `r formatC(Two_year24_0$SB100[2],big.mark=",",format="d")` \\
$\:\:\:\:\:\:B_{40\%}$                &  `r formatC(Two_year$SB40[1],big.mark=",",format="d")`  &  `r formatC(Two_year$SB40[2],big.mark=",",format="d")` 
                                      &  `r formatC(Two_year24_1a$SB40[1],big.mark=",",format="d")`  &  `r formatC(Two_year24_1a$SB40[2],big.mark=",",format="d")`  
                                      &  `r formatC(Two_year24_0$SB40[1],big.mark=",",format="d")` & `r formatC(Two_year24_0$SB40[2],big.mark=",",format="d")`  \\
$\:\:\:\:\:\:B_{35\%}$                &  `r formatC(Two_year$SB35[1],big.mark=",",format="d")`  &  `r formatC(Two_year$SB35[2],big.mark=",",format="d")`  
                                      &  `r formatC(Two_year24_1a$SB35[1],big.mark=",",format="d")`  &  `r formatC(Two_year24_1a$SB35[2],big.mark=",",format="d")`  
                                      &  `r formatC(Two_year24_0$SB35[1],big.mark=",",format="d")` & `r formatC(Two_year24_0$SB35[2],big.mark=",",format="d")` \\
$F_{OFL}$                             &  `r round(Two_year$F35[1],3)`   &  `r round(Two_year$F35[2],3)`   &    `r round(Two_year24_1a$F35[1],3)`   &  `r round(Two_year24_1a$F35[2],3)`   &  `r round(Two_year24_0$F35[1],3)`   & `r round(Two_year24_0$F35[2],3)`    \\
$maxF_{ABC}$                          &  `r round(Two_year$F40[1],3)`  &  `r round(Two_year$F40[2],3)`  &  `r round(Two_year24_1a$F40[1],3)`  &  `r round(Two_year24_1a$F40[2],3)`  &  `r round(Two_year24_0$F40[1],3)`    & `r round(Two_year$F40[2],3)`    \\
$F_{ABC}$                              &  `r round(Two_year$F40[1],3)`   & `r round(Two_year24_0$F40[2],3)`   &  `r round(Two_year24_1a$F40[1],3)`   & `r round(Two_year24_1a$F40[2],3)` &  `r round(Two_year24_0$F40[1],3)`   & `r round(Two_year24_0$F40[2],3)`  \\
$OFL$                                 &  `r formatC(Two_year$C_OFL[1],big.mark=",",format="d")`  &  `r formatC(Two_year$C_OFL[2],big.mark=",",format="d")` &  `r formatC(Two_year24_1a$C_OFL[1],big.mark=",",format="d")`  &  `r formatC(Two_year24_1a$C_OFL[2],big.mark=",",format="d")`  &  `r formatC(Two_year24_0$C_OFL[1],big.mark=",",format="d")` & `r formatC(Two_year24_0$C_OFL[2],big.mark=",",format="d")` \\
$maxABC$                              &  `r formatC(Two_year$C_ABC[1],big.mark=",",format="d")`  &  `r formatC(Two_year$C_ABC[2],big.mark=",",format="d")` 
 &  `r formatC(Two_year24_1a$C_ABC[1],big.mark=",",format="d")`  &  `r formatC(Two_year24_1a$C_ABC[2],big.mark=",",format="d")` &  `r formatC(Two_year24_0$C_ABC[1],big.mark=",",format="d")` & `r formatC(Two_year24_0$C_ABC[2],big.mark=",",format="d")` \\
$ABC$                                 &  `r formatC(Two_year$C_ABC[1],big.mark=",",format="d")`  &  `r formatC(Two_year$C_ABC[2],big.mark=",",format="d")`  &  `r formatC(Two_year24_1a$C_ABC[1],big.mark=",",format="d")`  &  `r formatC(Two_year24_1a$C_ABC[2],big.mark=",",format="d")`  & `r formatC(Two_year24_0$C_ABC[1],big.mark=",",format="d")` & `r formatC(Two_year24_0$C_ABC[2],big.mark=",",format="d")` \\
\hline
Status                               & `r thisyr-1`       & `r thisyr`   & `r thisyr-1`       & `r thisyr`      &  `r thisyr-1`             & `r thisyr`          \\
\hline
Overfishing                           & No         & n/a      & No         & n/a       & No                        & n/a                 \\
Overfished                           & n/a         & No   & n/a         & No       & n/a                        & No                  \\
Approaching overfished               & n/a         & No   & n/a         & No       & n/a                        & No                  \\
\hline
\end{tabular}
\begin{tablenotes}
\item 
  \end{tablenotes}
\end{table}




\pagebreak


`r table_nums(name = "Tier5_comparison", caption = " Summary table with a comparison of Models 13.4 and 24.2 proposed for 2024. Model 13.4 (2023) was the tier 5 model used in 2023. M represents natural mortality rate. Note the 2024 accepted ABC and OFL were adjusted from the 2024 model values for OFL and ABC.")`

``` {r Tier5_comparison, echo=FALSE, message=FALSE}

```

\begin{table}[ht]
\centering
\begin{tabular}{lrr|rr|rr}
  \hline
       & \multicolumn{2}{c|}{Model 13.4 (2023) } &\multicolumn{2}{c|}{Model 13.4 } & \multicolumn{2}{c}{Model 24.2 }  \\
       & \multicolumn{2}{c|}{As estimated or $\mathit{specified}$ } & \multicolumn{2}{c}{As estimated or $\mathit{recommended}$} &  \multicolumn{2}{c}{As estimated or $\mathit{recommended}$ }  \\
       & \multicolumn{2}{c|}{$\mathit{last}$ year for:}  & \multicolumn{2}{c}{$\mathit{this}$ year for: }  & \multicolumn{2}{c}{$\mathit{this}$ year for: }               \\
        Quantity & `r thisyr`      &`r thisyr+1`   & `r thisyr+1`      &`r thisyr+2`& `r thisyr+1`      &`r thisyr+2` \\ 
  \hline
	$M$ 	&	0.34	&	0.34	&	0.34	&	0.34 & 0.417 & 0.417 \\
Tier	&	5	&	5	&	5	&	5 &	5	&	5 \\
Biomass (t)	& `r T5TotBio_lastF` & `r T5TotBio_lastF` & `r T5TotBio_thisF` & `r T5TotBio_nextF` & `r T5TotBio_thisF` & `r T5TotBio_nextF`  \\
$F_{OFL}$    	&	0.34	&	0.34 	&	0.34	&	0.34	&	0.417	&	0.417	\\
$maxF_{ABC}$   	&	0.255	&	0.255	&	0.255	&	0.255 &	0.313	&	0.313  	\\
$F_{ABC}$       &	0.255	&	0.255	&	0.255	&	0.255 &	0.313	&	0.313        \\
$OFL$    	      & `r T5OFL_2024`  & `r T5OFL_2024`  & `r T5OFL_thisF`  & `r T5OFL_thisF`  & `r T5OFL_thisF24_2`  & `r T5OFL_thisF24_2`    \\
$maxABC$    	  & `r T5ABC_2024` & `r T5ABC_2024` & `r T5ABC_thisF` & `r T5ABC_thisF`  & `r T5ABC_thisF24_2` & `r T5ABC_thisF24_2`\\
$ABC$ 	       & `r T5ABC_2024` & `r T5ABC_2024` & `r T5ABC_thisF` & `r T5ABC_thisF` & `r formatC(Two_year$C_ABC[1],big.mark=",",format="d")` & `r formatC(Two_year$C_ABC[2],big.mark=",",format="d")`    \\
\hline
Status	                              &	`r thisyr-2`	      &	 `r thisyr-1`	      &		`r thisyr-1`             &	`r thisyr` &		`r thisyr-1`             &	`r thisyr`          \\
\hline
Overfishing	                          &	No	        &	n/a	      &	No	                       &	n/a  &	No	                       &	n/a                 \\
\hline
\end{tabular}
\end{table}

\pagebreak


\pagebreak

`r table_nums(name="TableCompare", caption = " Comparison of likelihoods and model parameters among Models 24.1, 24.1a, and 24.0. The likelihood components, derived quantities, and parameter estimates for each model are shown below Results.")`

``` {r TableCompare, echo=FALSE, message=FALSE}

TableCompare=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/data/TableComparisons1.csv",header=FALSE)
TableCompare=TableCompare[,1:4]
colnames(TableCompare)=c("Features","M24.1","M24.1a","M24.0")
TableCompare[1,2:4]=""
kable(TableCompare,"latex",linesep="",longtable=FALSE,align="r",digits=0)%>%
  kable_styling(font_size = 7.5, position="left")%>%row_spec(4,hline_after=T, extra_latex_after = "%")%>%row_spec(1,hline_after=T, extra_latex_after = "%")%>%row_spec(3,hline_after=T, extra_latex_after = "%")

#kable(TARGmatr2,"latex",linesep="",longtable=FALSE,align='l')%>%add_header_above(c("Year"=1,"Non-target"=7,"Target"=1,"Overall"=1))%>%kable_styling(font_size = 8)

```
 
\pagebreak

\pagebreak





`r table_nums(name = "histcatch", caption = " Catch of Pacific cod in the Aleutian Islands by foreign, domestic, and joint venture fisheries, 1964-1980. Note that joint venture fisheries did not commence until 1981, and domestic catch information is not available prior to 1988.")`

``` {r histcatch, echo=FALSE,message=FALSE}
hist=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/AIcatch_1964_1980.csv",header=TRUE)

kable(hist,col.names=c("Year","Foreign","Joint Venture","Domestic","Total"),"latex",booktabs=T,linesep="",longtable=FALSE,align=c(rep('r',times=5)))
``` 

\pagebreak

\pagebreak

`r table_nums(name = "catch81_90", caption = " Summary of catches of Pacific cod (t) in the Aleutian Islands by gear type. All catches include discards. Domestic annual catch by gear is not available prior to 1988.")`

``` {r catch81_90, echo=FALSE,message=FALSE}
catch81_90=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/Catch_1981_1990.csv",header=TRUE)
kable(catch81_90,col.names=c(" ","Trawl","Longline","Total","Trawl","Trawl","Longline and pot","Total",""),"latex",booktabs=T,linesep="",longtable=FALSE,align=c(rep('r',times=9)))%>%add_header_above(c("Year"=1,"Foreign"=3,"Joint Venture"=1,"Domestic"=3,"Total"=1))
``` 



\pagebreak

`r table_nums(name="Fish_catch",caption = paste(" Federal and state fishery Pacific cod catch in metric tons by year, 1991-",thisyr,". To avoid confidentiality problems, federal longline and pot catches have been combined. “Other” gear types include gill net and jig. Catches for ",thisyr," are through September 22. The state fishery catch is included in the total, and broken out as a separate column from 2006 onward. State data for 2024 are preliminary.",sep=""))`

``` {r Fish_catch, echo=FALSE, message=FALSE}
#AKFIN Analyst specific reports Thompson PCod catch by juristictoin.

fish_catch$Trawl=formatC(fish_catch$Trawl,big.mark=",",format="d")
fish_catch$Longline.Pot.Catch=fish_catch$Longline+fish_catch$Pot
fish_catch$Longline.Pot.Catch=formatC(fish_catch$Longline.Pot.Catch,big.mark=",",format="d")
fish_catch$Other=formatC(fish_catch$Other,big.mark=",",format="d")
fish_catch$Total=formatC(fish_catch$Total,big.mark=",",format="d")
fish_catch2=data.frame(seq(1991,2024,1),fish_catch$Trawl,fish_catch$Longline.Pot.Catch,fish_catch$Other,fish_catch$Total,c(rep(0,5),formatC(cgearSTATE,big.mark = ",",format="d")))
colnames(fish_catch2)=c("Year","Trawl","Longline+Pot","Other","Total","State")
#I used the report file to fill in the fish_catch1 file.
kable(fish_catch2,"latex",booktabs=T,linesep="",longtable=FALSE,align=c(rep('r',times=6)))
#col.names=c(" ","Trawl","Longline+Pot","Other","Total"," ").  %>%add_header_above(c("Year"=1,"Gear"=3,"Total"=1))

```


\pagebreak

\pagebreak



`r table_nums(name="catch_AI",caption = paste(" Summary of 1994-2024 catches (t) of Pacific cod in the AI, by NMFS statistical area (area breakdowns not available prior to 1994).  Catches for ",thisyr,"are through September 22."))`

```{r catch_AI,echo=FALSE, message=FALSE}
#setu p for catch_AI table
catAI=cat24

yrs=names(table(catAI$Year))
catAI1=matrix(0,length(yrs),6)
areas=c(543,542,541)
for(i in 1:length(yrs)){
  for(j in 1:3){
      catAI1[i,j]=sum(catAI$Catch..mt.[which(catAI$Year==yrs[i]&catAI$NMFS.Area==areas[j])])
  }
}
rownames(catAI1)=yrs
colnames(catAI1)=c(areas,areas)

tots=rowSums(catAI1)


catAI1[,4]=format(catAI1[,1]/tots,digits=1)
catAI1[,5]=format(as.numeric(catAI1[,2])/tots,digits=2)
catAI1[,6]=format(as.numeric(catAI1[,3])/tots,digits=3)
catAI1[,1]=formatC(round(as.numeric(catAI1[,1])),format="d",big.mark=",")
catAI1[,2]=formatC(round(as.numeric(catAI1[,2])),format="d",big.mark=",")
catAI1[,3]=formatC(round(as.numeric(catAI1[,3])),format="d",big.mark=",")

catAIplot=reshape2::melt(catAI1)
colnames(catAIplot)=c("Year","Area","Catch")
catAIplot2=catAIplot[1:99,]
#ggplot(data=catAIplot2)+geom_line(aes(x=Year,y=Catch,color=as.factor(Area)),lwd=1.5)+theme_bw()+ggtitle("Catch of Pacific cod by year in the Aleutian Islands")

kable(catAI1[4:nrow(catAI1),],col.names=c("Western","Central","Eastern","Western","Central","Eastern"),"latex",booktabs=T,linesep="",longtable=FALSE,align=c(rep('r',times=6)))%>%add_header_above(c("Year","Total Catch"=3,"Proportions"=3))

```

\pagebreak

\pagebreak

`r table_nums(name = "catchABC1", caption = paste(" Pacific cod catch in metric tons by year, total allowable catch (TAC), acceptable biological catch (ABC), and overfishing limit (OFL), 1991-",thisyr,". Note that specifications were combined for the Bering Sea and Aleutian Islands cod stocks through 2013 and are shown for the Aleutian Islands alone for 2013 onwards. Catch for ",thisyr," is through September 22. ABC, OFL, and TAC for ", thisyr ," are based on last year's model output.",sep=""))`
``` {r catchABC1, echo=FALSE,message=FALSE}


Fish_catch_ABCOFL=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/Fish_catch_ABCOFL.csv",header=TRUE)


kable(Fish_catch_ABCOFL[,1:5],col.names=c("Year","Catch (t)","ABC","TAC","OFL"),"latex",booktabs=T,linesep="",longtable=FALSE,align=c(rep('r',times=5)))%>%row_spec(23,hline_after=T, extra_latex_after = "%")
``` 

\pagebreak




`r table_nums(name = "TARGTAB", caption = " Catch of Pacific cod from 1993-2024 in non-target and targeted fisheries, as of September 22, 2024. Inconsistencies in the final year of catch estimates are the result of delays in data processing. Catches in the previous table are updated prior to catches in the catch by target fishery (this table).")`
```{r TARGTAB, echo=FALSE, message=FALSE}


#TARG=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/Groundfish Total Catch by Fishery_ALL_TARGETS_9_22_24.csv",header=TRUE)
#Target=names(table(TARG$Trip.Target.Group))
#Yrs=as.numeric(names(table(TARG$Year)))
#TARGmat=matrix(0,length(Yrs),length(Target))

#for(i in 1:length(Target)){
#  for(j in 1:length(Yrs)){
#    TARGmat[j,i]=round(sum(TARG$Catch..mt.[which(TARG$Year==Yrs[j]&TARG$Trip.Target.Group==Target[i])]))
#  }
#}
#rownames(TARGmat)=Yrs
#colnames(TARGmat)=Target

#TARGmat[,5]=TARGmat[,4]+TARGmat[,5]
#TARGmat=TARGmat[,c(1,2,3,5,6,7,8,9)]
#write.csv(TARGmat,"/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/TARGMAT.csv")

#TARGmat1=reshape2::melt(TARGmat,value.name="Year")
#colnames(TARGmat1)=c("Year","Target","Value")
#write.csv(TARGmat1,"/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/TARGMAT1.csv")

TARGmat=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/TARGMAT.csv")
TARGmat$Total=rowSums(TARGmat[,2:9])
TARGmat$NonTarget=rowSums(TARGmat[,2:9])-TARGmat[,6]
TARGmatr=TARGmat[,c(2,3,4,7,8,5,11,6,10)]


TARGmatr2=sapply(TARGmatr,formatC,big.mark=",",format="d")
rownames(TARGmatr2)=seq(1993,2024,1)
colnames(TARGmatr2)=c("Atka\nmackerel","Flatfish","Halibut","Pollock","Rockfish","Other","Total\nnon-target","Pacific\ncod","Total")

TARGmatr2[1:(nrow(TARGmatr2)-1),9]=Fish_catch_ABCOFL$Catch[3:(length(Fish_catch_ABCOFL$Catch)-1)]

kable(TARGmatr2[2:nrow(TARGmatr2),],"latex",linesep="",longtable=FALSE,align='r')%>%add_header_above(c("Year"=1,"Non-target"=7,"Target"=1,"Overall"=1))%>%kable_styling(latex_options = "hold_position")%>%
  kable_styling(font_size = 8)

```

\pagebreak

\pagebreak


`r table_nums(name="retdis",caption = paste0(" Discarded Pacific cod (t) and discard rates for Pacific cod caught in the Aleutian Islands, for the period 1993 - September 22, ",thisyr,". Note that Amendment 49, which mandated increased retention and utilization, was implemented in 1998."))`

```{r retdis,echo=FALSE, message=FALSE}
#retained discarded in groundfish total discards and do not select a target fishery
#retdis=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/GroundfishTotalDiscardsAICod_Aug14_1991_2024.csv",header=TRUE)

#yrs=names(table(retdis$Year))
#tretdis=matrix(0,length(yrs),4)
#for(i in 1:length(yrs)){
#  tretdis[i,1]=yrs[i]
#  tretdis[i,2]=round(sum(retdis$Discards..mt.[which(retdis$Year==yrs[i])]))
#   tretdis[i,3]=round(sum(retdis$Total.Catch..mt.[which(retdis$Year==yrs[i])]))
#    tretdis[i,4]=as.numeric(tretdis[i,2])/as.numeric(tretdis[i,3])
#}

#tretdis1=data.frame(cbind(yrs,formatC(as.numeric(tretdis[,2]),format="d",big.mark = ","),formatC(as.numeric(tretdis[,3]),format="d",big.mark = ","),formatC(as.numeric(tretdis[,4]),format="f",digits=3)))

#write.csv(tretdis1,"/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/tretdis1.csv")

tretdis1=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/tretdis1.csv",header=TRUE)
kable(tretdis1[,2:5],col.names=c("Year","Discards (t)","Total catch (t)","Proportion discarded"),booktabs=T,linesep="",longtable=FALSE,align=c(rep('r',times=4)))

thisyr=2024
```





\pagebreak


`r table_nums(name = "data_inline", caption = " Sources of data used in the age structured models, Model 24.0 and 24.1. *Data current through August 15, 2024.")`


```{r data_inline, echo=FALSE,message=FALSE}
data_inline=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/data_inline.csv",header=TRUE)

kable(data_inline[1:6,],"latex",linesep="",align='l')%>%kable_styling(latex_options="scale_down")%>%kable_styling(latex_options="hold_position")

#kable(hist,col.names=c("Year","Foreign","Joint Venture","Domestic","Total"),"latex",booktabs=T,linesep="",longtable=FALSE,align=c(rep('r',times=5)))

```

\pagebreak



`r table_nums(name = "srvbio1", caption = " Aleutian Islands bottom trawl survey biomass estimates, proportion, and standard error by NMFS area for Pacific cod, for all years used in the model.")`
``` {r srvbio1, echo=FALSE,message=FALSE}

#AKFIN biomass by NMFS reporting area
#Srv_bio2=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/data/survey.csv",header=TRUE)
#yrs=as.numeric(names(table(Srv_bio2$Year)))
#area=names(table(Srv_bio2$Area))
#area2=c("Central Aleutians","Eastern Aleutians","Western Aleutians")
#Srv_bio2mat=matrix(0,length(yrs),3)
#Srv_bio2var=matrix(0,length(yrs),3)
#for (i in 1: length(yrs)){
#  for(j in 1: length(area)){
#    Srv_bio2mat[i,j]=sum(Srv_bio2$Biomass[which(Srv_bio2$Year==yrs[i]&Srv_bio2$Area==area[j])])
#    Srv_bio2var[i,j]=sum(Srv_bio2$Var[which(Srv_bio2$Year==yrs[i]&Srv_bio2$Area==area[j])],na.rm=TRUE)
#      }
#}

#CV=format(sqrt(Srv_bio2var)/Srv_bio2mat,digits=2)

#Areavar=data.frame(cbind(yrs,Srv_bio2mat,Srv_bio2var))
#colnames(Areavar)=c("Year","Bio_CAI","Bio_EAI","Bio_WAI","Var_CAI","Var_EAI","Var_WAI")
#write.csv(Areavar,"/Users/ingrid.spies/Documents/AI_PCOD/2024/data/Areavar.csv")

#Srv_bio3=formatC(cbind(yrs,formatC(round(cbind(Srv_bio2mat,rowSums(Srv_bio2mat))), format="d", big.mark=",")),drop0trailing = FALSE)

#colnames(Srv_bio3)=c("Year",c("Western","Central","Eastern","Total"))    

#propS=format(Srv_bio2mat/rowSums(Srv_bio2mat),digits=3)

#Srv_bio4=cbind(yrs,propS,rep("1.000",length(yrs)))

#colnames(Srv_bio4)=c("Year",c("Western","Central","Eastern","Total"))

#Srv_bio5=formatC(cbind(yrs,CV,format(sqrt(rowSums(Srv_bio2var))/rowSums(Srv_bio2mat),digits=3)),drop0trailing = FALSE)
#colnames(Srv_bio5)=c("Year",c("Western","Central","Eastern","Total"))

#write.csv(Srv_bio3,"/Users/ingrid.spies/Documents/AI_PCOD/2024/data/Srv_bio3.csv")
#write.csv(Srv_bio4,"/Users/ingrid.spies/Documents/AI_PCOD/2024/data/Srv_bio4.csv")
#write.csv(Srv_bio5,"/Users/ingrid.spies/Documents/AI_PCOD/2024/data/Srv_bio5.csv")

Srv_bio3=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/data/Srv_bio3.csv",header=TRUE)
Srv_bio4=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/data/Srv_bio4.csv",header=TRUE)
Srv_bio5=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/data/Srv_bio5.csv",header=TRUE)


#here is how to get SE for SS input file
#sqrt(log(1+as.numeric(Srv_bio5[,5])^2))

kable(Srv_bio3[,2:6],"latex",booktabs=T,linesep="",longtable=FALSE,align="r")%>%add_header_above(c("Biomass (t)"=4),line=FALSE,align="l")

kable(Srv_bio4[,2:6],"latex",booktabs=T,linesep="",longtable=FALSE,align="r")%>%add_header_above(c("Proportion by area"=4),line=FALSE,align="l")

# formatC(Srv_bio5,drop0trailing = TRUE,digits=3),
kable(Srv_bio5[,2:6],"latex",booktabs=T,linesep="",longtable=FALSE,align="r",digits=3)%>%add_header_above(c("Biomass coefficient of variation"=4),line=FALSE,align="l")


``` 




\pagebreak

`r table_nums(name="nlens", caption = " The number of hauls in which length observations were taken for the fishery length composition data, by year, fishery effective sample size (ESS), survey input sample size (ISS), and survey ESS. The input sample size for the fishery was scaled to the mean ISS for the survey.")`

``` {r nlens, echo=FALSE, message=FALSE}
#Nsamp used in model
len_9_24=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/norpac_length_report_AI_Pcod_9_22_24.csv",header=TRUE)

haulz=vector()
yrs=seq(1991,2024,1)
for (i in 1:length(yrs)){
haulz[i]=length(table(len_9_24$Haul.Join[which(len_9_24$Year==yrs[i])]))  
}

ISSsurv=c(355,"-","-",136,"-","-",575,"-","-",664,"-",819,"-",1030,"-",538,"-","-","-",851,"-",733,"-",1067,"-",1158,"-",1009,"-","-","-",953,"-",629)
ESSsurv=c(14,"-","-",5,"-","-",23,"-","-",26,"-",33,"-",41,"-",21,"-","-","-",34,"-",29,"-",43,"-",46,"-",40,"-","-","-",38,"-",25)

ESS=round(haulz*0.074381)
fish_hauls_yr=cbind(yrs,haulz,ESS,ISSsurv,ESSsurv)



colnames(fish_hauls_yr)=c("Year","Fishery Hauls","Fishery ESS","Survey ISS","Survey ESS")



options(knitr.kable.NA = "-")
kable(fish_hauls_yr,"latex",booktabs=T,linesep="",longtable=FALSE,align="r",digits=0)

#round(Dat_base$lencomp$Nsamp[which(Dat_base$lencomp$FltSvy==2)]) survey ISS
#get ESS 0.039885 is from M24.1
#round(Dat_base$lencomp$Nsamp[which(Dat_base$lencomp$FltSvy==2)]*0.039885)
# [1] 14  5 23 26 33 41 21 34 29 43 46 40 38 25

``` 



\pagebreak



`r table_nums(name = "srvbio", caption = " Aleutian Islands bottom trawl biomass estimates (t) and longline survey relative population numbers and standard error for Pacific cod, for all years used in the models.")`
``` {r srvbio, echo=FALSE,message=FALSE}

#mods1 <- SS_output("/Users/ingridspies/admbmodels/AI_Pcod/2022_SS/AI_Pcod-SS/Stock_Synthesis_files/M22_1/",verbose=FALSE,printstats=FALSE)

#srvbio=cbind(
#mods1$cpue$Yr[which(mods1$cpue$Fleet_name=="Srv")], #mods1$cpue$Obs[which(mods1$cpue$Fleet_name=="Srv")], #mods1$cpue$SE_input[which(mods1$cpue$Fleet_name=="Srv")],
#mods1$cpue$Obs[which(mods1$cpue$Fleet_name=="LLSrv")],
#mods1$cpue$SE_input[which(mods1$cpue$Fleet_name=="LLSrv")])
#srvbio=data.frame(srvbio)
#srvbio=data.frame(srvbio)
#srvbio$X2=formatC(srvbio$X2,format="d",big.mark=",")
#srvbio$X2=na_if(srvbio$X2,"")
#srvbio$X4=formatC(srvbio$X4,format="d",big.mark=",")
#srvbio$X4=na_if(srvbio$X4,"")
#srvbio$X3=round(srvbio$X3,3)
#srvbio$X5=round(srvbio$X5,3)

#srvbio <- replace(srvbio, is.na(srvbio), "")
#srvbio=set_na(srvbio, na = c(var1 = 1.0, var2 = 1.0000000))
#srvbio <- replace(srvbio, is.na(srvbio), "")

#srvbio=set_na(srvbio, na = c(var1 = 1.0, var2 = 1.000))
#write.csv(srvbio,"/Users/ingridspies/Documents/WorkDellStuff/Assessments/AI_Cod/2022/srvbio.csv")
#I removed 1s by hand
srvbio=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/srvbio.csv",header=TRUE)


colnames(srvbio)=c("X","Year","Biomass (t)","S.E.","Index","S.E. ")

options(knitr.kable.NA = "-")


kable(srvbio[,2:6],"latex",booktabs=T,linesep="",longtable=FALSE,align="r",digits=3)%>%add_header_above(c(" "=1,"Trawl Survey"=2,"Longline Survey"=2),line=FALSE,align="l")


``` 


\pagebreak


`r table_nums(name="nages", caption = " Survey age composition sample size data, by year, including the number of individual fish, number of hauls, and effective sample size for each year. Input sample sizes were generated using the methodology of Hulson et al (2023). Effective sample sizes are tuned using Francis weighting methodology described in the text.")`

``` {r nages, echo=FALSE, message=FALSE}
#Nsamp used in model
#Pete Hulson calculated effective size
nhauls_age3=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/nhauls_age3.csv",header=TRUE)
nhauls_age3$ESS=round(nhauls_age3$Effective.size*0.163372)#from Model 24.1
colnames(nhauls_age3)=c("Year","Number of aged fish","Number of hauls","Input sample size","Effective sample size")

#Use GET_DOM_AGE.r to find out how many ages in each fishery
#geartype=1 is NPT non pelagic trawl
#gertype=2 is Pelagic Trawl 
#6 pot
#7 jig
#8 longline
#2020 trawl=6,pot=46,ll=194
#2021 ll=165,pot=31
options(knitr.kable.NA = "-")
kable(nhauls_age3[,],"latex",booktabs=T,linesep="",longtable=FALSE,align="r",digits=0)

``` 

\pagebreak


`r table_nums(name="M_estimates",caption=" Estimates of natural mortality, M, for Pacific cod throughout their range. Values marked with asterisks * have been used in stock assessments.")`
```{r M_estimates,echo=FALSE,message=FALSE}
Mest=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2022/M_estimates_stats2.csv",header=TRUE)

Mest1=data.frame(Mest)
colnames(Mest1)=c("Region","Reference Author","Year","M estimate")

kable(Mest1[,1:4],format="latex",booktabs=T,linesep="")%>%kable_styling(latex_options = "hold_position")



```


\pagebreak



`r table_nums(name = "maturity", caption = " Maturity at age ogives based on histological data (Stark 2007), and updated for this assessment with data from 2008-2024 (1,347 records), converted from lengths to ages. Observer-based maturity curves were used in age structured models.")`
``` {r maturity, echo=FALSE,message=FALSE}

#library(data.table)
#library(sizeMat)

##data_old<-data.table(read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2022/codmaturity_9_20_2021.csv"))
##data=data_old

#data<-data.table(read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/CodMaturity_9_22_24.csv"))
#dataAI<-data[NMFS_AREA%in%c(540:544)] 
#dataAI120<-dataAI[MONTH<=4&YEAR>2007&SEX=='F']   ## Restrict it to beginning of the year and years 2008-2019
#Lengths are for January, Feb, March, April
#1355 records (old was 1331 records)

#my_ogive_fq = gonad_mature(dataAI120,varNames = c("LENGTH", "MATURITY_DESCRIPTION"), inmName = "Immature",matName = c("Pre Spawn","Spawning","Spent","Developing","Resting" ), method = "bayes", niter = 999)

#length=seq(4,143,1)
#A=median(my_ogive_fq$A_boot)
#B=median(my_ogive_fq$B_boot)

#source("/Users/ingrid.spies/Documents/AI_PCOD/2024/R/read-admb.R")
#AI=read_rep("/Users/ingrid.spies/Documents/admbmodels/AI_Pcod/2021/AI_Pcod_1fishsel_2021.rep") #stark Model 19.0b


#2021 (1331 records)
#A=-8.084672; B=0.1471665
#mat_est21=(1/(1+exp(-(A + B*length))));
#mat_estAGE21=AI$lenage%*%mat_est

#2024 (1355 records)
#A=-8.143064;B=0.1483235
#mat_est24=(1/(1+exp(-(A + B*length))));
#mat_estAGE24=AI$lenage%*%mat_est

#In the SS3 model, 50% maturity at length is -A/B and the slope is -B!
#So for 2024,  
#-A/B 54.9007. 50% maturity
#-B -0.1483235 slope.

#Stark
#Ajs=-4.7143;Bjs=0.9654;
#mat_estSTARK=(1/(1+exp(-(Ajs + Bjs*(seq(1,10,1))))));

#matx=cbind(seq(1,10,1),mat_estSTARK,mat_estAGE21,mat_estAGE24)
#colnames(matx)=c("Age","Stark 2007","Observer data 2021","Observer data 2024")

#write.csv(matx,"/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/matx.csv")
matx=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/matx.csv",header=TRUE)

kable(matx[,2:4],col.names=c("Age","Stark 2007","Observer 2024"),"latex",booktabs=T,linesep="",longtable=FALSE,digits=3)
``` 




\pagebreak

`r table_nums(name="TotBio_est",caption=" Estimates of total biomass for Models 24.1, 24.1a, and 24.0.")`
```{r TotBio_est,echo=FALSE,message=FALSE}
Year=seq(1989,2024,1)
rec=data.frame(M24_1$timeseries$Bio_all,M24_1a$timeseries$Bio_all,M24_0$timeseries$Bio_all)
rec2=cbind(Year,rec[1:36,])



rec2[,2:4]=sapply(rec2[,2:4],formatC,big.mark=",",format="d")
rec2=data.frame(rec2)
colnames(rec2)=c(" ","Biomass (t)","Biomass (t)","Biomass (t)")

kable(rec2,format="latex",booktabs=T,linesep="",align="r")%>%kable_styling(latex_options = "hold_position")%>%add_header_above(c("Year", "Model 24.1"=1,"Model 24.1a" = 1,"Model 24.0" = 1))
```


\pagebreak

`r table_nums(name="SSB_est",caption=" Estimates of female spawning biomass (t) for Models 24.1, 24.1a, and 24.0, with upper and lower 95% confidence intervals.")`
```{r SSB_est,echo=FALSE,message=FALSE}
Year=seq(1989,2024,1)
rec=data.frame(mod.sum$SpawnBio$model1/2,mod.sum$SpawnBioLower$model1/2,mod.sum$SpawnBioUpper$model1/2,mod.sum$SpawnBio$model2/2,mod.sum$SpawnBioLower$model2/2,mod.sum$SpawnBioUpper$model2/2,mod.sum$SpawnBio$model3/2,mod.sum$SpawnBioLower$model3/2,mod.sum$SpawnBioUpper$model3/2)
rec2=cbind(Year,rec[1:36,])

colnames(rec2)=c(" ","Biomass (t)","LCI","UCI","Biomass","LCI","UCI","Biomass","LCI","UCI")

rec2[,2:10]=sapply(rec2[,2:10],formatC,big.mark=",",format="d")
rec2=data.frame(rec2)
colnames(rec2)=c(" ","Biomass (t)","LCI","UCI","Biomass","LCI","UCI","Biomass","LCI","UCI")

kable(rec2,format="latex",booktabs=T,linesep="",align="r")%>%kable_styling(latex_options = "hold_position")%>%add_header_above(c("Year", "Model 24.1"=3,"Model 24.1a" = 3,"Model 24.0" = 3))
```

\pagebreak

\pagebreak

`r table_nums(name="rec_est",caption=" Estimates of recruitment (age 0, x1,000) for Models 24.1, 24.1a, and 24.0 with upper and lower 95% confidence intervals.")`
```{r rec_est,echo=FALSE,message=FALSE}
Year=seq(1989,2024,1)
rec=data.frame(mod.sum$recruits$model1,mod.sum$recruitsLower$model1,mod.sum$recruitsUpper$model1,mod.sum$recruits$model2,mod.sum$recruitsLower$model2,mod.sum$recruitsUpper$model2,mod.sum$recruits$model3,mod.sum$recruitsLower$model3,mod.sum$recruitsUpper$model3)
rec2=cbind(Year,rec[1:36,])



rec2[,2:10]=sapply(rec2[,2:10],formatC,big.mark=",",format="d")
rec2=data.frame(rec2)
colnames(rec2)=c(" ","Recruitment","LCI","UCI","Recruitment","LCI","UCI","Recruitment","LCI","UCI")

kable(rec2,format="latex",booktabs=T,linesep="",align="r")%>%kable_styling(latex_options = "hold_position")%>%add_header_above(c("Year", "Model 24.1"=3,"Model 24.1a" = 3,"Model 24.0" = 3))
```

\pagebreak

\pagebreak

`r table_nums(name = "parsM24_1", caption = " Key parameter values estimated in Model 24.1.")`
 
```{r parsM24_1,echo=FALSE,message=FALSE}

M24_1P=M24_1$parameters[which(!is.na(M24_1$parameters$Parm_StDev)),]


M241_P2=data.frame(M24_1P$Label,round(M24_1P$Value,4),round(M24_1P$Parm_StDev,4))


colnames(M241_P2)=c("Parameter","Value","StDev")

kable(M241_P2,longtable=FALSE,"latex",linesep="",align=c('l','r','r'))%>%kable_styling(latex_options = "hold_position")%>%kable_styling(font_size = 8)%>%add_header_above(c(" "=1,"Model 24.1"=2),line=TRUE,align="c")

```

\pagebreak

`r table_nums(name = "parsM24_1a", caption = " Key parameter values estimated in Models 24.1a and 24.0.")`
 
```{r parsM24_1a,echo=FALSE,message=FALSE}


M24_1aP=M24_1a$parameters[which(!is.na(M24_1a$parameters$Parm_StDev)),]

M24_0P=M24_0$parameters[which(!is.na(M24_0$parameters$Parm_StDev)),]

M241_P2=data.frame(M24_1aP$Label,round(M24_1aP$Value,4),round(M24_1aP$Parm_StDev,4),
                   M24_0P$Label,round(M24_0P$Value,4),round(M24_0P$Parm_StDev,4))


colnames(M241_P2)=c("Parameter","Value","StDev","Parameter","Value","StDev")

kable(M241_P2,longtable=FALSE,"latex",linesep="",align=c('l','r','r','l','r','r'))%>%kable_styling(latex_options = "hold_position")%>%kable_styling(font_size = 8)%>%add_header_above(c("Model 24.1a"=3,"Model 24.0"=3),line=TRUE,align="c")

```


\pagebreak


\pagebreak

`r table_nums(name="T5Bio",caption=paste(" Biomass (t) estimated by the REMA model, 1991 - 2024, with lower (UCI) and upper (UCI) 95% confidence bounds."))`

```{r T5Bio, echo=FALSE,message=FALSE}

REMA[,3:5]=sapply(REMA[,3:5],formatC,big.mark=",",format="d")
kable(REMA[,c(2:5)],"latex",booktabs=T,linesep="",longtable=FALSE,align="r")



```


\pagebreak

`r table_nums(name = "Scenarios_M24_1", caption = " Projections of Aleutian Islands Pacific cod female future catch, full selection fishing mortality rates (F), and spawning biomass (SSB) for seven future harvest scenarios, based on Model 24.1. SSB and catch are in metric tons (t). For Scenario 2, we used the value 0.4 as a multiplier on $F_{ABC}$ for all future years, which yields F = 0.201 for all future years.")`

```{r Scenarios_M24_1,echo=FALSE,message=FALSE}

sM24_1_F=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run/AK_Scenarios/F_proj.csv",header=TRUE)

sM24_1_SSB=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run/AK_Scenarios/SSB_proj.csv",header=TRUE)

sM24_1_C=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run/AK_Scenarios/Catch_proj.csv",header=TRUE)

Catch=sM24_1_C[,2:9]
SSB=sM24_1_SSB[,2:9]
F=sM24_1_F[,2:9]
colnames(Catch)=c("Catch",seq(1,7,1))

colnames(F)=c("F",seq(1,7,1))

colnames(SSB)=c("SSB",seq(1,7,1))

kable(Catch,"latex",linesep="",longtable=FALSE,align='r')%>%add_header_above(c("Year"=1,"Scenarios"=7))%>%kable_styling(latex_options = "hold_position")%>%kable_styling(font_size = 8)

kable(F,"latex",linesep="",longtable=FALSE,align='r')%>%add_header_above(c("Year"=1,"Scenarios"=7))%>%kable_styling(latex_options = "hold_position")%>%kable_styling(font_size = 8)

kable(SSB,"latex",linesep="",longtable=FALSE,align='r')%>%add_header_above(c("Year"=1,"Scenarios"=7))%>%kable_styling(latex_options = "hold_position")%>%
  kable_styling(font_size = 8)


```


\pagebreak


`r table_nums(name = "Scenarios_M24_1a", caption = " Projections of Aleutian Islands Pacific cod female future catch, full selection fishing mortality rates (F), and spawning biomass (SSB) for seven future harvest scenarios, based on Model 24.1a. SSB and catch are in metric tons (t). For Scenario 2, we used the value 0.4 as a multiplier on $F_{ABC}$ for all future years, which yields F = 0.288 for all future years.")`

```{r Scenarios_M24_1a,echo=FALSE,message=FALSE}

sM24_1a_F=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1a_data/run/AK_Scenarios/F_proj.csv",header=TRUE)

sM24_1a_SSB=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1a_data/run/AK_Scenarios/SSB_proj.csv",header=TRUE)

sM24_1a_C=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1a_data/run/AK_Scenarios/Catch_proj.csv",header=TRUE)

Catch=sM24_1a_C[,2:9]
SSB=sM24_1a_SSB[,2:9]
F=sM24_1a_F[,2:9]
colnames(Catch)=c("Catch",seq(1,7,1))

colnames(F)=c("F",seq(1,7,1))

colnames(SSB)=c("SSB",seq(1,7,1))

kable(Catch,"latex",linesep="",longtable=FALSE,align='r')%>%add_header_above(c("Year"=1,"Scenarios"=7))%>%kable_styling(latex_options = "hold_position")%>%kable_styling(font_size = 8)

kable(F,"latex",linesep="",longtable=FALSE,align='r')%>%add_header_above(c("Year"=1,"Scenarios"=7))%>%kable_styling(latex_options = "hold_position")%>%kable_styling(font_size = 8)

kable(SSB,"latex",linesep="",longtable=FALSE,align='r')%>%add_header_above(c("Year"=1,"Scenarios"=7))%>%kable_styling(latex_options = "hold_position")%>%
  kable_styling(font_size = 8)


```


\pagebreak

`r table_nums(name = "Scenarios_M24_0", caption = " Projections of Aleutian Islands Pacific cod female future catch, full selection fishing mortality rates (F), and spawning biomass (SSB) for seven future harvest scenarios, based on Model 24.0. SSB and catch are in metric tons (t). For Scenario 2, we used the value 0.4 as a multiplier on $F_{ABC}$ for all future years, which yields F = 0.333 for all future years.")`

```{r Scenarios_M24_0,echo=FALSE,message=FALSE}

sM24_0_F=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_0_data/run/AK_Scenarios/F_proj.csv",header=TRUE)

sM24_0_SSB=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_0_data/run/AK_Scenarios/SSB_proj.csv",header=TRUE)

sM24_0_C=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_0_data/run/AK_Scenarios/Catch_proj.csv",header=TRUE)

Catch=sM24_0_C[,2:9]
SSB=sM24_0_SSB[,2:9]
F=sM24_0_F[,2:9]
colnames(Catch)=c("Catch",seq(1,7,1))

colnames(F)=c("F",seq(1,7,1))

colnames(SSB)=c("SSB",seq(1,7,1))

kable(Catch,"latex",linesep="",longtable=FALSE,align='r')%>%add_header_above(c("Year"=1,"Scenarios"=7))%>%kable_styling(latex_options = "hold_position")%>%kable_styling(font_size = 8)

kable(F,"latex",linesep="",longtable=FALSE,align='r')%>%add_header_above(c("Year"=1,"Scenarios"=7))%>%kable_styling(latex_options = "hold_position")%>%kable_styling(font_size = 8)

kable(SSB,"latex",linesep="",longtable=FALSE,align='r')%>%add_header_above(c("Year"=1,"Scenarios"=7))%>%kable_styling(latex_options = "hold_position")%>%
  kable_styling(font_size = 8)

#Set ABC ratio smaller than max:
#mods1$derived_quants[152:187,]$Value[34] F2024

#data.table(mods1$derived_quants)[Label=='annF_MSY']$Value #Fmsy

#mods1$derived_quants[152:187,]$Value[34]/data.table(mods1$derived_quants)[Label=='annF_MSY']$Value  #=0.22 Use this in AK scenarios scenario 2

```


\pagebreak

`r table_nums(name = "bycatchFMP", caption = " Incidental trawl catch of FMP species in the Aleutian Is. Pacific cod target fishery for Pacific cod, expressed as a proportion of the incidental catch of that species taken in all AI FMP fisheries, 1994-2024 (2024 current through October 1).")`
 
``` {r bycatchFMP, echo=FALSE,message=FALSE}
#Be sure to select all targets.
#byFMP=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/FMP Groundfish Bycatch_1991_Oct1_2024.csv",header=TRUE) #This is in the Pcod target fishery

#byFMP_HAL=byFMP[which(byFMP$Trip.Target.Name=="Pacific Cod"&byFMP$Gear%in%c("HAL")),]
#byFMP_Trawl=byFMP[which(byFMP$Trip.Target.Name=="Pacific Cod"&byFMP$Gear%in%c("BTR","NPT","PTR","TRW")),]

#yrs=names(table(byFMP_Trawl$Year))
#spp=names(table(byFMP_Trawl$Species.Group.Name))


#byFMPall=matrix(0,length(spp),length(yrs))#all target fisheries
#byFMPmatHAL=matrix(0,length(spp),length(yrs))#longline
#byFMPmatTRAWL=matrix(0,length(spp),length(yrs))#all trawl gear
#rownames(byFMPall)=spp;rownames(byFMPmatHAL)=spp;rownames(byFMPmatTRAWL)=spp
#colnames(byFMPall)=yrs;colnames(byFMPmatHAL)=yrs;colnames(byFMPmatTRAWL)=yrs
#for(i in 1:length(spp)){
#  for(j in 1:length(yrs)){
#    byFMPall[i,j]=sum(byFMP$Catch..mt.[which(byFMP$Year==yrs[j]&byFMP$Species.Group.Name==spp[i])],na.rm=TRUE)
#    byFMPmatHAL[i,j]=sum(byFMP_HAL$Catch..mt.[which(byFMP_HAL$Year==yrs[j]&byFMP_HAL$Species.Group.Name==spp[i])],na.rm=TRUE)
#    byFMPmatTRAWL[i,j]=sum(byFMP_Trawl$Catch..mt.[which(byFMP_Trawl$Year==yrs[j]&byFMP_Trawl$Species.Group.Name==spp[i])],na.rm=TRUE)
#  }
#}

#write.csv(byFMPall,"/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/byFMPall.csv")
#write.csv(byFMPmatTRAWL,"/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/byFMPmatTRAWL.csv")
#write.csv(byFMPmatHAL,"/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/byFMPmatHAL.csv")
#I combined species and edited in excel...maybe better in R.
#Make sure you pull data over all fisheries not just Pacific cod because it is proportion in all fisheries
byFMPall=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/byFMPall.csv",header=TRUE)
byFMPmatTRAWL=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/byFMPmatTRAWL.csv",header=TRUE)
byFMPmatHAL=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/byFMPmatHAL.csv",header=TRUE)

colnames(byFMPall)=c("Year",seq(1993,2024,1))
colnames(byFMPmatTRAWL)=c("Year",seq(1993,2024,1))
colnames(byFMPmatHAL)=c("Year",seq(1993,2024,1))


byFMPmatTRAWL2=round(byFMPmatTRAWL[,2:33]/byFMPall[,2:33],2)
colnames(byFMPmatTRAWL2)=seq(1993,2024,1)
rownames(byFMPmatTRAWL2)=byFMPmatTRAWL[,1]


byFMPmatHAL2=round(byFMPmatHAL[,2:33]/byFMPall[,2:33],2)
colnames(byFMPmatHAL2)=seq(1993,2024,1)
rownames(byFMPmatHAL2)=byFMPmatTRAWL[,1]

 
byFMPmatTRAWL3a=byFMPmatTRAWL2[,2:16]
byFMPmatTRAWL3b=byFMPmatTRAWL2[,17:ncol(byFMPmatTRAWL2)]

byFMPmatHAL3a=byFMPmatHAL2[,2:16]
byFMPmatHAL3b=byFMPmatHAL2[,17:ncol(byFMPmatHAL2)]

#rownames(byFMPmatTRAWL3a)[35]="SR/RE/SC/NR"
#rownames(byFMPmatTRAWL3b)[35]="SR/RE/SC/NR"

kable(list(byFMPmatTRAWL3a[3:nrow(byFMPmatTRAWL3a),]),"latex",booktabs=T,linesep="",longtable=FALSE,align="r",digits=3)%>%kable_styling(latex_options = "scale_down")%>%
  kable_styling(latex_options = "hold_position")

kable(byFMPmatTRAWL3b[3:nrow(byFMPmatTRAWL3b),],"latex",booktabs=T,linesep="",longtable=FALSE,align="r",digits=3)%>%kable_styling(latex_options = "scale_down")%>%
  kable_styling(latex_options = "hold_position")


```

\pagebreak


`r table_nums(name = "bycatchFMP_HAL", caption = " Incidental longline  catch of FMP species taken in the Aleutian Is. target fishery for Pacific cod, expressed as a proportion of the incidental catch of that species taken in all AI FMP fisheries, 1993-2024 (2024 current through October 1).")`
 
```  {r bycatchFMP_HAL, echo=FALSE,message=FALSE}

#rownames(byFMPmatHAL3a)[35]="SR/RE/SC/NR"
#rownames(byFMPmatHAL3a)[35]="SR/RE/SC/NR"

kable(byFMPmatHAL3a[3:nrow(byFMPmatHAL3a),],"latex",booktabs=T,linesep="")%>%kable_styling(latex_options = "scale_down")%>%
  kable_styling(latex_options = "hold_position")

kable(byFMPmatHAL3b[3:nrow(byFMPmatHAL3b),],"latex",booktabs=T,linesep="")%>%kable_styling(latex_options = "scale_down")%>%
  kable_styling(latex_options = "hold_position")

```

\pagebreak


`r table_nums(name = "OtherSP", caption = " Incidental catch of selected “Other Species” complex species taken in the AI Pacific cod trawl fisheries, 1991-2024 (2024 data current through October 1), expressed as a ratio of bycatch in all fisheries and gears.")`
``` {r OtherSP, echo=FALSE,message=FALSE}
#other=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/FMPOtherSpeciesBycatchCAS2_AI_1991_Oct1_2024.csv",header=TRUE)
#Stat area 540 used to be what is now 541, 542, and 543 so I will leave it in
#otherHAL=other[which(other$Gear.Code=="HAL"),]
#otherTRAWL=other[which(other$Gear.Code%in%c("BTR","NPT","PTR","TRW")),]

#yrs=names(table(otherTRAWL$Year))
#spp=names(table(otherTRAWL$Species.Common.Name))

#otherall=matrix(0,length(spp),length(yrs))
#othermatHAL=matrix(0,length(spp),length(yrs))
#othermatTRAWL=matrix(0,length(spp),length(yrs))
#for(i in 1:length(spp)){
#  for(j in 1:length(yrs)){
#    otherall[i,j]=sum(other$Weight.Posted..mt.[which(other$Year==yrs[j]&other$Species.Common.Name==spp[i])])
#    othermatTRAWL[i,j]=sum(otherTRAWL$Weight.Posted..mt.[which(otherTRAWL$Trip.Target.Name=="Pacific Cod"&otherTRAWL$Year==yrs[j]&otherTRAWL$Species.Common.Name==spp[i])])
#   othermatHAL[i,j]=sum(otherHAL$Weight.Posted..mt.[which(otherHAL$Trip.Target.Name=="Pacific Cod"&otherHAL$Year==yrs[j]&otherHAL$Species.Common.Name==spp[i])])
#  }
#}
#colnames(otherall)=yrs;colnames(othermatTRAWL)=yrs;colnames(othermatHAL)=yrs
#rownames(otherall)=spp;rownames(othermatTRAWL)=spp;rownames(othermatHAL)=spp

#othermatTRAWL_ratio=othermatTRAWL/otherall
#othermatTRAWL2=round(othermatTRAWL_ratio,2)

#othermatHAL_ratio=othermatHAL/otherall
#othermatHAL2=round(othermatHAL_ratio,2) 

#write.csv(othermatTRAWL2,"/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/othermatTRAWL2.csv")
#write.csv(othermatHAL2,"/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/othermatHAL2.csv")

othermatTRAWL2=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/othermatTRAWL2.csv",header=TRUE)
colnames(othermatTRAWL2)=c("Year",seq(1991,2024,1));rownames(othermatTRAWL2)=othermatTRAWL2[,1]

kable(othermatTRAWL2[,2:18],"latex",booktabs=T,linesep="",align="r")%>%kable_styling(latex_options = "scale_down")%>%
  kable_styling(latex_options = "hold_position")

kable(othermatTRAWL2[,19:ncol(othermatTRAWL2)],"latex",booktabs=T,linesep="",align="r")%>%kable_styling(latex_options = "scale_down")%>%
  kable_styling(latex_options = "hold_position")

```


\pagebreak


`r table_nums(name = "OtherSP_HAL", caption = " Incidental catch of selected “Other Species” complex species taken in the AI Pacific cod longline fisheries, 1991-2024 (2024 data current through October 1), expressed as a ratio of bycatch in all fisheries and gears.")`
``` {r OtherSP_HAL, echo=FALSE,message=FALSE}

othermatHAL2=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/othermatHAL2.csv",header=TRUE)
colnames(othermatHAL2)=c("Year",seq(1991,2024,1));rownames(othermatHAL2)=othermatHAL2[,1]

kable(othermatHAL2[,2:18],"latex",booktabs=T,linesep="",align="r")%>%kable_styling(latex_options = "scale_down")%>%
  kable_styling(latex_options = "hold_position")

kable(othermatHAL2[,19:35],"latex",booktabs=T,linesep="",align="r")%>%kable_styling(latex_options = "scale_down")%>%
  kable_styling(latex_options = "hold_position")



```


\pagebreak

`r table_nums(name = "prohibited", caption = " Incidental catch (herring and halibut in tons, salmon and crab in number of individuals) of prohibited species and discard mortality of halibut taken in the AI fisheries for Pacific cod (all gears), expressed as a proportion of the total for that species taken in all FMP AI fisheries, 1991-2024 (through October 1).")`
 
``` {r prohibited, echo=FALSE,message=FALSE}

#pro2=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/ProhibitedSpeciesCatchAI_1991_Oct1_2024.csv",header=TRUE)
#yrs=names(table(pro2$Year))
#spp=names(table(pro2$Species.Group.Name))
#pro_cod=matrix(0,length(spp),length(yrs))
#pro_all=matrix(0,length(spp),length(yrs))

#for(i in 1:length(spp)){
# for(j in 1:length(yrs)){
#    pro_cod[i,j]=sum(pro2$PSCNQ.Estimate....[which(pro2$Year==yrs[j]&pro2$Trip.Target.Name=="Pacific Cod"&pro2$Species.Group.Name==spp[i])])
#    pro_all[i,j]=sum(pro2$PSCNQ.Estimate....[which(pro2$Year==yrs[j]&pro2$Species.Group.Name==spp[i])])
#  }
#}


#pro_ratio=round(pro_cod/pro_all,2)
#colnames(pro_ratio)=yrs
#rownames(pro_ratio)=spp
#write.csv(pro_ratio,"/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/pro_ratio2.csv")

pro_ratio2=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/pro_ratio2.csv",header=TRUE)

colnames(pro_ratio2)=c("Year",seq(1991,2023,1));rownames(pro_ratio2)=pro_ratio2[,1]


kable(pro_ratio2[,2:17],"latex",booktabs=T,linesep="",align="r")%>%kable_styling(latex_options = "scale_down")%>%
  kable_styling(latex_options = "hold_position")

kable(pro_ratio2[,18:34],"latex",booktabs=T,linesep="",align="r")%>%kable_styling(latex_options = "scale_down")%>%
  kable_styling(latex_options = "hold_position")

```


\pagebreak

`r table_nums(name = "OtherSP2", caption = " Bycatch of Nontarget and Ecosystem Species for the Aleutian Islands Pacific cod fishery (all gear types), divided by the bycatch in all fisheries and gears in the same region. Bird bycatch is not included in this table. Data is from 1993-2024, and current through October 1 of the final year.  Continued on next page.")`

``` {r OtherSP2, echo=FALSE,message=FALSE}

#other2=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/NontargetSpeciesBycatchAI_2003_2024.csv",header=TRUE)
#yrs=names(table(other2$Year))
#spp=names(table(other2$Species.Group.Name))
#o2mat=matrix(0,length(spp),length(yrs))
#o2all=matrix(0,length(spp),length(yrs))
#for(i in 1:length(spp)){
#  for(j in 1:length(yrs)){
#    o2mat[i,j]=sum(other2$Nontarget.Estimate..mt.[which(other2$Trip.Target.Fishery=="Pacific Cod"&other2$Year==yrs[j]&other2$Species.Group.Name==spp[i])],na.rm=TRUE)
#    o2all[i,j]=sum(other2$Nontarget.Estimate..mt.[which(other2$Year==yrs[j]&other2$Species.Group.Name==spp[i])],na.rm=TRUE)
#  }
#}   
#Gotta check these
#spp[22]="Bryozoan Corals"
#spp[30]="Pacific Grenadier"
#spp[31]="Rattail Grenadier Unid."
#spp[40]="Lg. Sculpins - Hemilepidotus Unid."
#spp[41]="Lg. Sculpins - Myoxocephalus Unid."
#spp[23]="Bryozoan Red Tree Coral" 
#spp[22]="Bryozoan Unid." 

#o2=round(o2mat/o2all,2)
#colnames(o2)=yrs
#rownames(o2)=spp
#write.csv(o2,"/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/o2.csv")

o2=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/o2.csv",header=TRUE)
spp=o2[,1]
rownames(o2)=o2[,1];colnames(o2)=c("Year",seq(2003,2024,1))
keepspp=spp[which(rowSums(o2[,2:20],na.rm=TRUE)>0)]#These can be shown because they are non zero
#zerospp=spp[-which(rowSums(o2,na.rm=TRUE)>0)]

kable(o2[keepspp,2:12],"latex",booktabs=T,linesep="",align="r")%>%kable_styling(latex_options = "scale_down")%>%
  kable_styling(latex_options = "hold_position")

kable(o2[keepspp,13:ncol(o2)],"latex",booktabs=T,linesep="",align="r")%>%kable_styling(latex_options = "scale_down")%>%
  kable_styling(latex_options = "hold_position")
                             
```


\pagebreak

\begin{landscape}

`r table_nums(name = "bird", caption = " Bycatch of Nontarget and Ecosystem bird species for the Aleutian Islands Pacific cod fishery, expressed as a proportion of the incidental catch of that species group taken in the longline, trawl, and pot gear FMP AI fisheries 2003-2024 (through October 1).")`
 
``` {r bird, echo=FALSE,message=FALSE}
#spp=names(table(other2$Species.Group.Name))

#birdNPT=other2[which(other2$Trip.Target.Fishery=="Pacific Cod"&other2$Species.Group.Name%in%spp[2:17]&other2$Gear.Code=="NPT"),]
#birdHAL=other2[which(other2$Trip.Target.Fishery=="Pacific Cod"&other2$Species.Group.Name%in%spp[2:17]&other2$Gear.Code=="HAL"),]
#birdPOT=other2[which(other2$Trip.Target.Fishery=="Pacific Cod"&other2$Species.Group.Name%in%spp[2:17]&other2$Gear.Code=="POT"),]
#birdTOT=other2[which(other2$Species.Group.Name%in%spp[2:17]),]
#yrs=names(table(birdTOT$Year))
#spp=names(table(birdTOT$Species.Group.Name))

#birdHALmat=matrix(0,length(spp),length(yrs))
#birdNPTmat=matrix(0,length(spp),length(yrs))
#birdPOTmat=matrix(0,length(spp),length(yrs))
#birdALLmat=matrix(0,length(spp),length(yrs))

#for(i in 1:length(spp)){
#  for(j in 1:length(yrs)){
#    birdHALmat[i,j]=sum(birdHAL$Nontarget.Species.Count..Sum.[which(birdHAL$Year==yrs[j]&birdHAL$Species.Group.Name==spp[i])],na.rm=TRUE)
#    birdPOTmat[i,j]=sum(birdPOT$Nontarget.Species.Count..Sum.[which(birdPOT$Year==yrs[j]&birdPOT$Species.Group.Name==spp[i])],na.rm=TRUE)
#    birdNPTmat[i,j]=sum(birdNPT$Nontarget.Species.Count..Sum.[which(birdNPT$Year==yrs[j]&birdNPT$Species.Group.Name==spp[i])],na.rm=TRUE)
#    birdALLmat[i,j]=sum(birdTOT$Nontarget.Species.Count..Sum.[which(birdTOT$Year==yrs[j]&birdTOT$Species.Group.Name==spp[i])],na.rm=TRUE)
#  }
#}
#spp=c("Auklet","Black-footed Albatross","Cormorant", "Gull","Kittiwake",  "Laysan Albatross" , "Murre","Northern Fulmar"  ,"Other" ,                 "Other Alcid" , "Puffin" , "Shearwaters" , "Short-tailed Albatross", "Storm Petrels" , "Unidentified", "Unidentified Albatross" )

#rownames(birdHALmat)=spp;rownames(birdNPTmat)=spp
#colnames(birdHALmat)=yrs;colnames(birdNPTmat)=yrs
#colnames(birdPOTmat)=yrs;colnames(birdALLmat)=yrs
#rownames(birdALLmat)=spp;rownames(birdPOTmat)=spp

#birdHALmat2=round(birdHALmat/birdALLmat,2)[which(rowSums(birdHALmat,na.rm=TRUE)>0),]
#birdNPTmat2=round(birdNPTmat/birdALLmat,2)[which(rowSums(birdNPTmat,na.rm=TRUE)>0),]
#birdPOTmat2=round(birdPOTmat/birdALLmat,2)[which(rowSums(birdPOTmat,na.rm=TRUE)>0),]

#write.csv(birdHALmat2,"/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/birdHALmat2.csv")
#write.csv(birdNPTmat2,"/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/birdNPTmat2.csv")
#write.csv(birdPOTmat2,"/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/birdPOTmat2.csv")

birdHALmat2=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/birdHALmat2.csv",header=TRUE)

birdNPTmat2=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/birdNPTmat2.csv",header=TRUE)

birdPOTmat2=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/birdPOTmat2.csv",header=TRUE)

colnames(birdHALmat2)=c("Year",seq(2003,2024,1))
colnames(birdNPTmat2)=c("Year",seq(2003,2024,1))
colnames(birdPOTmat2)=c("Year",seq(2003,2024,1))
rownames(birdHALmat2)=birdHALmat2[,1]
rownames(birdNPTmat2)=birdNPTmat2[,1]
rownames(birdPOTmat2)=birdPOTmat2[,1]

kable(birdHALmat2[,2:23],"latex",booktabs=T,linesep="",align="r")%>%kable_styling(latex_options = "scale_down")%>%add_header_above(c("Longline"=23))%>%
  kable_styling(latex_options = "hold_position")

kable(birdNPTmat2[,2:23],"latex",booktabs=T,linesep="",align="r")%>%kable_styling(latex_options = "scale_down")%>%add_header_above(c("Non Pelagic Trawl"=23))%>%
  kable_styling(latex_options = "hold_position")

kable(birdPOTmat2[,2:23],"latex",booktabs=T,linesep="",align="r")%>%kable_styling(latex_options = "scale_down")%>%add_header_above(c("Pot Gear"=23))%>%
  kable_styling(latex_options = "hold_position")

```
\end{landscape}

\pagebreak




# Figures


```{r FishCondition,echo=FALSE}

```

\begin{center}
\includegraphics[width=6in,angle=0,origin=c]{/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/FishCondition.pdf}
\end{center}

`r figure_nums(name = "FishCondition", caption = paste(" Length-weight residuals for Pacific cod in the combined Aleutian Islands survey region (left), and by area (right), 1991 - 2024."))`

\pagebreak


```{r catch_gear, echo=FALSE,message=FALSE}


yrs=names(table(report$Year))
state=vector()
fed_trawl=vector()
fed_LLPOT=vector()
fed_POT=vector()
fed_LL=vector()
TOT=vector()
for(i in 1:length(yrs)){
  state[i]=sum(report$Catch..mt.[which(report$Year==yrs[i]&report$State.Federal.Fishery=="S")])
  fed_trawl[i]=sum(report$Catch..mt.[which(report$Year==yrs[i]&report$State.Federal.Fishery=="F"&report$Gear!="POT"&report$Gear!="HAL")])
  fed_LLPOT[i]=sum(report$Catch..mt.[which(report$Year==yrs[i]&report$State.Federal.Fishery=="F"&report$Gear%in%c("POT","HAL"))])
  fed_LL[i]=sum(report$Catch..mt.[which(report$Year==yrs[i]&report$State.Federal.Fishery=="F"&report$Gear=="HAL")])
  fed_POT[i]=sum(report$Catch..mt.[which(report$Year==yrs[i]&report$State.Federal.Fishery=="F"&report$Gear=="POT")])
  TOT[i]=sum(report$Catch..mt.[which(report$Year==yrs[i])])
}

#catch_gr1=cbind(yrs,state,fed_trawl,fed_LL,fed_POT)
catch_gr1=as.data.frame(cbind(rep(yrs,5),c(state,fed_trawl,fed_LL,fed_POT,TOT),c(rep("State",length(yrs)),rep("Trawl",length(yrs)),rep("Longline",length(yrs)),rep("Pot",length(yrs)),rep("TOTAL",length(yrs)))))
colnames(catch_gr1)=c("Year","Catch","Gear")
catch_gr1$Catch=as.numeric(catch_gr1$Catch)
catch_gr1$Gear=as.factor(catch_gr1$Gear)
catch_gr1$Year=as.numeric(catch_gr1$Year)

ggplot(catch_gr1)+geom_line(aes(x=Year,y=Catch,color=Gear))+theme_few()+geom_point(aes(x=2024,y=12431),color="blue")+ylab("Catch (t)")

#could add the M24.1 ABC geom_point(aes(x=2024,y=M24_1_Two_Year$C_ABC[1]),color="red")+geom_vline(xintercept = 2024)




```
`r figure_nums(name = "catch_gear", caption = " Aleutian Islands Pacific cod catch history, with federal catches by gear type, from 1991-2024 (through September 22). The blue dot represents the ABC for 2024.")`


\pagebreak

```{r dataSS3,echo=FALSE,message=FALSE}
dataSS3 <-  rasterGrob(as.raster(readPNG("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run/plots/data_plot2.png"), interpolate = FALSE))

grid.arrange(dataSS3)


```

`r figure_nums(name = "dataSS3", caption = " Data sources and relative weight used in Models 24.1, 24.1a, and 24.0.")`


\pagebreak


```{r catchbygear,echo=FALSE}

par(mfrow=c(3,1))
par(mar=c(3,3,2,1))
par(oma=c(1,1.5,0,0))

catz4=reshape2::melt(catz3,value.name=c("Month"))
colnames(catz4)=c("Month","Gear","Catch")
catz5=catz4[13:48,]

ggplot(catz5,aes(x=Month,y=Catch/5,fill=Gear))+geom_bar(stat="identity")+scale_x_continuous(breaks=seq(1,12,1))+ylab("Catch (t)")+theme_bw()


```

`r figure_nums(name="catchbygear",caption = paste(" Aleutian Islands Pacific cod average catch (t) by month per year and gear from January 1, 2020 - September 22, 2024. ",sep=""))`


\pagebreak


\pagebreak

```{r TARGET, echo=FALSE, message=FALSE}

#TARG=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/Groundfish Total Catch by Fishery_ALL_TARGETS_9_22_24.csv",header=TRUE)
#Target=names(table(TARG$Trip.Target.Group))
#Yrs=as.numeric(names(table(TARG$Year)))
#TARGmat=matrix(0,length(Yrs),length(Target))

#for(i in 1:length(Target)){
#  for(j in 1:length(Yrs)){
#    TARGmat[j,i]=round(sum(TARG$Catch..mt.[which(TARG$Year==Yrs[j]&TARG$Trip.Target.Group==Target[i])]))
#  }
#}
#rownames(TARGmat)=Yrs
#colnames(TARGmat)=Target

#TARGmat[,5]=TARGmat[,4]+TARGmat[,5]
#TARGmat=TARGmat[,c(1,2,3,5,6,7,8,9)]
#write.csv(TARGmat,"/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/TARGMAT.csv")

#TARGmat1=reshape2::melt(TARGmat,value.name="Year")
#colnames(TARGmat1)=c("Year","Target","Value")
#write.csv(TARGmat1,"/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/TARGMAT1.csv")

TARGmat1=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/TARGMAT1.csv")

ggplot() + geom_area(data=TARGmat1, aes(x=Year, y=Value, fill=Target)) + 
                 guides(fill = guide_legend(reverse=FALSE))+theme_bw()

```

`r figure_nums(name = "TARGET", caption = " Proportion of Pacific cod caught in targeted fisheries in the Aleutian Islands (541, 542, and 543) from 1991 through September 22, 2024.")`


\pagebreak




```{r REMA,echo=FALSE,message=FALSE}

#plot survbio

sb=data.frame(cbind(M24_1$cpue$Yr[which(M24_1$cpue$Obs>1)],M24_1$cpue$Obs[which(M24_1$cpue$Obs>1)],M24_1$cpue$SE[which(M24_1$cpue$Obs>1)]))
colnames(sb)=c("Year","Observation","SE")
LCI=sb$Observation-(sb$SE*sb$Observation*1.96)
HCI=sb$Observation+(sb$SE*sb$Observation*1.96)
sb2=cbind(sb,LCI,HCI)

ggplot(sb2)+geom_line(aes(x=Year,y=Observation))+geom_ribbon(aes(x=Year,y=Observation,ymin=LCI,ymax=HCI),alpha=0.2)+theme_bw(base_size=14)
```

\begin{center}
\includegraphics[width=6in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/REMA/REMA.pdf}
\end{center}

`r figure_nums(name = "REMA", caption = " Survey index of biomass with 95% confidence intervals (above) and Tier 5 fit to the survey index using a random effects model (below).")`

\pagebreak

```{r Bio_area,echo=FALSE,message=FALSE}

Areavar=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/data/Areavar.csv",header=TRUE)
CAI_LCI=Areavar$Bio_CAI-1.96*sqrt(Areavar$Var_CAI)
CAI_HCI=Areavar$Bio_CAI+1.96*sqrt(Areavar$Var_CAI)

Areavar2=data.frame(cbind(c(rep(Areavar$Year,3)),c(Areavar$Bio_CAI,Areavar$Bio_EAI,Areavar$Bio_WAI),c(Areavar$Var_CAI,Areavar$Var_EAI,Areavar$Var_WAI)))

colnames(Areavar2)=c("Year","Biomass","Variance")
Areavar2$LCI=Areavar2$Biomass-1.96*sqrt(Areavar2$Variance)
Areavar2$HCI=Areavar2$Biomass+1.96*sqrt(Areavar2$Variance)
Areavar2$Area=c(rep("CAI",14),rep("EAI",14),rep("WAI",14))

ggplot(sb2)+geom_line(aes(x=Year,y=Observation))+geom_ribbon(aes(x=Year,y=Observation,ymin=LCI,ymax=HCI),alpha=0.2)+theme_bw(base_size=14)

ggplot(Areavar2)+geom_line(aes(x=Year,y=Biomass,col=Area))+geom_ribbon(aes(x=Year,y=Biomass,ymin=LCI,ymax=HCI,fill=Area),alpha=0.2)+theme_bw(base_size=14)

```
`r figure_nums(name = "Bio_area", caption = " Survey estimate of Pacific cod biomass in the Aleutian Islands, by area.")`


```{r CAAL,echo=FALSE}

```

\begin{center}
\includegraphics[width=6in,angle=0,origin=l]{/Users/ingrid.spies/Documents/AI_PCOD/2024/data/CAAL.pdf}
\end{center}

`r figure_nums(name = "CAAL", caption = " Conditional age at length for the final five years of age data used in the model, for Models 24.1, 24.0, and 24.1a. Models 24.1 and 24.0 fit the Richards growth curve, while Model 24.1a uses the von Bertalanffy growth curve.")`

\pagebreak
\pagebreak

```{r agerainbow,echo=FALSE,message=FALSE}

#update cob_barplot to include 2022 data.

#age1=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2023/CodAges2.csv",header=TRUE)
#age2=age1[!is.na(age1$AGE),]
#Length=seq(130,1180,10)
#Age=seq(1,13,1)
#cod_barplot2=matrix(0,length(Length),13)#13 is for the number of ages
#colnames(cod_barplot2)=Age
#rownames(cod_barplot2)=Length
#for(i in 1:13){ #ages
#  for (j in 1:length(len)){
#    cod_barplot2[j,i]=length(age2$AGE[which(age2$LENGTH==Length[j]&age2$AGE==Age[i])])
#  }
#}

#cod_barplot22=reshape2::melt(cod_barplot2)
#colnames(cod_barplot24)=c("Length","Age","Frequency")
#write.csv(cod_barplot24,"/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/cod_barplot22.csv")

cod_barplot22=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/cod_barplot22.csv",header=TRUE) #this is the old one from 2018 data
ggplot(cod_barplot22, aes(x=Length,y=Frequency,fill = factor(Age)))+geom_density(stat="identity",alpha=0.8,color=NA)+theme_bw()+scale_color_brewer()+xlab("Length (mm)")+theme_bw()+guides(fill=guide_legend(title="Age"),colour=FALSE,size=FALSE)


#library(plotly)#

#cod_barplot=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2019/cod_barplot.csv",header=TRUE)
#a=ggplot(cod_barplot, aes(x=Length,y=Frequency,fill = factor(Age)))+geom_density(stat="identity",alpha=0.8,color=NA)+theme_bw()+scale_color_brewer()+xlab("Length (mm)")+theme_bw()+guides(fill=guide_legend(title="Age"))+guides(colour=FALSE,size=FALSE)
#gplotly(a)

#Mean size
#Age 1 2 3 4
#22.1. 38.3  48.9 58.7
#CV
#1.71 1.2 1.2 1.28

#Age 10: 99cm

#Age1m=sum(cod_barplot22$Length[which(cod_barplot22$Age==1)]*cod_barplot22$Frequency[which(cod_barplot22$Age==1)])/sum(cod_barplot$Frequency[which(cod_barplot22$Age==1)])/10 #22.1

#Variance
#Age1s=rep(cod_barplot22$Length[which(cod_barplot22$Age==1)],cod_barplot22$Frequency[which(cod_barplot22$Age==1)])

#CV1=sqrt(var(Age1s))/Age1m#1.81

#Age2m=sum(cod_barplot22$Length[which(cod_barplot22$Age==2)]*cod_barplot22$Frequency[which(cod_barplot22$Age==2)])/sum(cod_barplot22$Frequency[which(cod_barplot22$Age==2)])/10 #38.3
#Variance
#Age2s=rep(cod_barplot22$Length[which(cod_barplot22$Age==2)],cod_barplot22$Frequency[which(cod_barplot22$Age==2)])

#CV2=sqrt(var(Age2s))/Age2m#1.54

#Age3m=sum(cod_barplot22$Length[which(cod_barplot22$Age==3)]*cod_barplot22$Frequency[which(cod_barplot22$Age==3)])/sum(cod_barplot22$Frequency[which(cod_barplot22$Age==3)])/10 #48.9
#Variance
#Age3s=rep(cod_barplot22$Length[which(cod_barplot22$Age==3)],cod_barplot22$Frequency[which(cod_barplot22$Age==3)])


#CV3=sqrt(var(Age3s))/Age3m#1.3 

#Age4m=sum(cod_barplot22$Length[which(cod_barplot22$Age==4)]*cod_barplot22$Frequency[which(cod_barplot22$Age==4)])/sum(cod_barplot22$Frequency[which(cod_barplot22$Age==4)])/10

#Age4s=rep(cod_barplot22$Length[which(cod_barplot22$Age==4)],cod_barplot22$Frequency[which(cod_barplot22$Age==4)])
#CV4=sqrt(var(Age4s))/Age4m #1.28

#Age10m=sum(cod_barplot22$Length[which(cod_barplot22$Age==10)]*cod_barplot22$Frequency[which(cod_barplot22$Age==10)])/sum(cod_barplot22$Frequency[which(cod_barplot22$Age==10)])/10 #98.8

```
`r figure_nums(name="agerainbow",caption = " Length frequency by age of cod collected from surveys 1991-2022.")`

\pagebreak




```{r Selectivity,echo=FALSE}

```

\begin{center}

Model 24.1

\includegraphics[width=4in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run/plots/sel01_multiple_fleets_length1.png}

Model 24.1a

\includegraphics[width=4in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1a_data/run/plots/sel01_multiple_fleets_length1.png}

Model 24.0

\includegraphics[width=4in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_0_data/run/plots/sel01_multiple_fleets_length1.png}
\end{center}

`r figure_nums(name = "Selectivity", caption = " Survey and fishery selectivity curves for Models 24.1, 24.1a, and 24.0.")`

\pagebreak






```{r Regime, echo=FALSE,message=FALSE}

heat <-  rasterGrob(as.raster(readPNG("/Users/ingrid.spies/Documents/AI_PCOD/2024/AIPcod2024/AI_heatwavedays.png"), interpolate = FALSE))

grid.arrange(heat)

```

`r figure_nums(name = "Regime", caption = " The number of days under heatwave conditions for the western, central, and eastern Aleutian Islands, 1982 - 2023.")`

\pagebreak


```{r Heatwave_2024, echo=FALSE,message=FALSE}

```

\begin{center}
\includegraphics[width=6in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/data/Heatwave_2024.pdf}
\end{center}


`r figure_nums(name = "Heatwave_2024", caption = " Aleutian Islands mean sea surface temperature for the western, central, and eastern region, 2024 compared with 2023 and previous years. In 2024 there were several short periods considered heatwave conditions in the Aleutian Islands, but less than in previous years. ")`

\pagebreak


```{r Index,echo=FALSE}

```

\begin{center}

Model 24.1

\includegraphics[width=4in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run/plots/index2_cpuefit_Srv.png}

Model 24.1a

\includegraphics[width=4in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1a_data/run/plots/index2_cpuefit_Srv.png}

Model 24.0

\includegraphics[width=4in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_0_data/run/plots/index2_cpuefit_Srv.png}

\end{center}

`r figure_nums(name = "Index", caption = " Model fit to survey index for Models 24.1, 24.1a, and 24.0.")`

\pagebreak



```{r FracUnfished,echo=FALSE}

```

\begin{center}
\includegraphics[width=5in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/AIPcod2024/FracUnfished.pdf}
\end{center}


`r figure_nums(name = "FracUnfished", caption = " Spawning biomass relative to unfished for all Models 24.1, 24.1a, and 24.0.")`

\pagebreak


\pagebreak



```{r Growth,echo=FALSE}

```



\begin{center}

Model 24.1

\includegraphics[width=4in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run/plots/bio1_sizeatage.png}

Model 24.1a

\includegraphics[width=4in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1a_data/run/plots/bio1_sizeatage.png}

Model 24.0

\includegraphics[width=4in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_0_data/run/plots/bio1_sizeatage.png}
\end{center}

`r figure_nums(name = "Growth", caption = " Growth curves for Models 24.1, 24.1a, and 24.0. Model 24.1a used the von Bertalanffy growth curve, the other models used the Richards growth curve.")`

\pagebreak

```{r survey_index,echo=FALSE,message=FALSE}

```


\begin{center}

Model 24.1

\includegraphics[width=4in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run/plots/index2_cpuefit_Srv.png}

Model 24.1a

\includegraphics[width=4in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1a_data/run/plots/index2_cpuefit_Srv.png}

Model 24.0

\includegraphics[width=4in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_0_data/run/plots/index2_cpuefit_Srv.png}


\end{center}

`r figure_nums(name = "survey_index", caption = " Model 24.0 (upper panel), Model 24.1a (middle panel), and MOdel 24.0 (lower panel) fit to survey index, 1991 - 2024.")`

\pagebreak


```{r Recruitment,echo=FALSE}

```

\begin{center}
\includegraphics[width=5in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/AIPcod2024/RecDev.pdf}
\end{center}

`r figure_nums(name = "Recruitment", caption = " Recruitment estimates for all Models 24.1, 24.1a, and 24.0.")`

\pagebreak

```{r comp_lenfit,echo=FALSE}


```


Model 24.1\hspace{95mm}Model 24.1a

\includegraphics[width=4in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run/plots/comp_lenfit_flt2mkt0.png}
\includegraphics[width=4in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1a_data/run/plots/comp_lenfit_flt2mkt0.png}

Model 24.0

\includegraphics[width=4in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_0_data/run/plots/comp_lenfit_flt2mkt0.png}

`r figure_nums(name = "comp_lenfit", caption = " Survey length composition and fit to the length composition, Model 24.1 (upper left), Model 24.1a (upper right), and M24.0 (lower left).")`

\pagebreak

\pagebreak


```{r LengthFit,echo=FALSE}

```


Model 24.1\hspace{95mm}Model 24.1a


\begin{center}
\includegraphics[width=3in,angle=0,origin=l]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run/plots/comp_lenfit__aggregated_across_time.png}
\includegraphics[width=3in,angle=0,origin=r]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1a_data/run/plots/comp_lenfit__aggregated_across_time.png}

Model 24.0 

\includegraphics[width=3in,angle=0,origin=l]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_0_data/run/plots/comp_lenfit__aggregated_across_time.png}
\end{center}

`r figure_nums(name = "LengthFit", caption = " Model fit to observed length frequencies for survey and fishery data, aggregated across time Model 24.1 (upper left) and 24.1a (upper right), 24.0 (below).")`

\pagebreak


\pagebreak


```{r phase24_1, echo=FALSE, message=FALSE}

Dir_M24_1=("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run")

mods1 <- M24_1

#Gotta get the years to line up.
Bmsy=data.table(mods1$derived_quants)[Label=="SSB_MSY"]$Value
BoverBmsy=(mods1$timeseries$SpawnBio[3:38]/Bmsy)
Fabc=data.table(mods1$derived_quants)[Label=='annF_MSY']$Value
Fmsy=data.table(mods1$derived_quants)[Label=='annF_Btgt']$Value
FoverFmsy=data.table(mods1$derived_quants)[152:187,]$Value/Fmsy
FoverFmsy[length(FoverFmsy)-1]=mean(c(FoverFmsy[length(FoverFmsy)-2],FoverFmsy[length(FoverFmsy)-3]))
FoverFmsy[length(FoverFmsy)]=mean(c(FoverFmsy[length(FoverFmsy)-1],FoverFmsy[length(FoverFmsy)-2])) #Set next years F/Fmsy=this years F/Fmsy
SSB0  = data.table(mods1$derived_quants)[Label=='SSB_unfished']$Value
SB100=SSB0
Fmsy= data.table(mods1$derived_quants)[Label=='annF_Btgt']$Value
Fabc= data.table(mods1$derived_quants)[Label=='annF_MSY']$Value
xlim=c(0,2.2);ylim=c(0,1.6)
jitter.fac=0
Modelname="Model 24.1"

   
   plot(x=base::jitter(BoverBmsy,jitter.fac),y=base::jitter(FoverFmsy,jitter.fac),type="l",las=1,
          yaxs="i",xaxs="i",col="gray50",pch=20,xlim=xlim,ylim=ylim,xlab=expression(B/B[35~'%']),ylab=expression(F/F[35~'%']),
        main=paste(Modelname))
yr=seq(1991,2026,1)

  for(i in 1:(length(BoverBmsy)-2)){
    text(BoverBmsy[i],FoverFmsy[i],paste(yr[i]),cex=0.85)
    }
 for(i in (length(BoverBmsy)-1):length(BoverBmsy)){
    text(BoverBmsy[i],FoverFmsy[i],paste(yr[i]),col="purple",cex=0.85,font=2)
    }



k=Fabc/Fmsy*(((SSB0*0.2)/(SSB0*0.35))-0.05)/(1-0.05)
k2<-0.05
k3<-(0.2*SSB0)/(SSB0*0.35)
points(c(k3,k3),c(0,10),type="l",lty=3)
points(c((SB100*0.40)/(SB100*0.35),xlim[2]),c(Fabc/Fmsy,Fabc/Fmsy),type="l",lwd=5)
points(c((SB100*0.40)/(SB100*0.35),xlim[2]),c(1,1),type="l",lwd=5,col="red")
points(c(k3,(SB100*0.40)/(SB100*0.35)),c(k,Fabc/Fmsy),type="l",lwd=2)
points(c(k3,k3),c( 0,k),type="l",lwd=2)
points(c(k2,(SB100*0.40)/(SB100*0.35)),c(0,1.0),type="l",lwd=2,col="red")

text(xlim[2]-0.8,ylim[2]-0.1,"FOFL",pos=4)
text(xlim[2]-0.8,ylim[2]-0.2,"Max ABC",pos=4)
text (xlim[2]-0.8,ylim[2]-0.3,"B20% (no targeting)",pos=4)

points(c(xlim[2]-1.1,xlim[2]-0.8),c(ylim[2]-0.1,ylim[2]-0.1),lwd=2,type="l",col="red")
points(c(xlim[2]-1.1,xlim[2]-0.8),c(ylim[2]-0.2,ylim[2]-0.2),lwd=2,type="l")
points(c(xlim[2]-1.1,xlim[2]-0.8),c(ylim[2]-0.3,ylim[2]-0.3),type="l",lty=3)

abline(h=1,lty=2)
abline(v=1,lty=2)

```

`r figure_nums(name = "phase24_1", caption = " Phaseplane plot for Model 24.1, with forecasted results through 2026. Fishing mortality for 2025 and 2026 were estimated as the mean of the previous two years. ")`

\pagebreak


```{r phase24_0,echo=FALSE,message=FALSE}

Dir_M24_0=("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_0_data/run")

mods1 <- M24_0

#Gotta get the years to line up.
Bmsy=data.table(mods1$derived_quants)[Label=="SSB_MSY"]$Value
BoverBmsy=(mods1$timeseries$SpawnBio[3:38]/Bmsy)
Fabc=data.table(mods1$derived_quants)[Label=='annF_MSY']$Value
Fmsy=data.table(mods1$derived_quants)[Label=='annF_Btgt']$Value
FoverFmsy=data.table(mods1$derived_quants)[152:187,]$Value/Fmsy
FoverFmsy[length(FoverFmsy)-1]=mean(c(FoverFmsy[length(FoverFmsy)-2],FoverFmsy[length(FoverFmsy)-3]))
FoverFmsy[length(FoverFmsy)]=mean(c(FoverFmsy[length(FoverFmsy)-1],FoverFmsy[length(FoverFmsy)-2])) #Set next years F/Fmsy=this years F/Fmsy
SSB0  = data.table(mods1$derived_quants)[Label=='SSB_unfished']$Value
SB100=SSB0
Fmsy= data.table(mods1$derived_quants)[Label=='annF_Btgt']$Value
Fabc= data.table(mods1$derived_quants)[Label=='annF_MSY']$Value
xlim=c(0,2.2);ylim=c(0,1.6)
jitter.fac=0
Modelname="Model 24.0"

   
   plot(x=base::jitter(BoverBmsy,jitter.fac),y=base::jitter(FoverFmsy,jitter.fac),type="l",las=1,
          yaxs="i",xaxs="i",col="gray50",pch=20,xlim=xlim,ylim=ylim,xlab=expression(B/B[35~'%']),ylab=expression(F/F[35~'%']),
        main=paste(Modelname))
yr=seq(1991,2026,1)

  for(i in 1:(length(BoverBmsy)-2)){
    text(BoverBmsy[i],FoverFmsy[i],paste(yr[i]),cex=0.85)
    }
 for(i in (length(BoverBmsy)-1):length(BoverBmsy)){
    text(BoverBmsy[i],FoverFmsy[i],paste(yr[i]),col="purple",cex=0.85,font=2)
    }



k=Fabc/Fmsy*(((SSB0*0.2)/(SSB0*0.35))-0.05)/(1-0.05)
k2<-0.05
k3<-(0.2*SSB0)/(SSB0*0.35)
points(c(k3,k3),c(0,10),type="l",lty=3)
points(c((SB100*0.40)/(SB100*0.35),xlim[2]),c(Fabc/Fmsy,Fabc/Fmsy),type="l",lwd=5)
points(c((SB100*0.40)/(SB100*0.35),xlim[2]),c(1,1),type="l",lwd=5,col="red")
points(c(k3,(SB100*0.40)/(SB100*0.35)),c(k,Fabc/Fmsy),type="l",lwd=2)
points(c(k3,k3),c( 0,k),type="l",lwd=2)
points(c(k2,(SB100*0.40)/(SB100*0.35)),c(0,1.0),type="l",lwd=2,col="red")

text(xlim[2]-0.8,ylim[2]-0.1,"FOFL",pos=4)
text(xlim[2]-0.8,ylim[2]-0.2,"Max ABC",pos=4)
text (xlim[2]-0.8,ylim[2]-0.3,"B20% (no targeting)",pos=4)

points(c(xlim[2]-1.1,xlim[2]-0.8),c(ylim[2]-0.1,ylim[2]-0.1),lwd=2,type="l",col="red")
points(c(xlim[2]-1.1,xlim[2]-0.8),c(ylim[2]-0.2,ylim[2]-0.2),lwd=2,type="l")
points(c(xlim[2]-1.1,xlim[2]-0.8),c(ylim[2]-0.3,ylim[2]-0.3),type="l",lty=3)

abline(h=1,lty=2)
abline(v=1,lty=2)
```

`r figure_nums(name = "phase24_0", caption = " Phaseplane plot for Model 24.0, with forecasted results through 2026. Fishing mortality for 2025 and 2026 were estimated as the mean of the previous two years.")`

\pagebreak

\pagebreak

```{r plotREC,echo=FALSE,message=FALSE}

#REC=data.frame(M24_1$natage$Yr[which(M24_1$natage$`Beg/Mid`=="B")],M24_1$natage[which(M24_1$natage$`Beg/Mid`=="B"),][,13])
#colnames(REC)=c("Year","Recruitment")

#ggplot(REC)+geom_point(aes(x=Year,y=Recruitment))+theme_bw(base_size=14)

#var(REC$Recruitment[3:26])

#Recruitment deviations are calculated for years 1991 - 2021.
#Early recdevs are calculated in 1989 and 1990.
#mean(REC$Recruitment[3:23])
#sqrt(var(REC$Recruitment[3:26]))/mean(REC$Recruitment[3:23])

#REC=data.frame(M24_1$natage$Yr[which(M24_1$natage$`Beg/Mid`=="M")],M24_1$natage[which(M24_1$natage$`Beg/Mid`=="M"),][,13])
#colnames(REC)=c("Year","Recruitment")

#ggplot(REC)+geom_point(aes(x=Year,y=Recruitment))+theme_bw(base_size=14)

#var(REC$Recruitment[3:26])

#sqrt(var(REC$Recruitment[3:26]))/mean(REC$Recruitment[3:26])



#This is rec and stdev from the Report.sso M24_1. 
#REC2=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/M24_1_Rec_fromReportSSO.csv",header=TRUE)


#ggplot(REC2) +geom_bar( aes(x=Year, y=Recruitment), stat="identity", fill="skyblue", alpha=0.9) +geom_errorbar( aes(x=Year, ymin=Recruitment-1.96*StdDev, ymax=Recruitment+1.96*StdDev), colour="black", alpha=.6)+theme_bw(base_size=11)+scale_x_continuous(breaks = seq(1954,2040, by = 2))+theme(axis.text.x=element_text(angle=45,hjust=1))+ylab("Age-0 Recruitment (x1,000)")+ggtitle("Model 24.1 Age-0 recruitment with error bars")

#This is rec and stdev from the Report.sso M24_1. 
#REC2=read.csv("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_0_data/M24_0_Rec_fromReportSSO.csv",header=TRUE)


#ggplot(REC2) +geom_bar( aes(x=Year, y=Recruitment), stat="identity", fill="skyblue", alpha=0.9) +geom_errorbar( aes(x=Year, ymin=Recruitment-1.96*StdDev, ymax=Recruitment+1.96*StdDev), colour="black", alpha=.6)+theme_bw(base_size=11)+scale_x_continuous(breaks = seq(1954,2040, by = 2))+theme(axis.text.x=element_text(angle=45,hjust=1))+ylab("Age-0 Recruitment (x1,000)")+ggtitle("Model 24.0 Age-0 recruitment with error bars")

#mean(REC2$Recruitment[3:10])

# mean(REC2$Recruitment[11:21])

# mean(REC2$Recruitment[22:31])

#Plot SSB

#SSB2=data.frame(c(rep(seq(1991,2039),2)),c(M24_1$derived_quants$Value[3:51],M24_0$derived_quants$Value[3:51]),c(rep("M24_1",49),rep("M24_0",49)))
#colnames(SSB2)=c("Year","SSB","Model")
#ggplot(SSB2)+geom_line(aes(x=Year,y=SSB,col=Model))+theme_bw()



```

\begin{center}
\includegraphics[width=5in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/AIPcod2024/Rec_24_1.pdf}
\includegraphics[width=5in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/AIPcod2024/Rec_24_0.pdf}
\end{center}

`r figure_nums(name = "plotREC", caption = " Age-0 recruitment for Aleutian Islands Pacific cod, Model 24.1 (above), Model 24.0 (below). Recruitment deviations are calculated for years 1991 - 2021, and initial/virgin recruitment estimates used for years <1991. Mean recruitment is used for years 2022 onward.")`


\pagebreak

```{r OSA24_1, echo=FALSE,message=FALSE}
#Use runOSA.r where it says JS code. Change model number and fleet (1=fishery, 2=survey)
```


\begin{center}
\includegraphics[width=4.2in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/OSA/M24_1_fishery.pdf}
\includegraphics[width=4.2in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/OSA/M24_1_survey.pdf}
\end{center}

`r figure_nums(name = "OSA24_1", caption = " One-step-ahead (OSA) residuals for Model 24.1, fishery (above) and survey (below).")`

\pagebreak


```{r OSA24_1a, echo=FALSE,message=FALSE}
#Use runOSA.r where it says JS code. Change model number and fleet (1=fishery, 2=survey)
```


\begin{center}
\includegraphics[width=4.2in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/OSA/M24_1a_fishery.pdf}
\includegraphics[width=4.2in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/OSA/M24_1a_survey.pdf}
\end{center}

`r figure_nums(name = "OSA24_1a", caption = " One-step-ahead (OSA) residuals for Model 24.1a, fishery (above) and survey (below).")`


\pagebreak



```{r OSA24_0, echo=FALSE,message=FALSE}
#Use runOSA.r where it says JS code. Change model number and fleet (1=fishery, 2=survey)
```


\begin{center}
\includegraphics[width=4.2in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/OSA/M24_0_fishery.pdf}
\includegraphics[width=4.2in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/OSA/M24_0_survey.pdf}
\end{center}

`r figure_nums(name = "OSA24_0", caption = " One-step-ahead (OSA) residuals for Model 24.0, fishery (above) and survey (below).")`


\pagebreak

```{r yrclass, echo=FALSE,message=FALSE}
#Dir_M24_1=("/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run")
#ages=seq(1,13,1)
#lengths=seq(.5,142.5,by=1)

#index=lengths
#library(here)
#library(maditr)
##library(compResidual)
#library(cowplot)
#library(ggplot2)

#model=Dir_M24_1;lengths=seq(.5,142.5,by=1);fleet=2; sx=1; stck='AI_COD';surv='AI'

#mods1<-r4ss::SSgetoutput(dirvec=model)
#age<-data.table::data.table(mods1[[1]]$lendbase[,c(1,6,13,16:19)])[Bin%in%lengths & Fleet==fleet &Sex==sx]
#age<-data.table::data.table(melt(age,c('Yr','Fleet','Sex','Bin')))
#o<-age[variable=='Obs']
#o<-maditr::dcast(o,Yr~Bin)
#p<-age[variable=='Exp']
#p<-maditr::dcast(p,Yr~Bin)
#years<-o$Yr
#o <- as.matrix(o[,-1])
#p <- as.matrix(p[,-1])

#This is the survey
#yrs=years
#lengths=seq(.5,142.5,by=1)
#Frequency=c(p)
#Length=rep(lengths,each=length(years))
#Year=rep(years,length(lengths))
#fishlen=data.frame(Year,Length,Frequency)
#ggplot(fishlen, aes(Length,Year)) +
# geom_tile(aes(fill = Frequency),colour = "lightgrey")+
# scale_fill_gradient(low = "white",high = "darkblue")+
# ggtitle("AI cod survey predicted length frequencies")+theme_bw()+ 
# scale_y_continuous(breaks=years,labels=years)+xlim(0,115)

#This is the fishery
#p=p[,c(40:130)]
#o=o[,c(40:130)]
#yrs=seq(1991,2024,1)
#years=yrs
#lengths=seq(39.5,129.5,1)
#lengths=seq(.5,142.5,by=1)
#Frequency=c(p)
#Length=rep(lengths,each=length(years))
#Year=rep(years,length(lengths))
#fishlen=data.frame(Year,Length,Frequency)
#ggplot(fishlen, aes(Length,Year)) +
# geom_tile(aes(fill = Frequency),colour = "lightgrey")+
# scale_fill_gradient(low = "white",high = "darkblue")+
# ggtitle("AI cod fishery predicted length frequencies")+theme_bw()+ 
# scale_y_continuous(breaks=years,labels=years)


#ages
#age1=read.csv("/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2023/CodAges2.csv",header=TRUE)
#age2=age1[which(age1$AGE>0),]
#age2=age2[which(age2$Year>1990),]
#nyrs=length(table(age2$Year))
#Yrs=as.numeric(names(table(age2$Year)))
#Year=rep(Yrs,each=nyrs)
#Age=rep(seq(1,13),nyrs)
#Frequency=matrix(0,13,13)

#for (i in 1:nyrs){
#  for(j in 1:13){
#    Frequency[j,i]=length(age2$AGE[which(age2$AGE==j&age2$Year==Yrs[i])])
#  }
#}
#rownames(Frequency)=c(1:13)
#colnames(Frequency)=Yrs

#FrequencyN=matrix(0,13,13)
#for(i in 1:13){
#FrequencyN[,i]=Frequency[,i]/sum(Frequency[,i])
#}

#Freq=reshape2::melt(FrequencyN)

#AF=data.frame(cbind(Year,Freq,Age))
#colnames(AF)=c("Year","Var1","Year2","Frequency","Age")

#ggplot(AF, aes(Age,Year)) +
# geom_tile(aes(fill = Frequency),colour = "lightgrey")+
# scale_fill_gradient(low = "white",high = "darkblue")+
# ggtitle("AI cod survey observed age frequencies (normalized by year)")+theme_bw()+ 
# scale_y_continuous(breaks=Year,labels=Year)


```


\begin{center}
\includegraphics[width=5in,angle=0,origin=c]{/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/FishObsLF.pdf}
\includegraphics[width=5in,angle=0,origin=c]{/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/FishPredLF.pdf}
\end{center}

`r figure_nums(name = "yrclass", caption = " Length frequency of observed (above) and predicted (below) fishery length frequency of Aleutian Islands Pacific cod, 1991 - 2024.")`

\pagebreak



```{r yrclassSrv, echo=FALSE,message=FALSE}

```


\begin{center}
\includegraphics[width=5in,angle=0,origin=c]{/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/SrvObsLF.pdf}
\includegraphics[width=5in,angle=0,origin=c]{/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/SrvPredLF.pdf}
\end{center}

`r figure_nums(name = "yrclassSrv", caption = " Length frequency of observed (above) and predicted (below) survey length frequency of Aleutian Islands Pacific cod, 1991 - 2024.")`

\pagebreak



```{r yrclassAge, echo=FALSE,message=FALSE}

```


\begin{center}
\includegraphics[width=6in,angle=0,origin=c]{/Users/ingrid.spies/Documents/WorkDellStuff/Assessments/AI_Cod/2024/SrvObsAge.pdf}
\end{center}

`r figure_nums(name = "yrclassAge", caption = " Observed age frequencies of Aleutian Islands Pacific cod, 1991 - 2024.")`



\pagebreak

```{r jitter,echo=FALSE,message=FALSE}

#/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/M24_1/run/jitter/plots/Jittering results at converged solution.png

```


\includegraphics[width=6in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run/jitter/plots/Jitter24_1.pdf}

`r figure_nums(name = "jitter", caption = " The Model 24.1 jitter diagnostic for global convergence conducted on the Aleutian Islands Pacific cod assessment model 24.1. In the upper left panel, solid black circles represent the total likelihood obtained from 200 jittered model runs and the red horizontal dashed line represents the total likelihood value from the base-case model. The upper right panel is the spawning stock biomass (SSB) from jittered model runs. The lower panel shows the estimate of absolute fishing mortality, F, with F=sum(full Fs).")`

\pagebreak

```{r jitter24_1a,echo=FALSE,message=FALSE}

#/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/M24_1a/run/jitter/plots/Jittering results at converged solution.png

```


\includegraphics[width=6in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1a_data/run/jitter/plots/Jitter24_1a.pdf}


`r figure_nums(name = "jitter24_1a", caption = " The jitter diagnostic for global convergence conducted on the Aleutian Islands Pacific cod assessment Model 24.1a. In the upper left panel, solid black circles represent the total likelihood obtained from 200 jittered model runs and the red horizontal dashed line represents the total likelihood value from the base-case model. The upper right panel is the spawning stock biomass (SSB) from jittered model runs. The lower panel shows the estimate of absolute fishing mortality, F, with F=sum(full Fs).")`

\pagebreak

```{r jitter24_0,echo=FALSE,message=FALSE}

#/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/M24_0/run/jitter/plots/Jittering results at converged solution.png

```


\includegraphics[width=6in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_0_data/run/jitter/plots/Jitter24_0.pdf}


`r figure_nums(name = "jitter24_0", caption = " The Model jitter diagnostic for global convergence conducted on the Aleutian Islands Pacific cod assessment model 24.0. In the upper left panel, solid black circles represent the total likelihood obtained from 200 jittered model runs and the red horizontal dashed line represents the total likelihood value from the base-case model. The upper right panel is the spawning stock biomass (SSB) from jittered model runs. The lower panel shows the estimate of absolute fishing mortality, F, with F=sum(full Fs).")`


\pagebreak

\pagebreak



```{r lnQ,echo=FALSE}

```

\begin{center}
\includegraphics[width=6in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run/LikelihoodProfile_LnQ/lnQ.pdf}
\end{center}

`r figure_nums(name = "lnQ", caption = " Likelihood profile over the log of catchability (q), from -0.8 to 0.8 in increments of 0.1.")`

\pagebreak




```{r R0,echo=FALSE,message=FALSE}


```



\begin{center}
\includegraphics[width=6in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run/LikelihoodProfile_R0/R0.pdf}
\end{center}

`r figure_nums(name = "R0", caption = " Likelihood profile over initial recruitment, R0, from 10.4 to 12.")`

\pagebreak

\pagebreak

```{r retro_M2,echo=FALSE,message=FALSE}


```



\begin{center}
\includegraphics[width=4in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_0_data/run/retrospectives/Retro_Uncertainty.pdf}
\includegraphics[width=4in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run/retrospectives/Retro_Uncertainty.pdf}
\end{center}

`r figure_nums(name = "retro_M2", caption = " Retrospective plot of spawning biomass, Model 24.0 (upper panel), and Model 24.1 (lower panel). Mohn's rho is presented for each retrospective, followed by the forecasted estimate of rho in parentheses. The shaded area represents the 95% confidence interval around the mean of the base year (full time series).")`

\pagebreak




```{r Kobe24_1,echo=FALSE}

```

\begin{center}
\includegraphics[width=6in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/Kobe.pdf}
\end{center}

`r figure_nums(name = "Kobe24_1", caption = paste(" A Kobe plot demonstrating the stock status uncertainty over SSB/SSBmsy and F/Fmsy, indicates a 96.4% probability that the stock status is between SSB8% and SSB35%, and that the fishing mortality rate is below F40%. The triangle represents the first year (1991) and the large circle the final year (2024). Grey dots provide uncertainty among Model 24.1 runs for the stock status in the final year."))`




\pagebreak



```{r Kobe24_1a,echo=FALSE}

```

\begin{center}
\includegraphics[width=6in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1a_data/Kobe.pdf}
\end{center}

`r figure_nums(name = "Kobe24_1a", caption = paste(" A Kobe plot demonstrating the stock status uncertainty over SSB/SSBmsy and F/Fmsy, indicates a 95.3% probability that the stock status is between SSB8% and SSB35%, and that the fishing mortality rate is below F40%. The triangle represents the first year (1991) and the large circle the final year (2024). Grey dots provide uncertainty among Model 24.1a runs for the stock status in the final year."))`


\pagebreak


```{r Kobe24_0,echo=FALSE}

```

\begin{center}
\includegraphics[width=6in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_0_data/Kobe.pdf}
\end{center}

`r figure_nums(name = "Kobe24_0", caption = paste(" A Kobe plot demonstrating the stock status uncertainty over SSB/SSBmsy and F/Fmsy, indicates a 47.7% probability that the stock status is between SSB17.5% and SSB35%, 52.3% probability that the stock status is greater than SSB35%. The triangle represents the first year (1991) and the large circle the final year (2024). Grey dots provide uncertainty among Model 24.0 runs for the stock status in the final year."))`


\pagebreak

```{r Catch_scenarios24_1,echo=FALSE,message=FALSE}


```

\begin{center}

Model 24.1

\includegraphics[width=6in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1_data/run/AK_Scenarios/AK_Scenarios_CatchALL.pdf}

\end{center}

`r figure_nums(name = "Catch_scenarios24_1", caption = " Catch under seven NPFMC projection scenarios for Model 24.1.")`

\pagebreak

```{r Catch_scenarios24_1a,echo=FALSE,message=FALSE}


```

\begin{center}

Model 24.1a

\includegraphics[width=6in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1a_data/run/AK_Scenarios/AK_Scenarios_CatchALL.pdf}

\end{center}

`r figure_nums(name = "Catch_scenarios24_1a", caption = " Catch under seven NPFMC projection scenarios for Model 24.1a.")`

\pagebreak

```{r Catch_scenarios24_0,echo=FALSE,message=FALSE}


```

\begin{center}

Model 24.0

\includegraphics[width=6in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_0_data/run/AK_Scenarios/AK_Scenarios_CatchALL.pdf}
\end{center}

`r figure_nums(name = "Catch_scenarios24_0", caption = " Catch under seven NPFMC projection scenarios for Model 24.0.")`

\pagebreak



```{r Projection_scenarios24_1,echo=FALSE,message=FALSE}


```

\begin{center}

 Model 24.1

\includegraphics[width=6in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_0_data/run/AK_Scenarios/AK_Scenarios_ProjALL.pdf}

\end{center}

`r figure_nums(name = "Projection_scenarios24_1", caption = " Projected spawning stock biomass under seven NPFMC projection scenarios for Model 24.1.")`

\pagebreak



```{r Projection_scenarios24_1a,echo=FALSE,message=FALSE}


```

\begin{center}

 Model 24.1a

\includegraphics[width=6in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_1a_data/run/AK_Scenarios/AK_Scenarios_ProjALL.pdf}

\end{center}


`r figure_nums(name = "Projection_scenarios24_1a", caption = " Projected spawning stock biomass under seven NPFMC projection scenarios for Model 24.1a.")`


\pagebreak



```{r Projection_scenarios24_0,echo=FALSE,message=FALSE}


```

\begin{center}

 Model 24.0

\includegraphics[width=6in,angle=0,origin=c]{/Users/ingrid.spies/Documents/AI_PCOD/2024/Models/Sensitivity_Anal/November2024/M24_0_data/run/AK_Scenarios/AK_Scenarios_ProjALL.pdf}

\end{center}


`r figure_nums(name = "Projection_scenarios24_0", caption = " Projected spawning stock biomass under seven NPFMC projection scenarios for Model 24.0.")`


